<div id='9_'>
<h1>9. Conclusion</h1>

<p>In this final chapter, we look back on the work accomplished so far and comment on its contribution to advances in developmental modeling, and the difficulties encountered. We discuss our design decisions, the validity and limitations of the results obtained&mdash;and the results not obtained but expected. In Section 9.1, we recapitulate the criteria for a <q>good</q> model. This is followed by a review of the choices that we made, both on the biomechanic side (Section 9.2) and the molecular/genetic side (Section 9.3). Finally, we globally reassess the case studies (Section 9.4) and conclude with directions for future work (Section 9.5).

<!-- ====================================================================== -->
</div><div id='9_1_'><h2>9.1. Criteria for a <q>Good</q> Model</h2>
<!-- ====================================================================== -->

<p>The most striking characteristic of multicellular systems is their deep <i>heterogeneity</i> of properties and behaviors. As Henri Atlan stated through his famous metaphorical phrase <q>Between Crystal and Smoke</q> <a name='Atlan:1986ts' class='ref'></a>, the structure of living matter is somewhere between the very regularly organized and the very randomly disorganized. The strongest principle in any attempt to model such systems should be to provide a framework that allows the capture this heterogeneity. We presented in Chapter 2 our view about the most suited level of organization for this task.</p>

<h4>Microscopic models are too complex and uncontrollable</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>The molecular scale would be a tempting reductionist approach, since interactions between molecules are well described by models such as Molecular Dynamics. However, the hugeness of their number and the smallness of the timescale of their interactions render the practical simulation of multicellular tissue impossible with today's computing power. The other drawback of such an extreme bottom-up approach would be the difficulty to interpret the generated dataset. Large-scale modeling and simulation of similar magnitude have been investigated in other fields of experimental science, such as the <a href="http://bluebrain.epfl.ch/">Blue Brain Project</a> which now reproduces a cortical column containing tens of thousands of neurons and millions of synaptic contacts on highly detailed dendritic trees studded with ionic channels, and aims at scaling up to a whole human brain by the end of the next decade. There too, however, the twin problems of massive parameter tuning and <q>divergence of the emergence</q> remain as important as ever. In short: there are probably too many degrees of freedom and not enough a priori modeling assumptions.</p>

<h4>Macroscopic models are too constrained and not expressive enough</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>On the other side of the spectrum, macroscopic formal descriptions offer great conciseness with general principles of conservation and continuity. Whether differential equations in continuous Euclidean space or statistical laws in state space, a few parameters can be sufficient to accurately reconstitute and predict certain dynamical systems phenomena. For example, the field of statistical mechanics has been successfully exported from thermodynamics and adapted to other well-mixed systems than gases such as sand dunes, bird flocks or human crowds. It can provide important analytical results about their dynamical regimes, typically <q>critical</q> parameter values at which <q>phase transitions</q> occur. In the case of developmental biology, disciplines of physics such as fluid or solid mechanics can provide interesting models of cell densities and collective trajectories, acccounting for some aspects of tissue deformation. Here, however, the complexity and realism of these solutions is limited by the homogeneity of the parameters ruling the dynamics and the assumed continuity of the spatiotemporal distribution of variables. In contrast to the above scenario, we could say that the degrees of freedom are too few and the models too constraining.</p>

<h4>Mesoscopic models provide the best compromise</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>This is why we believe that the best level to study morphogenesis in multicellular organisms is somewhere inbetween those two extremes. Continuous equations have to be coupled with additional local rules, and in the end necessitate a characterization of the diversity of the underlying cell types and behaviors. This approach requires a discrete representation of the tissue based on individual components. In our own study of early embryo development, we chose the single-cell level as it is allows this local description of cell properties in relatively simple terms. The main criteria driving the choice of an appropriate dynamical framework are (a) the similarities between the variables involved in the model, on the one hand, and the observations and concepts construed and realized by the embryologists, on the other hand (discussed in Section 9.1.1); and (b) the simplicity of the description and the manageable number of parameters (discussed in Section 9.1.2). This should not only allow computationally feasible simulations by today's standards but, most importantly, a biologically meaningful interpretation of their outcome.</p>

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_1_1_'><h3>9.1.1. Similarities Between Variables and Concepts</h3>
<!-- ---------------------------------------------------------------------- -->

<p>One could say that embryology itself has greatly <q>evolved</q> since the <i>Entwicklungsmechanik</i> revolution started by Wilhem Roux. He transformed a descriptive area into an experimental discipline, which explored the <i>causal</i> links of development&mdash;whether mechanical, chemical, thermal, or electrical&mdash;leading to developed organisms. The principal debate of that time revolved around whether parts self-differentiated or depended on external influences. Since then, this puzzling question has been answered with increasing precision, first at the tissue level by the concept of <q>induction</q>, then at the cellular level with the advent of molecular biology. The great improvement of observation tools is now opening a new era that will master the automated integration of the cell fates of a complete organism (<q>in toto</q>). An experimental paradigm shift was triggered in 2006 with the mapping and reconstruction of all 959 cells of the <i>C. elegans</i> <a name='Bao:2006jc' class='ref'></a>. It now unfolds through projects and platforms such as BioEmergences in more complex species such as the zebrafish (see Section 7.1), to which we contributed our own methods of imaging data processing (see Section 7.2). While this is a work in progress, it offers a fundamentally new way of representing and understanding embryogenesis: the capacity of mapping a complete <i>cell lineage tree</i> augmented with local information such as cell positions <a name='Keller:2008km' class='ref'></a>, cell shapes, internal molecular states, gene expression through cis-regulatory reporters <a name='Dmochowski:2002ej' class='ref'></a>, and extracellular ligand concentrations. Based on these exhaustively reconstructed datasets, various correlations among quantitative observations, and among qualitative concepts, can then be detected and inferred by biologists.</p>

<p>This complex systems approach gives access to a new and vast empirical domain, which must be explored and charted from both the <q>classical</q> (see Section 1.1) and <q>theoretical</q> perspectives (see Section 2.1). Our point here is that the choice of a modeling paradigm must take into account this new phenomenology introduced by 3D+time reconstructions. Reconstructions and simulated embryos must <q>use the same vocabulary</q> to be able to define the grammar that links those two worlds. In practical terms, dynamical trajectories of the models gain in relevance if they reside in the same variable space as the live 3D+time reconstructions.</p>

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_1_2_'><h3>9.1.2. Computational Feasibility</h3>
<!-- ---------------------------------------------------------------------- -->

<p>The second criterion that drives the choice of a good modeling strategy is its suitability to current computer capacity. A balance has to be found between the number of variables and time steps, on the one hand, which impact the computational size and duration of a single embryo simulation, and the number of parameters that can be explored, on the other hand, which is combinatorially related to the total number of simulated embryos. It follows that these dimensions affect theoretical choices, too. For example, an <i>in toto</i> reconstruction of the zebrafish embryo until the bud stage, i.e. at about 25,000 cells and 10 hours of development, imposes much greater demands on the design of an encompassing model than a few dozen cells during a couple of hours. A concrete example can be found in the work of Sandersius, Newman et al. on the Subcellular Element Model (ScEM, see Section 3.1.1). The number of subcellular elements they use for single-cell studies is by far greater than in many-cell populations: typically 4,000 elements in a rheological experiment involving a single cell <a name='Sandersius:2008ia' class='ref'></a> versus only 20 elements per cell to model the large-scale vortices appearing during the primitive streak formation <a name='Sandersius:2011ew' class='ref'></a>).</p>

<p>In summary, these two criteria (variable-concept resemblance and practical computation) led us to make particular choices for (i) the cell biomechanics (discussed in Section 9.2) and (ii) the molecular/genetic regulation and signaling (discussed in Section 9.3).</p>

<!-- ====================================================================== -->
</div><div id='9_2_'><h2>9.2. Choices Concerning the Biomechanics</h2>
<!-- ====================================================================== -->

<p>The biomechanical paradigm used in MECAGEN is a discrete description of cells represented by single particles. In Chapter 3, we presented our framework by distinguishing two steps in the elaboration of the model: first, the construction of the cell contacts (see Section 3.2.2), and second, the forces acting between neighboring cells (Sections 3.2.3-4). We discuss these choices here.</p>

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_2_1_'><h3>9.2.1. Spatial Neighborhood</h3>
<!-- ---------------------------------------------------------------------- -->

<p>Cell-cell interactions represented by particles must follow a strict principle: particles exert forces only over those particles with which they are in physical contact. No (direct) distant action similar to gravity or magnetic forces are applicable in the context of cell tissue&mdash;other than long-range axonal and dendritic synaptic transmission among neurons, but this dynamics is not yet in existence at the early embryonic stages. Propagation of mechanical constraints across a tissue can also be considered long-distance action, but it actually happens through local cell-to-cell contacts. In a model representation, this physical-contact principle translates into <i><q>neighborhood links</q></i> that must be defined and selected.</p>

<h4>Metric neighborhoods are simple but can create unstable dynamics</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>A common and easy way to determine a spatial neighborhood is to set a maximal Cartesian distance within which cells can be in contact: this is the scope-limited <q>nearest neighbors</q> scheme, also called <q>metric</q> neighborhood, which we denoted by \(\mathcal{N}^m_i\). Our early attempts to model cell-cell interactions were based on this metric notion, but we quickly observed that this approach had an important drawback: increasing the attraction part of the force potential (coefficient \(w_{\mathrm{adh}}\)) inevitably led to unstable dynamics, where cells collapsed upon each other and eventually fused. The reason was simple: cells were simultaneously repelling their closest neighbors (those below the equilibrium distance \(r^{\mathrm{eq}}_{ij}\)) and attracting the farther away neighbors (those beyond \(r^{\mathrm{eq}}_{ij}\)). Problems happened when cells started attracting neighbors located <i>behind</i> other neighbors. In this situation, a large adhesion coefficient would trigger a <q>greedy</q> attraction of more and more neighbor cells, leading to a dramatic shrinkage and collapse of the tissue.</p>

<h4>Topological neighborhoods are rational but computationally expensive</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>The solution we adopted to alleviate this problem was then to use a more realistic neighborhood definition that would take into account the nearby cells \(j\) truly <q>in contact</q> with a given cell \(i\). This is the <i>topological</i> criterion \(\mathcal{N}^t_i\), which formally defines the concept of being located <q>behind</q> something else. This solution has been demonstrated by the work of M. Meyer-Hermann and G. Schaller, who introduced the systematic use of the weighted Delaunay triangulation in a biomechanical simulation <a name='MeyerHermann:2008do' class='ref'></a>. In 2D, the Delaunay triangulation algorithm allows to define neighborhood links based on a geometrical criterion, i.e. by forbidding the links that would produce triangles containing particles in their circumcircles. The 3D counterpart extends this criterion to the equivalent simplex: the tetrahedron. This approach, however, is computationally intensive as it has to deal with heavy data structures, so we decided to use instead a related algorithm based on the Gabriel graph <a name='Gabriel:1969tw' class='ref'></a>. This solution is easier to implement and faster to simulate, as the modified exclusion criterion is that every circle of which a neighborhood link is the diameter must not contain any other particles (than the link's two extremities). We extended this criterion in 3D with spheres instead of circles. This strategy allowed us to avoid the collapsing behavior caused by high adhesion coefficients in the metric case.</p>

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_2_2_'><h3>9.2.2. Interaction Potential and Behavioral Forces</h3>
<!-- ---------------------------------------------------------------------- -->

<p>Once spatial relationships between neighbors are established, the next step is to define <i><q>forces</q></i> responsible for the mechanical interactions. The main principle of the motion equation is that, in multicellular systems, the cells are so small and their interactions so sticky that their motion is not well described at their scale by classical mechanics. Instead of the acceleration, it is rather the velocity of an element that is proportional to the sum of the forces applied to it. The consequence is that mass-based inertia plays no part in cell motion, which is rather due to the instantaneous action of forces <a name='Purcell:1977tk' class='ref'></a>. This principle is widely adopted in various models of multicellular assembly <a name='Odell:1981vy' class='ref'></a> <a name='Newman:2005ux' class='ref'></a> <a name='Sherrard:2010ii' class='ref'></a>. For the definition of the forces <i>per se</i>, we decided to dissociate two types of interactions: <q>passive</q> adhesion/repulsion interactions responsible for the relaxed state of the assembly, and <q>active</q> forces (such as protrusion) responsible for the inidividual behavior of the cells.</p>

<h4><q>Passive</q> adhesion/repulsion interaction potential</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>Concerning the first type of interactions, we explored several models of potentials offering long-range attraction and short-range repulsion: the Morse potential <a name='Newman:2005ux' class='ref'></a>, the Lennard-Jones potential <a name='Flenner:2008fi' class='ref'></a>, Dissipative Particle Dynamics <a name='Ranft:2010gm' class='ref'></a>, and contact mechanic models derived from the Hertz model such as Johnson-Kendall-Roberts <a name='Chu:2005ti' class='ref'></a>. The conclusion that we drew from this practical study was that, even if each model had its own specificity as it was designed to study other phenomena, they all shared the same peculiarity and limitation the single-particle cell framework: here, the repulsion part of the potential is responsible for both the integrity of the cellular domain and the stiffness of the response. Thus it is difficult to pretend that the interaction potential model correctly integrates the biomechanical properties of cortical tension in a dense multicellular context. Other single-particle cell models have added a <i>volume conservation</i> force to avoid cells becoming smaller when the adhesion coefficient increases, even in the topological neighborhood context <a name='MeyerHermann:2008do' class='ref'></a>. In our study, we decided to select a simple solution, in which elastic forces are scaled by our estimation of the surface of contact. In addition to displaying a similar profile (Fig. <a name='potential_potential_superpose' class='fig'></a>) with a vanishing force beyond a maximum distance limit, this solution also properly deals with rescaling the force when cells' size is dramatically reduced during the early stages of development. The repulsion coefficient \(w_{\mathrm{rep}}\) controls both the cell's volume conservation and stiffness of response until equilibrium, as these two properties are inseparable. The adhesion coefficient \(w_{\mathrm{adh}}\) controls the adhesive property of the cells beyond equilibrium.</p>

<h4><q>Active</q> behavioral forces</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>The second type of interaction forces introduced in this work models the specialized proactive behaviors of the cells. Within the possible repertoire of a cell behavior ontology (CBO, Section 5.1.1.), we focused in this study on a central mechanism that we extensively used in the cases studies in Chapter 8: mesenchymal cell <i>protrusion</i>. Protrusive behavior constitutes a principle that is rarely used in multicellular studies (one recent publication, posterior to and independent from us, uses it for the same purpose <a name='Yamao:2011eh' class='ref'></a>). Yet, it plays a key role in active migration and intercalation <a name='Keller:2002ko' class='ref'></a> <a name='Keller:2003tl' class='ref'></a> <a name='Keller:2005kx' class='ref'></a>. Several models have proposed mechanisms to explain this behavior in a single cell, for example Weliky et al. introduced an explicit deformation of the cell shape in their study of the convergence-extension in the <i>Xenopus</i> notochord <a name='Weliky:1991vf' class='ref'></a>. We believe, however, that a formalization of this behavior makes sense only if the protruding cell has a support to exert its action upon. It means that models of cell migration should always associate to the migration force intrinsically generated by a protruding cell a counterpart force on its physical support (i.e. the surrounding cells in the neighborhood) oriented in the opposite direction. Without this counterpart force, the model would be like the action of an external actor pushing the cell in their migratory direction. This principle, when applied to an assembly of protruding cells, creates a non-trivial emergent behavior of tissue deformation and legitimates in itself the need of computer simulation to observe and analyze it. We defend the notion that this mechanism could explain various embryonic phenomena observed during the zebrafish early development (Case Studies 8.5, 8.6). Another interesting approach is proposed by Sandersius et al. in their Subcellular Element Model framework (ScEM) <a name='Sandersius:2011ew' class='ref'></a>: they model the process of cytoskeletal polymerization involved in protrusion by adding subcellular elements at the tip of the protruding pole and removing other subcellular elements at the other end of the cell. This mechanism, coupled with cell-cell adhesion, also induces a migratory behavior in the cell population.</p>

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_2_3_'><h3>9.2.3. External Objects</h3>
<!-- ---------------------------------------------------------------------- -->

<p>One of the main advantages of particle-based modeling (whether single- or multiple-particle) is the possibility to seamlessly aggregate to the multicellular model external physical objects that have their own dynamical properties. This point is illustrated in our thesis with the addition of the large yolk cell supporting zebrafish development. It is clear that no relevant insight on the mechanics of the zebrafish development could be gained without taking into account the yolk structure. The particle physics framework can alleviate the discontinuities generated by these external structures relatively easily: they, too, are modeled with particles (here, the yolk cell is a high-degree polyhedron) and contacts between structure particles and cells are exactly of the same kind as cell-cell contacts, only with different force parameters. One could compare this framework to <q>Lego</q> bricks, which despite their different shapes and sizes all have the same pegs and holes by which they can attach to each other. In future studies, this standardized framework can be extended to other types of extracellular interactions in a straightforward manner.</p>

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_2_4_'><h3>9.2.4. Single-Cell vs. Subcellular Mechanics</h3>
<!-- ---------------------------------------------------------------------- -->

<p>The single-particle cell model also has drawbacks, however. Among them, the <q>coarse-graining</q> performed at this level of abstraction is too high to integrate the subcellular structures involved in the mechanics of embryonic morphogenesis. An example of this is the extra-cellular matrix (ECM) that has been shown to participate in the establishment of morphogenetic domain boundaries <a name='Rozario:2009fz' class='ref'></a>. ECM is a dynamical network of filament molecules such as fibronectin and fibrillin <a name='Czirok:2006ia' class='ref'></a> secreted by the cells. Despite this, the absence of ECM in the model can be partly justified by the results of a recent study showing that it displays actually little relative motion with respect to the surrounding cells, both ECM and cells moving together as a single mass <a name='Zamir:2008ge' class='ref'></a>. This would imply, conversely, that cells apply only little force on the ECM because, similarly to our cell-cell protrusion mechanism, any force would induce ECM motion in the opposite direction, which is not observed. The mechanical influence of the ECM also appears limited in the early stages of zebrafish development as its expression is only observed after 80 percent epiboly, coinciding with the convergence-extension episode <a name='Latimer:2009hl' class='ref'></a>.</p>

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_2_5_'><h3>9.2.5. Spherical-Voronoi Approximation</h3>
<!-- ---------------------------------------------------------------------- -->

<p>Another issue with the single-particle paradigm is the lack of explicit cell shape description, making it necessary to use an approximation (i.e. a statistical average) to calculate the contact surface area between cells. In the project, as we lacked real embryo membrane segmentations during the later stage of zebrafish development, we opted for a empirical law of the surface defined by weighted Voronoi domains based on abstract numerical experiments. We expect that this approximation is acceptable when cells are densily packed <a name='LuengoOroz:2008vc' class='ref'></a>, but when they present <q>exposed</q> facets (i.e. not touching other cells), the hybrid spherical-voronoi approximation of the particles does not properly account for cell shapes. This is particularly striking during the first stages of the zebrafish development, where blastomeres form large cells spread out on the surface of the yolk. There, the model would benefit from finer-grained representations such as the ones provided by the Subcellular Element Model or the Cellular Potts Model.</p>

<!-- ====================================================================== -->
</div><div id='9_3_'><h2>9.3. Choices Concerning the Molecular/Genetic Regulation and Signaling</h2>
<!-- ====================================================================== -->

<p>This part of our model follows the same criteria of vocabulary and computational suitability. The interface we use here to specify the parameters is the BioTapestry software, which is the most popular visualization tool for gene regulatory networks (GRNs) involved in animal development. We believe that this strategy should facilitate the communication with researchers familiar with this environment. From the theoretical perspective, in order to describe the regulation kinetics, we preferred the conventional system of ordinary differential equations (ODEs) over other paradigms. For sure, more accurate simulations of the molecular dynamics could be obtained from finer-grained frameworks such as stochastic event-based models using Gillespie's method <a name='Gillespie:1977ww' class='ref'></a> or rule-based models like the <a href="http://www.kappalanguage.org/">Kappa language</a> representing protein interaction networks. It has been shown that important molecular dynamical phenomena occur when the number of molecules is low <a name='Fedoroff:2002by' class='ref'></a>, as even individual molecules can play a role in the stability of the cell state. Thus models at the molecular scale might better capture this singular dynamics than ODEs describing average concentrations. Yet again, just like in the mechanical part, their computational cost is also higher and does not suit our objectives in terms of simulation time. Looking in the opposite direction, even faster simulations could be obtained with Boolean networks instead of real-value variables, with adequate performance fitness defined on real embryo datasets <a name='Peter:2012dy' class='ref'></a>. But while Boolean network are well suited to the cis-regulatory logic of GRNs (which we included in our model, see Chapter 4), concerning other components of molecular and genetic regulation, such as protein-protein interactions or cell signaling, the parametric space of ODEs is better adapted to direct experimental measures and validation.</p>

<!-- ====================================================================== -->
</div><div id='9_4_'><h2>9.4. Assessment of the Case Studies</h2>
<!-- ====================================================================== -->

<h4>Translating state-of-the-art biological knowledge into models</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>The life sciences have generated a remarkable wealth of knowledge on multiple scales. A model such as the one that we presented in this thesis exploits only a small fraction of this immense corpus. Many improvements are still possible within the currently available <q>state of the art</q>. Yet, modeling also implies <i>selecting</i> specific subparts of this information, and requires an extensive production of experimental measures to calibrate the various parameters involved in the equations. Thus, on the modeling side, the main problem is the high number of parameters, making their exploration time consuming and difficult.</p>

<p>The strategy that we adopted in the case studies was to separate the parameters into two categories in each case study: ones set to fixed values based on qualitative estimations, and others left open to exploration. Naturally, the choices presiding over the first category are subjective and debatable. It is possible that changing the values of some of the <q>fixed</q> parameters, i.e. transferring them to the second category, could generate simulated phenotypes that would perform equivalently or even better than the ones subject to reevaluation.</p>

<p>On the positive side, this subjective, <q>hand-made</q> aspect confers a particular meaning to the important mechanisms that are involved. On the negative side, it also signals a lack of systematicity or exhaustiveness which would be ideally required for an in-depth understanding of the model and, through the model, the biological mechanisms themselves. A classical solution is to calibrate some of the parameters via direct measurements obtained from the real systems. In our project, this calibration phase was not done because (a) the direct relationship between most of the parameters and their real counterparts needed further analysis, and (b) some of the experimental measures were just not yet available.</p>

<ul>
  <li>(a) The calibration phase of the mechanical parameters of the model could be done through specific <i>in silico</i> experiments aiming at characterizing the microscopic behavior of the system. We may cite as an example the values attributed to the adhesion and repulsion coefficient of the passive relaxation forces. In the project, this evaluation was reverse-engineered via an evaluation of the macroscopic behavior of the system. In the first case study, <q>Investing the Yolk Biomechanical Properties</q> (Section 8.1), the optimal values were obtained through a macroscopic perturbation of the system&mdash;in this case, a pressure exerted over the yolk surface. This type of the study could be generalized at the cell level through systematic perturbation experiments applied on single cells. This would be similar to the adhesion measures in germ-layer progenitors by single-cell force spectroscopy in Krieg et al. <a name='Krieg:2008jd' class='ref'></a>, or at the tissue level, to the study of the effective viscosity of cell aggregates in Marmottant et al. <a name='Marmottant:2009cw' class='ref'></a>.</li>

  <li>(b) Most parameters used in the model would also require <i>in vivo</i> measurements that have not been realized yet. Two of the most important parameters in MECAGEN are the intensity \(f^A\) of the protrusive forces and the polarization axes \(\vec{U}_i\). Calibrating the protrusive forces could already be feasible with <i>in vitro</i> measures of migrating individual cells. Experiments have been designed to quantitatively assess the cellular traction stresses by traction force microscopy in 2D culture cells <a name='Parker:2002ek' class='ref'></a>. It has also been done in a 3D environment as shown in a recent study of 3T3 fibroblasts migration <a name='Franck:2011jb' class='ref'></a>. This study aims at deciphering the protrusive force patterns occurring in cell-ECM migration. The next step would be to apply this methodology in a multicellular assembly, first <i>in vitro</i> on cells extracted from developing embryos, then eventually <i>in vivo</i>. Calibrating the polarization axes appears to be a more immediate challenge, as the conjunction of intracellular molecular markers of polarization and large-scale <i>in vivo</i> reconstruction platforms like BioEmergences should produce 3D+time maps of the polarization fields at the scale of the embryo.</li>
</ul>

<h4>What is needed from biology to make a modeling endeavor successful</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>The whole process of calibrating the parameters would be facilitated by provided access to databases of reconstructed embryos. The sophisticated reconstruction methods that are being developed today to generate <i>in toto</i> measurements of the spatial-temporal properties of the cells in developing embryos will soon become standard tools in the lab of tomorrow. Today's solutions, however, are not yet at this stage of completion or generality, as they are still very much dependent on particular microscope setups. Future advances in microscope technology, both in the acquisition time and resolution dimensions, should produce images that could be reconstructed on plain laptops. In the meantime, theoretical studies of development would greatly benefit from an easier access to the existing reconstructed embryos. This request raises two challenges: the computational infrastructure and, more importantly, the standardization of the storage format. This latter point meets the challenge of designing a <q>Cell Behavior Ontology</q> that would be accepted by the whole community.</p>

<h4>The persistent enigmas of zebrafish early development</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>Among the multiple questions that the first ten hours of zebrafish development raise, on of the most puzzling concerns the shape transition observed at the sphere stage. Although it has been observed and recognized for decades now, this famous transition characterized by a flattening of the yolk-blastoderm interface has not been studied extensively enough by the zebrafish research community. In the fifth case study, about <q>Intercalation Patterns</q> (Section 8.5), we were simply unable to generate a simulated phenotype that would resemble the sphere stage embryo. This may be due to the existence of an additional, unknown mechanism that was not taken into account in our model. To our knowledge, however, no extra mechanism has been proposed or hypothesized by developmental biologists. Since this transition does not seem to produce a new apparent embryonic structure, this deformation could be the consequence of an unknown mechanism by which the zebrafish would <q>skip</q> the sphere stage (simply transitioning from the oblong stage to dome stage) without losing a morphological structure. However, this transition appears to be conserved in teleost development, as illustrated by the developmental stage 11 shown in <a name='Iwamatsu:2004el' class='ref'></a>.</p>

<h4>The help of agent-based modeling in elucidating those questions</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>The sphere stage transition is the only developmental stage that poses a problem in the first 10 hours of the zebrafish development. All the other major morphogenetic events are likely to be reproduced by the implemented rules presented in this project. This highlights the capacity of the MECAGEN simulation platform to handle the great number of heterogeneous interactions occurring during embyronic development. However, the current state of this study is situated somwhere between a <q>proof of concept</q> validating the methodology and the actual realization of the complete morphogenetic development that is planned for the future.</p>

<h4>About reconstructed data and measures that were not exploited by our model</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>One type of measures that was evoked in Section 1.3 about methodological considerations but not been exploited in the case studies comprise the <q>extensive microscopic</q> measures. They consist of <i>in toto</i> <q>fields</q>, ranging from local displacement vectors to more structured objects such as rearrangement pattern tensors <a name='Graner:2008ew' class='ref'></a>. This type of measure raises the issue of the automated spatiotemporal registration of embryos (simulated or live). In the third case study, <q>Shaping the Blastula</q> (Section 8.3), we mentioned that we planned on superimposing the spatiotemporal coordinates of the cell centers from simulated and live specimens (such as shown in Movie <a name='Case_2_Cleavage_video_example_Cleavage_example' class='fig'></a>). This registration could be done manually by visual estimation of the geometrical transformation parameters. However, the ideal solution necessitates the design of a <i>prototype embryo</i>. This concept is based on the idea that all developing embryos are mere variations on a single structural theme, the <q>prototype</q>, whose components may slightly vary in each individual specimen. Designing the prototypical embryo is an important project that has not been realized yet, and would itself require building macroscopic measures. Using data-mining methods to <q>learn</q> the different associations between these observations would allow to automatically recognize the morphogenetic events occurring in a phenotypic trajectory. The spatiotemporal coordinates of the embryonic structures would thus be associated to the other macroscopic measures. Such a system could then detect the macroscopic features extracted from raw data imaging (e.g. an internalizing tissue in the dorsal region) and recover their spatiotemporal coordinates in a prototypic template (e.g. the imaged data have a the spatial coordinates of the generic internalization shield and the associated timing is 6 hpf). This prototyping tool would contribute to great advances in the comparison of embryos and the fusion of partial heterogeneous datasets.</p>

<h4>Installing and exploiting an evolutionary search</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>The final parameter exploration carried out in our various case studies was performed by regularly sampling or <q>sweeping</q> parameter space, and producing more or less complete landscapes of the associated measures or fitness functions. This strategy is far from optimal if we aim at finding the <q>best</q> set of parameters, especially in higher dimensions and larger intervals. Other strategies of optimization, such as gradient descent, simulated annealing, or probably best if all, an <i>evolutionary search</i>, would be more efficient as they would converge faster toward a solution. However, as we were not only trying to find the best set of parameter, but also understand the relationship between the parameters and macroscopic measures, we decided to survey and chart whole regions of parameter space&mdash;which was difficult in 4D and beyond. Advanced <q>genetic algorithms</q> evaluating multi-objective functions could prove a fruitful direction to improve our treatment of parameter space, especially in higher dimensions. But the problem of visualization would remain. Another issue with automated exploration techniques is that they may exploit every flaw or idiosynchratic aspect of the measures in the <q>wrong</q> way, meaning that they can find hyperspecialized solutions that perform really well but cannot be interpreted or generalized (e.g. the margin-rolling phenotype observed in Case Study 8.5, see Movie <a name='Case_4_Intercalation_yolk_perforation' class='fig'></a>).</p>

<!-- ====================================================================== -->
</div><div id='9_5_'><h2>9.5. Future Work</h2>
<!-- ====================================================================== -->

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_5_1_'><h3>9.5.1. Portability to Other Animal Models</h3>
<!-- ---------------------------------------------------------------------- -->

<p>The MECAGEN project was envisioned as a generic modeling platform for all types of animal development. Its foundational principles should be applicable to any multicellular system involving biomechanics and molecular/genetic regulation and signaling. The next step in the development of this platform is to expand its cell behavior ontology (CBO) to integrate other types of epithelial behaviors. Not many improvements should be necessary to fulfill this objective. The adhesion coefficient of <q>lateral</q> neighborhood links could be increased and the epithelial behaviors mentioned in Section 5.1.1., apical constriction and active intercalation in the epithelial sheet, should allow the platform to simulate monolayered embryos such as <i>Drosophila</i>. Apical constriction can be modeled by adding couples of intrinsic and extrinsic behavioral forces, \(\vec{F}^{A,int}\) and \(\vec{F}^{A,ext}\), where the latter must be oriented toward the apico-basal axis above its apical region. This simple mechanism should trigger the invagination of the epithelium. Cell intercalation in the <i>Drosophila</i> epithelium is thought to operate by local junction remodeling <a name='Rauzi:2008gz' class='ref'></a> and involve a molecular mechanism that differs from the protrusive behavior observed in mesenchymal cells. Yet, we believe that this mechanism could also be modeled by the same couple of behavioral forces that we used in mesenchymal protrusion. The only difference would consist of imposing that the polarization axis supporting this intercalation be constrained in the epithelial plane.</p>

<p>This being said, however, the current MECAGEN implementation is also not compatible with every animal cell behavior. Its major limitation resides in the single-particle cell abstraction, which is not adapted to cellular shapes that depart greatly from spheroids or cuboids. Other types of animal morphogenesis are based on cell elongation, as in <i>Phallusia</i> or nematodes. In those cases, an ellipsoid particle model such as Palsson's <a name='Palsson:2001um' class='ref'></a> provides a reasonable solution for asymmetric cells. In later stages of vertebrates, too, cells differentiate into extremely stretched shapes such as muscle cells, for which the single-particle framework is clearly inappropriate. As mentioned previously, another solution is to descend to a lower scale involving a finer-grain description based on subcellular elements, which should be able to mimic those unconventional cell shapes.</p>

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_5_2_'><h3>9.5.2. Interactive Exploration of Parameter Space</h3>
<!-- ---------------------------------------------------------------------- -->

<p>Exploring parameter space in the case studies was not without frustration, as the interpretation of the fitness landscapes did not clearly point to the underlying phenotypes associated to each measured value. A future avenue of research that should facilitate this exploration would include the design of <i>interactive</i> landscapes. One can envision fitness landscapes that would be <q>clickable</q> by a human user, and the selection of certain points in that space would display the associated phenotypes in the measure space (e.g. the temporal plot of the doming macroscopic measures shown in Section 8.5), and also in the variable space via 3D rendering of the developmental trajectories. This work would require much more software engineering but all the elements are in place to rapidly develop a workable solution. The phenotypic visualization could be pre-calculated but in the case of thousands of embryos that each comprises millions of time-dependent variables, new simulations would be triggered by this hand-selection of parameters. To our knowledge, no interactive exploration solution of this type exists yet, although it would be of great benefit to the study of all kinds of complex dynamical systems.</p>

<!-- ---------------------------------------------------------------------- -->
</div><div id='9_5_3_'><h3>9.5.3. Toward an <q>Evo-Devo</q> Perspective</h3>
<!-- ---------------------------------------------------------------------- -->

<p>Another promising extension of this work, once the <q>MECA</q> and <q>GEN</q> parts of the model are finally connected and their dynamics truly coupled, will be its potential application to <q>evolutionary developmental</q> questions (mentioned in the introduction of Section 2.2). Evo-devo does not generally aim at directly comparing simulated phenotypes and real embryos but rather askking how evolution is able to generate new structures and behaviors at the level of the whole embryo. This would involve modeling and simulating a higher level of organization yet, in which the whole embryo behaves and potentially interacts with its environment. It can be seen as another way to explore the model's parameter space without isolating the developmental trajectories around a target phenotype (zebrafish in this project). The evaluation would emerge from a Darwinian selection process in an artificial environment&mdash;whether by survival or reproduction of particular behaviors. A particularly fascinating exploration would be to start from a simple individual organism such as the <i>urbilaterian</i>, a hypothetical ancestor of all animals exhibiting bilateral symmetry <a name='Erwin:2002wh' class='ref'></a> <a name='Hejnol:2008gl' class='ref'></a>, and guide its evolution through mutation of its gene regulatory networks toward multiple descendant species. To our knowledge, no computational evo-devo study has proposed a developmental model presenting as many similarities with real biological systems as MECAGEN. It would be a great opportunity to better decipher and understand the evolutionary forces that drive the diversity of life.</p>

</div>
 