<!-- ********************************************************************** -->
<div id='4_'><h1>4. MECAGEN Genetic and Molecular Regulatory Networks</h1>
<!-- ********************************************************************** -->

<p>The goal of this chapter is to briefly explain the principles of gene regulatory networks (GRNs), describe the components of GRN models, and give examples. In the present work, our particular objective was to design a simple and easily computable model of the molecular and genetic interactions that occur during development. This domain is a subject of intense research [ref book U. Alon XXXXX], at the crossroads between bioinformatics, systems biology and chemical kinetics; nonetheless, we believe that relevant insight can already be gained by using simple rules. We articulate our model around three types of rules:</p>
                    
<ul>
	<li>rules driving the dynamics of intracellular gene/protein reactions</li>
	<li>rules driving the dynamics of cellular secretion and transduction</li>
	<li>rules driving the dynamics of extracellular reactions, transport and diffusion</li>
</ul>

<p>To facilitate the specification of the parameters of molecular and genetic regulation, we interface the model with the <a href="https://www.systemsbiology.org/biotapestry">BioTapestry</a> software developed by W. Longabaugh and H. Bolouri at the Eric Davidson Lab, Caltech <a name='Longabaugh:2005bq' class='ref'></a> <a name='Longabaugh:2009kp' class='ref'></a> (Fig. <a name='model_grns_zebra_grn_yuh' class='fig'></a>). This software has been adopted by the developmental biology community as the standard tool to visualize gene regulatory networks. We use it to draw our own GRN and add different module parameters, then export it as an XML file used as an input into our model.</p>

<figure id='model_grns_zebra_grn_yuh'>
	<img src="../../images/model_grns/zebra_grn_yuh.png" width="600">
	<figcaption>BioTapestry representation the development GRN of the Zebrafish presented in <a name='Chan:2009er' class='ref'></a>.</figcaption>
</figure>

<h4>Intra-Cellular Gene/Protein Reactions</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>To simplify the interactions inside the cell, we restrict the variables to real-valued quantities \(\mathbf{p}=\{p_a\}\) representing the different proteins \(\{P_a\}\), with \(a=0...N_p\!-\!1\), and binary states \(\mathbf{g}=\{g_b\}\) representing the expression levels of genes \(\{G_b\}\) involved in the GRN, with \(b=0...N_g\!-\!1\). Note that \(p_a\) can stand either for an actual "number" of molecules or for a "concentration" of molecule type (i.e. the number divided by the volume of the cell). This can be justified by assuming that every cell occupies about the same volume, and considering that the extracellular space also belong to the Voronoi territories of the cells (i.e. the abstract "borderline" between two cell territories runs along the middle of their interstitium).</p>

<p>The role of RNA is bypassed, as transcription and translation are merged into a single process. Thus in the following, the term "protein" denotes a unique molecule type synthesized by a corresponding gene, whether it is actually a protein or mRNA. Then, this "protein" may either react in a protein-protein interaction, or act as a transcription factor reacting on a gene regulatory element.</p>

<p>Here, each gene \(G_b\) produces a single protein type \(P_{a=b}\), but on the other hand there can be more protein types than genes (i.e. \(N_g \leq N_p\)) because the GRN model covers only a small part of the whole genome and some proteins present in the cell can originate from extracellular signaling by neighboring cells. Proteins synthesized internally are in the same order as the genes in the interval \([0, N_g\!-\!1]\), then proteins of external origin are indexed in the interval \([N_g,N_p\!-\!1]\), if it is not empty. Protein concentrations \(\mathbf{p}\) evolve according to three rules:</p>

<ul>
	<li>protein-protein interactions (first subsection below)</li>
	<li>synthesis by encoding genes (second subsection)</li>
	<li>degradation by the molecular environment (third subsection)</li>
</ul>

<p>Gene activities are regulated by the proteins via Boolean functions representing a logical combination of promoters and repressors (fourth subsection).</p>

<h5>Protein-Protein Interactions</h5>
<!-- ............................ -->

<p>Here we consider only protein-protein reactions involving two reactants and one product, generally expressed by:</p>

<p>$$\alpha_0 P_0 + \alpha_1 P_1 \overset{k}{\rightarrow} P_2$$</p>

<p>where \(k\) is the rate coefficient. The corresponding rate equation reads:

<p>$$\frac{dp_2}{dt} = k\;p_0^{\alpha_0} p_1^{\alpha_1}$$</p>

<p>Further simplifying assumptions involve uniform stochiometric coefficients \(\alpha_0 = \alpha_1 = 1\), and one of the two protein reactants in much greater concentration than the other, e.g. \(p_0 \gg p_1\), in which case the rate equation boils down to a first-order type:</p>

<p>$$\frac{dp_2}{dt} = -\frac{dp_1}{dt} = k' p_1$$</p>

<p>where \(k' = kp_0\) is the new "pseudo-coefficient" of the variation. Naturally, this very simple schema has important limitations. Yet, when applied to many nodes of a large network of molecular interactions inside each cell, and also combined with the other regulatory reactions described below and spatial diffusion by cell-to-cell signalling, it can already give rise to complex spatiotemporal dynamics in cellular tissue.</p>

<h5>Synthesis by Encoding Genes</h5>
<!-- ........................... -->

<p>We assume here that if a gene \(G_b\) is active, i.e. \(g_b = 1\), the concentration \(p_b\) of the protein \(P_b\) that it encodes simply increases with a constant rate \(\gamma_b\), which is characteristic of the gene:</p>

<p>$$\frac{dp_b}{dt} = \gamma_b g_b$$</p>

<h5>Degradation by the Molecular Environment</h5>
<!-- ........................................ -->

<p>In addition to specific protein-protein interactions and gene-to-protein synthesis, proteins \(P_a\) are also degraded by various molecular elements present in the cell. We model this process by another simple equation, based on a constant degradation coefficient \(\kappa_a\) characteristic of the protein:</p>

<p>$$\frac{dp_a}{dt} = -\kappa_a$$</p>

<p>Up to this point, combining the three laws above, the extended protein equations read:</p>

<p>$$\begin{cases}
       \frac{dp_1}{dt} & = -k' p_1 + \gamma_1 g_1 - \kappa_1 \\[.5em]
       \frac{dp_2}{dt} & = +k' p_1 + \gamma_2 g_2 - \kappa_2
     \end{cases}$$</p>
     
<p>(assuming that they are both synthesized by genes, i.e. \(N_g \geq 2\)). In the sections below, we will add more terms coming from transduction and diffusion mechanisms, which represent the communication with the cell's exterior.

<h5>Cis-Regulatory Boolean Model of Gene Expression</h5>
<!-- ............................................... -->

<p>The activity of a gene \(G_b\) is enhanced by the "presence" of certain promoting transcription factors (TFs, subsumed under the term "protein" here) and/or the "absence" of certain repressing TFs. Both types of TFs bind to the cis-regulatory sites of the gene, which are regions of the DNA near the gene sequence. As multiple TFs \(P_a\) can potentially participate in the regulation of a cis-regulatory module \(G_b\), something we denote by \(P_a \curvearrowright G_b\), Boolean logic is a well-suited schematization of these interactions (<a name='Peter:2012dy' class='ref'></a>). To minimize the number of variables, we represent the potential participation of a TF combined with its effective presence/absence in a regulatory module by a unique matrix of Boolean variables \(\mathbf{\Gamma}=\{\Gamma_{ab}\}\), which is globally indexed by \(a=0...N_p\!-\!1\) and \(b=0...N_g\!-\!1\), and depends on concentration levels \(\mathbf{p}\):</p>

<p>$$\Gamma_{ab}(t) =
	 \begin{cases}
	 	 1 & \text{if}\;\; P_a \curvearrowright G_b \;\;\text{and}\;\; p_a(t) \geq \theta_{ab} \\[.5em]
	   0 & \text{if}\;\; P_a \curvearrowright G_b \;\;\text{and}\;\; p_a(t) \lt \theta_{ab}
	 \end{cases}$$</p>

<p>where the \(\theta_{ab}\) parameters are concentration thresholds. The activity of gene \(G_b\) is then determined by the Boolean output of a logic function \(f_b\), which is a combination of the Boolean operators AND, OR, and NOT:</p>

<p>$$g_b(t) = f_b\left(\mathbf{\Gamma}(\mathbf{p}(t))\right)$$</p>

<p>For example, if \(f_b\) is a pure AND operator, then all promoters must be present and all repressors absent for the gene to be activated. If it is a pure OR operator, then a single promoter suffices to activate \(G_b\).</p>

<h4>Signal Secretion and Transduction Modules</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>Cells in the developing embryo communicate through various means. One of the most common mechanisms are the secretion (typically by exocytosis) and the transduction (via receptors) of various molecules, such as proteins or metabolites, out of their physical domain through the cell membrane. The interfacing module connected to the GRN that internalizes and externalizes these molecules, globally denominated <i>ligands</i>, is presented in this section.</p>

<h5>Signal Secretion Module</h5>
<!-- .......................... -->

<p>Ligands can be externalized from the cellular domain by means of secretion. A gene output that is connected to the <i>signal secretion</i> module in the GRN sends a certain quantity of its synthesized ligand \(P_a\) into the space between cells called "interstitium", creating a concentration \(q_a\) of externalized ligand \(Q_a\) (denoting the same molecule type as \(P_a\), but outside the cell membrane) with a rate coefficient \(\sigma_a\) characteristic of ligand \(a\). This can be represented by the reaction:</p>

<p>$$P_a \overset{\sigma_a}{\rightarrow} Q_a$$</p>

<p>and the rate equations:</p>

<p>$$\frac{dq_a}{dt} = - \frac{dp_a}{dt} = \sigma_a p_a$$</p>

<p>Some externalized ligands diffuse to great distances compared to a typical cell size, while others remain attached to the cell membrane and affect only neighboring cells. Both scenarios are treated in the section below about extracellular dynamics.</p>

<h5>Signal Transduction Module</h5>
<!-- ............................. -->

<p>Conversely, an extracellular signal is transduced into an intracellular protein through a <i>signal transduction</i> module. Three molecular actors are involved here: a receptor protein \(R_{ab}\) on the membrane, the extracellular ligand \(Q_a\) binding to the receptor, and the transduced protein downstream of the receptor \(P_b\), which may or may not be the same molecular type as the ligand, i.e. \(a=b\) or \(a\neq b\) in our indexing of proteins. The transduction module is active simply if its receptor is present in the cell membrane. The receptor and ligand may or may not be consumed during this process: the receptor molecule can either stay in the membrane or disappear (e.g. internalized at the same time), while the ligand molecule can either be recycled or disappear from the interstitium (e.g. internalized or degraded). The following generic reaction summarizes these possible scenarios:</p>

<p>$$Q_a + R_{ab} \overset{\tau_a}{\rightarrow} P_a \;(+ Q_a) \;(+ R_{ab})$$</p>

<p>and the kinetic equations of the various molecular actors are:

<p>$$\begin{cases}
	 	   \frac{dp_a}{dt} & = \tau_a q_a r_{ab} \\[.5em]
	 	   \frac{dq_a}{dt} & = - \epsilon^q_a \frac{dp_a}{dt} \\[.5em]
	 	   \frac{dr_{ab}}{dt} & = - \epsilon^r_{ab} \frac{dp_a}{dt}
	   \end{cases}$$</p>

<p>where \(r_{ab}\) is the concentration of receptor in the membrane, and \(\epsilon^q_a\) and \(\epsilon^r_{ab}\) are binary parameters equal to 1 if the ligand, resp. receptor, is consumed during the transduction process, or 0 if not.</p>

<h4>Extra-Cellular Reactions, Transport and Diffusion</h4>
<!-- ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->

<p>Various detailed models of the spatial configuration of the intersitium have been elaborated (e.g. <a name='Kojic:2010jo' class='ref'></a>), but we prefer using here the abstract graph of neighborhood relationships (derived from a Delaunay triangulation), which was described in section 3.2.2, to serve as the infrastructure of ligand diffusion. Ligands diffuse in the interstitial regions between cells, delimited by their membranes, and the neighborhood graph connecting the centers of the cells is the "dual" representation of this space. This network also offers a spatial representation of the embryo that is robust with respect to the deformation of the multicellular assembly.</p>

<p>The macroscopic dynamic describing the diffusion of molecules is based on Fick's law. It states that the ligands move from regions of high concentration to regions of low concentration with an amplitude proportional to the spatial gradient of the concentration. The flux \(\vec{J}_a\) measuring the quantity of extracellular ligand \(Q_a\) that passes through a small section during a small time interval is given by:</p>

<p>$$\vec{J}_a = - D_a \vec{\nabla}q_a $$</p>

<p>where \(D_a\) is the diffusion coefficient of the ligand and \(q_a\) its concentration.</p>


<p>
	The diffusion occurs between neighbor cells, we consider that each ligand quantity \(q_a\) flows between neighbor cell centers. We denote by \(q_a(i)\) the concentration of the ligand \(Q_a\) localized on the surface of the cell \(i\), and ideally localized at the cell center position in our approximation. We denote by \(\vec{J}_{a}(i,j)\) the linear approximation of the flux of ligand \(Q_a\) between two neighbor cells \(i\) and \(j\). 
</p>

<p>
	$$\vec{J}_{a}(i,j) = - D_a  \frac{q_{a}(j) - q_{a}(i)}{r_{ij}} \vec{u}_{ij}$$
</p>

<p>
	where \(\vec{u}_{ij}\) is the unit vector from \(i\) to \(j\).
</p>

<p>
	Faut-il mentionner que \(\vec{J}_{a}(i,j) = \vec{J}_{a}(j,i)\) ??? (on inverse Ã  la fois le signe du delta et de \(u_{ij}\))
</p>







<figure id='model_grns_diffusion_schematic'>
	<img src="../../images/model_grns/diffuse_volume_augmented_crop.png" width="800">
	<figcaption>xx</figcaption>
</figure>

<p>The temporal evolution of the concentration is determined by the <i>continuity equation</i>, which is a local form of conservation law. The divergence theorem gives the integral form of the continuity equation, applied on the volume of the cell. Its continuous expression reads:</p>

<p>$$\frac{\partial q_a}{\partial t} +  \iint_O \vec{J_a}.\vec{dA} = \Upsilon_a + \Psi_a $$</p>

<p>
	where \(dA\) is the surface normal vector of the closed surface of the cell, \(\Upsilon_a\) (resp. \(\Psi_a\)) is a sink (resp. source) term <strike>whose definition is given in the previous section</strike> which corresponds to the quantity of ligands \(Q_a\) transduced (resp. secreted) by the cells.
</p>

<p>
	$$ \Upsilon_a = - \epsilon^q_a \frac{dp_a}{dt} $$
</p>

<p>
	$$ \Psi_a = \sigma_a p_a$$
</p>


<p>
	The local approximation of the closed surface integral is expressed with the topological neighbor list introduced in section XXXXX.
</p>

<p>
	$$ \left ( \frac{\partial q_a(i)}{\partial t} \right ) \Big |_{t=t_0} \approx  - \sum_{j \in \mathcal{N}^t_a} \left( A_{i,j} \left \| \vec{J}_{a}(i,j) \right \| \right)\Big |_{t=t_0} + \Upsilon_a(i)\Big |_{t=t_0} + \Psi_a(i) \Big |_{t=t_0}$$
</p>





</div>

