<div id='5_'>
<h1>5. MECAGEN Model of Mechanic-Genetic Coupling </h1>
 
<p>
	In this section, we present two mode of interaction...
</p>

<ul>
	<li>an ideal coupling mode, the proper MECAGEN model of coupling, which relates the biomechanical model introduced in Chapter 3 and the genetic and molecular regulation model introduced in Chapter 4. This coupling is what has driven the design choices behind the mechanical and genetical model. However, the principles behind the coupling will only be introduced here as, in its current state, the model has not been tested already.</li>
	<li>a simplified model of cell behavior specification, inspired by Waddington's epigenetic landscape, which bypasses the molecular and genetic regulation and allows us to test the mechanical hypotheses introduced in Chapter 3.</li>
</ul>
</div>

 	
 
 
<div id='5_1_'>
<h2>5.1. Toward a Cell Behavior Ontology </h2>
 

<h4>MECAGEN coupling</h4>

<p>
	Biological system modeling requires the choice of an ontology. In our domain, the adapted ontology is a categorization of the various cell behaviors occurring during a developmental process.

	Among a number of initiatives, we mention the current work of <a href="http://bioportal.bioontology.org/ontologies/49581?p=terms">XXXX</a>, which performs an in-depth classification of heterogeneous objects and processes involved in multi-cellular systems.

	Ontologies are funded by hierarchical categories reflecting the vision of the ontology designer. 

	In MECAGEN, we propose to introduce a simple ontology based on the dynamical rules introduced in Chapter 3 and Chapter 4.

	The objective of this ontology is establish a mapping between the genetic and molecular state of the cell and its mechanical behavior.

	Ideally, all mechanical parameters must be defined by the molecular state of the cell. The spatio-temporal specification of the mechanical behavior of each cell is ruled by the dynamical outputs of its molecular and genetic regulation. 

	In practical terms, a predefined set of intracellular concentration levels \(p_a\) is monitored, and determines the value taken by the biomechanical parameters:
</p>

<ul>
	<li>\(w_{\mathrm{adh}}\), which modulates the attractive part of the <i>relaxation</i> force \(\vec{F}^P_{ij}\)</li>
	<li>a boolean determining with the protrusive force \(\vec{F}^A_{ij}\) is operating</li>
	<li>polarization axis (axes) \(\vec{u}_i\)</li>
</ul>

<p>
	Some cellular mechanisms are decoupled from the intracellular molecular and genetic regulation in MECAGEN:
</p>

<ul>
	<li>cell cycle length</li>
	<li>cell volume control</li>
</ul>

<p>
	Other mechanisms are not considered:
</p>

<ul>
	<li>cell death</li>
	<li>extracellular matrix</li>
</ul>



<h5>Cell Behavior Ontology</h5>



	<p>
		The major dichotomy in our cell behavior ontology distinguishes the mesenchymal cell behaviors and the epithelial cell behaviors.
	</p>

	<p>
		Mesenchymal cell behavior is characterized by its motility. We identify one &quot;active&quot; behavior for mesenchymal cells: protrusion. By protruding, mesenchymal cell are able to rearrange their neighborhood and/or migrate through it. 
	</p>

	adhesion modulation
	
	polarization
	
	active behavior: protrusion (mono/bipolar)


	<p>
		Epithelial cell behavior is characterize to a reinforcement of lateral cellular junction. In addition, we identify two potential &quot;active&quot; behavior for the epithelial cells: (a)  intercalation in the sheet orthogonal to the apical-basal axis, and (b) apical constriction which allows a bending of the epithelial sheet (invagination).
	</p>


	apico-basal adhesion
	lateral adhesion
	specification of an apical-basal polarization axis
	reinforced lateral adhesion
	active behavior1: apico-basal contraction (torquing )
	active behavior2: active intercalation in the lateral plane (only, require an additional axis).

</p>

	

<h5>Cell adhesion</h5>

<p>
	Cell adhesion is the mechanical phenomenon that is the easiest to relate to an output of the molecular and genetic regulation. The intensity of the adhesion between two neighbor cells is a function of the surface densities of adhesion molecules. Assuming that the adhesive molecules are uniformly distributed on the cell membrane and that the intracellular protein \(P_a\) is the adhesive molecule in the following, we define, at the interface between two neighbor cells \(i\) and \(j\), the surface concentration \(c_{a,i,j}\) (resp. \(c_{a,j,i}\)) of \(P_a\) on the membrane of \(i\) (resp. \(j\)):
</p>

<p>
	$$ c_{a,i,j} = \frac{p_{a,i} A_{i,j}}{4 \pi {R_i}^2} $$
</p>

<p>
	$$ c_{a,j,i} = \frac{p_{a,j} A_{i,j}}{4 \pi {R_j}^2} $$
</p>

<p>
	where \(p_{a,i}\) (resp. \(p_{a,j}\)) is the internal concentration of protein \(P_a\) in cell \(i\) (resp. \(j\)), A_{i,j} is the surface of contact between \(i\) and \(j\) introduced in Chapter 3 and \(R_i\) (resp. \(R_j\)) is the radius of cell \(i\) (resp. \(j\)).
</p>

<p>
	We also assume that adhesive molecule bind individually at the cell-cell interface <a name='Boggon:2002da' class='ref'></a> (other modes of binding are explored in <a name='Zhang:2011ca' class='ref'></a>), the equation of the adhesion coefficient \(w_{\mathrm{adh}, a, i, j}\) of the &quot;relaxation&quot; force \(\vec{F_{ij}^P}\) induced by the adhesion molecule \(P_a\) reads:
</p>

<p>
	$$ w_{\mathrm{adh}, a} = k_{\mathrm{adh},a} c_{a,i,j} c_{a,j,i} $$
</p>

<p>
	where \(k_{\mathrm{adh},a}\) is an adhesion constant proper to \(a\).
</p>

<p>
	Multiple adhesive molecules type may be involved in between two neighbor cells, we assume that each molecule type exerts homotypic adhesion, i.e. they bind only with their own kind. For this reason, if multiple cell adhesion molecules \(P_a\) with \(a=0...N_{\mathrm{adh}}\) are involved at a cell-cell interface, we sum each molecule type contribution:
</p>

<p>
	$$ w_{\mathrm{adh}} = \sum_{a = 0}^{N_{\mathrm{adh}}} w_{\mathrm{adh},a} $$
</p>

<h5>Cell polarization</h5>

<p>
	Every &quot;active&quot; cell behaviors exploited in MECAGEN require a polarization axis. In real cells, polarization translates into an asymmetry of the intracellular molecules. However in MECAGEN, there is no spatialization of intracellular material. We represent this asymmetry by a 3D vector passing through the center of the cell. Moreover, as the developmental process unfolds, a MECAGEN cell is potentially polarized through different polarization mechanisms. For a generic cell \(i\), we denote these \(N_d\) potential polarization axes \(\vec{u}_{d,i}\) with \(d=0...N_d\!-\!1\) and introduced an additional boolean variable \(\mathbf{\Delta} = \Delta_d\), which is 1 (resp. 0) if the \(\textrm{d}^{\textrm{th}}\) polarization mechanism is active (resp. not). Ideally, mesenchymal have at most one non-null \(\Delta_d\) whether a mesenchymal active behavior is acting, and an epithelial cell have at least one non-null \(\Delta_d\) (for the calculation of the apical-basal axis) and at most two non-null \(\Delta_d\) whether an epithelial active behavior is acting. We introduce here three different modes of determination of the axis of polarization: (a) a local gradient-based mode, (b) a cell-cell contact propagation mode, and (c) a force induced mode. Additionally, a default mode (d) is activated if a polarized cell lacks any inputs for the three aforementioned mechanisms. The default mode stochastically re-orients the polarization axis until another polarization mode take over.
</p>

il manque la règle qui détermine la valeur des Delta_d

<ul>
	<li>(a) local gradient-based mode. This mode expresses a classical vision of polarization, e.g. often used in chemotactic cell behaviors, and supposes that the cell is able to detect an asymmetry of the extracellular ligand concentration in its local vicinity. In MECAGEN, this asymmetry is determined using the abstract graph of neighborhood relationships (Section 3.2.2) and the extracellular ligand quantities \(q_a\) (Section 4.2.1). This mode of axis specification is associated with a ligand type \(Q_a\) and define the axis polarization (\(\vec{u}_i = \vec{u}_{a,i}\)) as the average of the neighborhood links weighted by a polynomial function of the difference of local ligand quantities:</li>
</ul>

<p>
	$$ \vec{u}_{d,i} = \sum_{j \in \mathcal{N}^t_i} (q_{a,j} - q_{a,i})^n \vec{u}_{ij} $$, à normaliser...
</p>

<p>
	where \(\mathcal{N}^{t}_i\) is the list of topological neighbors of cell \(i\), \(q_{a,i}\) (resp. \(q_{a,j}\)) is the local quantity of extracellular ligand \(Q_a\) around \(i\) (resp. \(j\)), \(n\) is a integer controlling the sensibility of the detection of the local asymmetry (\(n\) have to be an odd number, and typically is equal to 3), and \(\vec{u}_{ij}\) the unit vector pointing from \(i\) to \(j\).
</p>


<ul>
	<li>
		(b) cell-cell contact propagation mode. This mode expresses another way of propagating 	polarization axes in a cellular field. It is based on the notion that the asymmetric spatialization of intracellular material in a polarized cell translates into a surfacic asymmetry that neighbor cells can sense. Only polarized neighbors (\(\Delta_d = 1\)) influence their vicinity. Thus, the cell-cell contact propagation mode define the polarization axis of cell \(i\) has the average of its polarized neighbors' polarization axes.
	</li>
</ul>

<p>
	$$ \vec{u}_{d,i} = \sum_{j \in \mathcal{N}^t_i} \Delta_{d,i} \vec{u}_{d,j} $$
</p>

<p>
	This mode is systematically used for the determination of the apical-basal axis in an epithelial sheet of cells. Only the lateral neighbors belonging to the topological list of neighbors \(\mathcal{N}^t_i\) are considered in the sum (i.e. neighbor whose normalized neighboring link and apical-basal polarization axis forms a absolute scalar product less .25).
</p>


<ul>
	<li>
		(c) force-induced mode. Recent studies unveil a novel mechanisms for the propagation of polarization axis based on the mechanical interaction <a name='Weber:2011hi' class='ref'></a> <a name='Dumortier:2012jk' class='ref'></a>. Weber et al. <a name='Weber:2011hi' class='ref'></a> demonstrates that a mechanical traction exerted at one end of a <i>Xenopus</i> cell triggers the formation of protrusion in the opposite direction through a reorganization of internal keratin intermediate filaments. In MECAGEN, we idealize this mechanism by orienting the polarization axis in the direction opposite to the averaged force exerted by neighbor protruding cells.
	</li>
</ul>

<p>
	$$ \vec{u}_{d,i} = - \frac{1}{N_{\textrm{protrusion}}} \sum_{j \in \mathcal{N}^t_i} \vec{F}^A_{ji}  $$
</p>

<p>
	where \(\vec{F}^A_{ji}\) is the &quot;active&quot; force exerted by the neighbor \(j\) on \(i\), and \(N_{\textrm{protrusion}}\) is the number of neighbor cells exerting protrusion on j.
</p>


Default polarization mode: random orientation


Update inertia 




<h5>Mechanotransduction input of the GRN</h5>


<p>
	Recent studies have shown that some genes are upregulated by mechanical forces exerted on the cell <a name='Desprat:2008ei' class='ref'></a>. This mechanism is among the funding principles in MECAGEN, allowing a real coupling of the mechanical domain to the molecular and genetic regulation. Practically, we add a new module to our molecular and genetic simulation platform. If the module is active, i.e. the sum of the forces exerted on the cell are above a given threshold, this module increases its target protein \(P_a\) concentration by a simple equation, based on a constant production coefficient \(\xi_a\) characteristic of the module. 
</p>

<p>$$\frac{dp_a}{dt} = \xi_a$$</p>



Cell protrusion


a. monopolaire ou bi_polaire (chapitre 3)
b. intensité de la protrusion (chapitre 3)
c. protrusion target, function of cell types (chapitre 5)
d. axe de polarization


protrusion -> module ((rho rac gtpase))
a set of molecule whose state determine if the cell will perform protrusive activity
protrusion -> axis, monopolar, bipolar, substrate

We expect a protruding cell to be active if it orients its protrusion(s) along 

protrusion -> axis, monopolar, bipolar, substrate



polarization -> eventuellement relier au module de protrusion

necessite une spatialization au niveau subcellulaire, assymetrie cytoplasmique.

par defaut random...


mode ligand
mode average
mode mechanotransduction
both

update pas instantané...



epithelial behaviors:

apico-basal specification

stronger lateral adhesion

additional mechanisms: apical-constriction
 
</div>
 
 
 
 
 
<div id='5_2_'>
<h2>5.2. Waddingtonian Timeline Specification  </h2>

Expliquer pourquoi on n'utilise pas le model complet dans la suite

<p>
	Even if, ultimately in MECAGEN, the real causes (causing factors/inputs of the model) behind the embryo development are the molecular and genetic parameters specified by a BioTapestry GRN, we also aim at studying development through the spectrum of cell forces alone. As Odell et al. mentioned in <a name='{Odell:1981vy}' class='fig'></a>, cell forces, equivalently to &quot;clocks, morphogens, or potentials&quot;,  are the &quot;cause&quot; which &quot;explains&quot; the &quot;morphogenetic motion&quot;. For this goal, we decide to uncouple to coupled MECAGEN model and bypass the molecular and genetic interaction. The cell behaviors will be attributed not on the ground of the GRN outputs, but rather as the direct reading a novel parameter specification tool, called <i>Waddingtonian landscape specification</i>. This tool is inspired by the famous <i>epigenetic landscape</i> introduced by Waddington in &quot;The Strategy of the Genes&quot; <a name='Waddington:1957te' class='ref'></a>. The epigenetic landscape views the development as a dynamical system in which the cells evolve through time towards attractors represented as foldings (he called &quot;creodes&quot;) on a hillside (Fig. <a name='Development_Review_waddington_epigenetic_nature_mod_small' class='fig'></a>). The cell fate, symbolized by a ball rolling down the hillside, falls into on or another attractor under the influence of inducing signals. 
</p>

<figure id='Development_Review_waddington_epigenetic_nature_mod_small'>
	<img src="../../images/Development_Review/waddington_epigenetic_nature_mod_small.png" width="800">
	<figcaption><strong>Waddington's epigenetic landscape.</strong> A: The ball represents a cell evolving in the epigenetic landscape. Its fate is determined by the canals in which the ball is rolling. B: A view of the landscape's behind the scene. The landscape relief is dynamically controlled by hidden wires which symbolized the gene expression and interactions. Image and legend adapted from Slack <a name='Slack:2002kg' class='ref'></a></figcaption>
</figure>


<p>
	In the following, we present how are specified the parameters of a Waddingtonian timeline. A specific graphical interface has been developed in this purpose.
</p>


<h5>Cell types</h5>

<p>
	The first step of a Waddingtonian timeline specification is to &quot;carve the hillside&quot; by attributing <i>cell types</i> that the cell can potentially adopt during the developmental process. We define <i>stages</i> which are the time point from which a bifurcation of cell type is possible. Then, we specify the conditions which will induce a cell to change its type. We call this process <i>differentiation</i>. The differentiation are simply expressed by a position information-like mechanism: if a specified ligand has a local value exceeding a given threshold, the cell switch its type. The cell types are represented as a tree which expands itself through time (Fig. <a name='waddingtonian_timeline_timeline' class='fig'></a>).
</p>

<figure id='waddingtonian_timeline_timeline'>
	<img src="../../images/MECAGEN/waddingtonian_timeline/timeline.png" width="800">
	<figcaption><strong>Waddingtonian timeline.</strong> The grey bar on top is the time axis, starting from the left to the right. The thin white bar is the current time step of a simulation. Thick red bars symbolize buckets from which the whole state of a simulation is recorded as the white time bar passes through it. It is a convenient way of specifying simulation snapshots. The colored bifurcating branches are the cell type that the cells can potentially take. The dots are the stages from which the simulation parameters are specified. A stage comprise all the vertical dots at a given time step, thus we can observe 5 stages in the timeline.</figcaption>
</figure>

<h5>Ligands sinks and sources</h5>

<p>
	Once the backbone of the waddingtonian timeline has been designed, the diffusing ligands must be specified. At each stages, we define sources and sinks for each ligands. Each cell types can secrete or absorb any ligands (Fig. <a name='waddingtonian_timeline_sources_sinks' class='fig'></a>). Additionally, spatial specification of the ligands sources are possible. External sources of ligands may also be added (e.g. in Chapter 8, the yolk egg is a source of ligand).
</p>


<figure id='waddingtonian_timeline_sources_sinks'>
	<img src="../../images/MECAGEN/waddingtonian_timeline/sources_sinks.png" width="700">
	<figcaption><strong>Ligands cells' sources and sinks specification.</strong> Left: </figcaption>
</figure>

<h5>Passive force's adhesion modulation</h5>

<p>
	The adhesion coefficient \(w_{\mathrm{adh}}\) of the relaxation force \(\vec{F}^P\) is specified for each potential cell type's interaction (Fig. XX).
</p>

<figure id='waddingtonian_timeline_adhesion'>
	<img src="../../images/MECAGEN/waddingtonian_timeline/adhesion.png" width="400">
	<figcaption><strong>Adhesion coefficient in the Waddingtonian timeline gui.</strong> The matrix must be symmetrical so only half of the coefficients are displayed. Cell types' color code is the same as Fig. <a name='waddingtonian_timeline_timeline' class='fig'></a>.</figcaption>
</figure>



<h5>Active cell behavior specification</h5>

<p>
	Once the ligand sources and sinks are defined, the active cell behavior are specified by adding behavioral module for a given cell type at a given stage. A protrusion module (Fig. <a name='waddingtonian_timeline_protrusion' class='fig'></a>) is associated to a cell type and gets four parameters: the target cell type on which the protrusion takes effect, the id of the axes of protrusion (axes of protrusion are calculated by the ligand mode (a) defined in Section 5.1), the intensity of the protrusive force, and an boolean of value 0 (resp. 1) if the cell is monopolar (resp. bipolar).
</p>


<figure id='waddingtonian_timeline_protrusion'>
	<img src="../../images/MECAGEN/waddingtonian_timeline/protrusion2.png" width="400">
	<figcaption><strong>Protrusion module in the Waddingtonian timeline specification.</strong> Three cell types are involved here: red, green and blue. In each cell type's rounded rectangle, the protrusion module is represented by four rectangles with: top left, the target cell type, top right, the polarization axis id, bottom left, the intensity of the protruding force, and bottom right, the type of protrusion, either monopolar or bipolar.</figcaption>
</figure>

<p>
	The Waddingtonian timeline is a novel, yet limited, method of specifying the cell behaviors through space and time. It allows a partial exploitation of the principles involved in MECAGEN but it is sufficient to explore the mechanical space, in particular, it will be used in the specific study of the Zebrafish mechanical development developed in the second part of the manuscript.
</p>

</div>
 
 
</div>
 
 
 
<div id='5_3_'>
<h2>5.3. Illustration on Artificial Cell Sorting</h2>
 


	

<h4>Cell sorting: short revisit of a classical problem</h4>

see <a name='Zhang:2011ca' class='ref'></a>




<h5>DAH, strong homotypic adhesion, weak heterotypic adhesion, random protrusion axis</h5>


<figure id='Case_theo_sorting_nosource_allmigr_DAH'>
	<video src="../../videos/MECAGEN/Case_theo/sorting_nosource_allmigr_DAH.ogg" width="800" controls></video>
	<figcaption></figcaption>
</figure>



<h5>Oriented protrusion exterior morphogenetic field, source left right</h5>


<figure id='Case_theo_sorting_sourceext_extmigr'>
	<video src="../../videos/MECAGEN/Case_theo/sorting_sourceext_extmigr.ogg" width="800" controls></video>
	<figcaption></figcaption>
</figure>



<h5>Oriented protrusion exterior morphogenetic field, source center</h5>



<figure id='Case_theo_sorting_sourcecenter_extmigr'>
	<video src="../../videos/MECAGEN/Case_theo/sorting_sourcecenter_extmigr.ogg" width="800" controls></video>
	<figcaption></figcaption>
</figure>


 <h4>Single cell migration</h4>

 <figure id='Case_theo_migr_raz_simu'>
	<video src="../../videos/MECAGEN/migrRaz.ogg" width="800" controls></video>
	<figcaption></figcaption>
</figure>

 <figure id='Case_theo_migr_raz_live'>
	<video src="../../videos/video_modeling/migrationRaz/sdf.ogg" width="800" controls></video>
	<figcaption></figcaption>
</figure>
 
</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
