
 
 
 
 
<div id='7_'>
<h1>7. Experimental Reconstruction of the Zebrafish Early Development </h1>
 
 
</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_1_'>
<h2>7.1. Bioemergences Reconstruction Workflow  </h2>
 
 
	

<figure id='bioemergences_c_elegans_lineage_kipreos_2005'>
	<img src="../../images/Reconstruction/bioemergences/c_elegans_lineage_kipreos_2005.png" width="900">
	<figcaption>Cell lineage from the <i>Caenorhabditis elegans</i> embryonics and larval stages from Kipreos, E.T., 2005. C. elegans cell cycles: invariance and stem cell divisions. Nature reviews Molecular cell biology, 6(10), pp.766–776. <a name='Kipreos:2005im' class='ref'></a></figcaption>
</figure>



<p>
	<strong>Objective</strong>: Reconstruction of the complete cell lineage tree of the Zebrafish early development. The lineage tree is augmented by the spatial coordinates and the shape of the cells (3D+time digital embryo) and by the quantification of various gene expression of the cells (Atlas of genetic expression).
</p>


<p>
	The reconstruction task is a very tedious one and semi or complete automation is the only solution to tackle such an enormous lineage tree.
</p>

<p>
	In this section, we will introduce the past and current strategies originated by the European project Embryomics and Bioemergences and pursued the Bioemergences team. This section is divided as the following:
</p>

<ul>
	<li>1. Embryo Preparation and Acquisition</li>
	<li>2. Cell lineage reconstruction workflow</li>
	<ul>
		<li>1. Workflow</li>
		<li>2. Visualization / Correction</li>
		<li>3. Validation</li>
	</ul>
	<li>3. Post-processing / Higher level reconsruction / Exploitation</li>
	<li>4. 4D atlas of gene expression</li>
	<li>5. <i>In toto</i> cell lineage</li>
</ul>



Discussion
->Prototype


<p>
	<strong>Various challenges</strong>: in addition to the challenges to design the aforementioned reconstruction strategy, various challenges arises from the limitation of the current imaging setups. 
</p>



<p>
	Workflow inputs: raw data
</p>


<ul>
	<li>The workflow is efficient if the spatio-temporal resolution of the raw data produced by the microscope device satisfies some required signal properties.</li>
	<ul>
		<li>spatially, the signal-to-noise ratio must be limited and the local resolution of the 
		<li>temporally, the difference between two images of the same region must be small enough as the image must not move too much.</li>
	</ul>
</ul>

<p>
	Unfortunately, these two requirements are antagonists and a trade-off must be found
</p>




<p>
	Solution: 
</p>


<p>
	Most of the improvement must be applied in the imaging technique area but, given their limitations, algorithmic bandages must be designed to overcome them. 
</p>

<p>
	We distinguish between the fundamental reconstruction workflow which, in its idealized form, will remain relevant for a long time (pas terrible XXXX) and the modules which are subjugated to the limits of microscope performance (signal and scale of the raw data) and will become obsolete as soon as the microscope improves.
</p>



<i>in toto</i>

error-less: automated vs manual (introduire application thierry),

channel limitation



The prototype




















<figure id='bioemergences_FigureWorkflow_Workflow2'>
	<img src="../../images/Reconstruction/bioemergences/FigureWorkflow/Workflow2.png" width="600">
	<figcaption>Workflow of reconstruction of the cell lineage</figcaption>
</figure>







<p>
	review of embryogenesis reconstruction :
</p>


<p>
	Keller, P.J. et al., 2008. Reconstruction of zebrafish early embryonic development by scanned light sheet microscopy. Science, 322(5904), pp.1065–1069. <a name='Keller:2008km' class='ref'></a>
</p>


<p>
	Giurumescu, C.A. et al., 2012. Quantitative semi-automated analysis of morphogenesis with single-cell resolution in complex embryos. Development, 139(22), pp.4271–4279. <a name='Giurumescu:2012ej' class='ref'></a>
</p>




<p>
	example reconstruction:
</p>


<p>
	Xiong, Y. amp;glesias, P.A., 2010. Tools for analyzing cell shape changes during chemotaxis. Integrative Biology, 2(11-12), pp.561–567. <a name='Xiong:2010ck' class='ref'></a>
</p>





<p>
	review -> Khairy, K. amp;eller, P.J., 2011. Reconstructing embryonic development P. M. Kulesa, M. E. Dickinson, amp;.-K. Hadjantonakis, eds. genesis, 49(7), pp.488–513. <a name='Khairy:2011ht' class='ref'></a>
</p>



</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_1_1_'>
<h3>7.1.1. Embryo Preparation and Acquisition  </h3>
 
 


<p>
	Embryo Preparation from paper methodo, à modifier:
</p>


<p>
	Wild-type zebrafish embryos were injected at the one cell stage with 200pg mCherry/H2B RNA and 200pg eGFP-ras prepared from PCS2+ constructs [Moriyoshi, 1996 #35;Shaner, 2005 #29]. Although mCherry, unlike eGFP, bleached significantly through imaging, this colour combination was the best compromise allowing proper staining of the cell membranes for further segmentation. Injected embryos were raised at 28.5°C for the next 3 hours. Embryos were mounted in a 3cm Petri dish with a glass coverslip bottom, sealing a hole of 0.5mm at the Petri dish centre where a Teflon tore (ALPHAnov) with a hole of 780µm received the dechorionated embryo. The embryo was maintained and properly oriented by infiltrating around it 0.5% low melting point agarose (Sigma) in embryo medium [Westerfield, 2000 #36]. Temperature control in the room resulted in a temperature of about 26°C under the objective slightly slowing down development with respect to the standard 28.5°C developmental table [Kimmel, 1995 #11] (Table S1). After the imaging procedure, embryo morphology was checked under the dissecting binocular and the animal was raised for at least 24h to assess morphological defects. Embryo survival depends on total imaging duration, average laser power, and image acquisition frequency (or time step ∆t). 070418a and 070420a correspond to a standardised procedure with 50s < ∆t < 70s allowing 75-110 sections per time point for 10 hours with an average laser power of 80 mW delivered to the sample. Lowering the laser power to less than 60 mW and lengthening ∆t up to 3.5min allowed imaging embryos for more than 20 hours, then raising them until adulthood. These conditions were used for making up to 320 sections at 400Hz line scan rate (bidirectional scanning), or 200Hz to improve signal-to-noise ratio, as for the image data set 080322a shown in movie 1-3. 
</p>






<p>
	Image acquisition from paper methodo, à modifier:
</p>

<p>
	Imaging was achieved with a Leica upright microscopes SP5 MLSM equipped with a 20/0.95NA W dipping lens objective (Olympus) or Leica 20/1NA W dipping lens objective. Axial resolution at the sample surface (1.5µm) was estimated by recording 3D images of 0.1 or 1µm fluorescent polystyrene beads (Invitrogen) at the surface of an agarose gel. Field size was 700x700 or 775x775 microns in x, y, 140µm in z for 070418a and 100µm in z for 070420a; voxel size was 1.37x1.37x1.37µm or 1.51x1.51x1.51µm. Simultaneous dual wavelength excitation with pulses at two different wavelengths (980 and 1030nm). At 1030 nm, pulsed laser beam (50Mhz, 200fs) was provided by a solid-state Ytterbium femtosecond oscillator (T-pulse 20, Amplitude Systèmes). At 980 nm, pulsed laser beam (80Mhz, 100fs) was provided by a Ti-Sapphire femtosecond oscillator (Mai Tai HP, Newport Spectra physics). Details of the optical bench are provided in supplementary material. Raw data visualisation was done using Amira software (Mercury Computer Systems). 
</p>



</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_1_2_'>
<h3>7.1.2. Cell Lineage Reconstruction Workflow  </h3>
 
 




<h4>Workflow</h4>


<figure id='bioemergences_FigureWorkflow_Workflow2_lineage_only'>
	<img src="../../images/Reconstruction/bioemergences/FigureWorkflow/Workflow2_lineage_only.png" width="800">
	<figcaption></figcaption>
</figure>






<h5>Filtering</h5>


<p>
	Raw data images are filtered with specific nonlinear geometrical PDEs called &quot;Geodesic Mean Curvature Flow&quot; (GMCF). In GMCF, the contrast-invariant flows achieved nonlinear multiscale analysis on scalar intensity functions in the voxel intensity histogram <a name='Rizzi:2007bz' class='ref'></a> <a name='Kriva:2010cs' class='ref'></a>.. This iterative strategy eliminate most of the noisy signal.
</p>

 

<h5>Cell identification</h5>

<p>
	Once, the nuclei and membrane images have been filtered, the nuclear centers are detected by a nonlinear PDE-based multiscale strategy called &quot;Flux-Based Level Set Center Detection&quot; (FBLSCD). On the nuclei image, the FBSLSCD iteratively decreases the size of the nuclei signal until a given stopping condition is reached. The last remaining local maxima gives the position of the cell centers <a name='Frolkovic:2007ti' class='ref'></a>.
</p>


<h5>Shape Segmentation</h5>


<figure id='segmentation_images_segmentation'>
	<img src="../../images/Reconstruction/bioemergences/segmentation/images_segmentation.png" width="800">
	<figcaption>Top. Left. A triangle with subjective contour <a name='Sarti:2000jw' class='ref'></a>. Right. Evolution of the subjective surface technique towards the contour <a name='Sarti:2000jw' class='ref'></a>. Bottom. Segmentation of membrane with an uncomplete contour <a name='Zanella:2007vh' class='ref'></a>.</figcaption>
</figure>



<p>
	The proper segmentation of both the surface of the cell membranes and cell nuclei is then performed from the position of the cell centers. The algorithm used is called the &quot;Subjective Surface Technique&quot; (SST) <a name='Sarti:2000jw' class='ref'></a>. For each cell, the segmentation is performed from the point of view of the center. The surface grows around it until it reaches an obstacle, the surface then deforms to adopt the shape of the surrounding signal. The key advantage of this technique is its ability to deal with the incomplete information of the image and still propose a hole-less segmentation. 
</p>

<p>
	The segmentation of the nuclei and membrane are used to automatically correct some errors in the center detection step. If two centers are detected but the segmentation provide the same subjective surface, the duplicated center. Similarly, segmentation are used to detect mitosis if two mitosis are inducing the same membrane. The information of a probable cell duplication is used by the following cell tracking algorithm.
</p>


<p>reference recente: Mosaliganti, K.R. et al., 2012. ACME: Automated Cell Morphology Extractor for Comprehensive Reconstruction of Cell Membranes R. F. Murphy, ed. PLoS Computational Biology, 8(12), p.e1002780. <a name='Mosaliganti:2012gj' class='ref'></a></p>


<h5>Cell tracking</h5>


<p>
	The cell tracking algorithm is the key module of this workflow. It relates temporally the identity of the cell in between time steps. From the initial cell population, these temporal links form a tree whose branches split when a cell divides. The tracking is build in three steps:
</p>

<ul>
	<li>an <i>a priori</i> step which specifies the strong local constraints that restricts the list of putative links. The constraints are mainly the suppression of all putative links which relates cell which would have move above a given maximum distance and the number of daughter links.</li>
	<li>a central step which consists in minimizing a heuristic function which combines the information extracted from the processed images (center positions, large scale vector fields, segmentation) with a biophysical regularization constraint (see Kirkpatrick 1983 XXXX).</li>
	<li>an <i>a posteriori</i> step which processes the obtained tree to minimize the number of false-positive and false-negative errors detected in the segmentation module.</li>
</ul>






<h4>Visualization / Correction</h4>



<p>
	Moveit
</p>



<p>
	A major challenge for the reconstruction of developing embryos is the design of tools adapted the handling of the produced data sets.
</p>

<p>
	The specificity of these new kind of data (a mapping between a graph = lineage tree and various type of local cell data = morphological, positional, genetic expression...) make them mostly incompatible with existing data-handling software solution.
</p>


new type of phenomological 

a new kind of digital specimen

original

augmented
reconstructed phenomenology







dedicated tool for visualization and correction/validation of the reconstruction of developing embryos.





Necessité d'un outil qui répond à une double demande


Data visualization:

	exhaustive visualization: from the raw data to the (cell centers + cell nuclear and membrane segmentation)  + and every intermediate data used in the workflow ?
	
	multiscale: from the fields of microscopic data and the focus on local data

	
	direct biological insight extraction: 

		
		fate mapping : avant non seulement flou, lien continu entre repere et injection

		cell fate given by the lieneage

		evolution of the cell proliferation rate

		clonal population selection

		origin polyclonal des organes: 

		exemple hors bioemergences -> lineage exploitation for stem cell studies (Simons, B.D. &amp; Clevers, H., 2011. Strategies for Homeostatic Stem Cell Self-Renewal in Adult Tissues. Cell, 145(6), pp.851–862. <a name='Simons:2011ec' class='ref'></a>)

from Lombardot, B., 2010. Description intégrative des cinématiques de la structure embryonnaire : morphogenèse précoce du poisson zébré. Ecole Polytechnique EDX Paris.

"Inter^et de l'etude des destinees : L'etude des destinees est tres informative au premier
abord sur la morphogenese embryonnaire. Les cartes de destinees et les lignages
cellulaires indiquent ou vont les cellules et comment (dans le cas du lignage). Cependant
ca n'est pas le seul inter^et de l'etude des destinees. Comme nous venons de le voir les
destinees dans l'embryon peuvent dependre du lignage mais aussi d'une regulation de l'environnement.
Les cartes de destinees ou les lignages ne donnent pas d'information directe
sur la regulation, par contre, l'etude du lignage, les analyses clonales et leur association a
des informations complementaires sur le comportement des cellules, leur activite biochimique
et leur mode de division peuvent ^etre indicatifs d'une restriction du potentiel d'une
cellule ou d'une population de cellules. L'analyse clonale (ie. l'analyse des descendants
d'une cellule) permet de s'interroger sur le lignage qui permet la transformation observ
ee : nombre de divisions, nombre de cycles cellulaires, distribution des types cellulaires,
organisation spatiale des descendants [38]. L'etude de ces grandeurs permet de s'interroger
sur l'activite specique associee a la mise en place d'une structure. Si on etudie les
caracteristiques d'un territoire presomptif au cours du temps, il pourrait ^etre possible de
caracteriser la periode d'apparition des comportements speciques a la descendance de la
region. Indirectement il serait donc possible de distinguer les periodes auxquelles se font
les restrictions du potentiel developpemental des cellules."





















Data modification:

	The automated reconstruction workflow does not produce error-less reconstruction. 


expliquer comment sont corrigées les centres, liens grace à l'interface

correction -> existence and position of cell centers, cell link between two consecutive time steps



Key points:


Mapping between the flat representation of the lineage tree and the 3D 

Time controlling: backward and forward

Propagation of measures on a clonal, backward and forward


from papier methodo


Processing supervision, data annotation, validation, correction and analysis were performed with the Mov-IT interactive visualization software




<h4>Validation</h4>




Validation:


Manual validation versus automated validation



Manual validation: statistical analysis from the hand-corrected tracking links




"The tracking error rate was measured (make it only for shorter time) with the sampling procedure described in Figure 4d and Methods section. Altogether, over 15,000 links between the t and t+1 positions of cells have been manually checked at different time points. For the first 300 time steps (5h34 imaging, end of gastrulation) the total error rate is lower than 2%, i.e., more than 98% of the cells were correctly tracked (Figure 4d)."

False positives in nuclear detection were kept below 0.5% and were not the major source of tracking errors. About half of the tracking errors were identified as false trajectories (the tracking jumped from one cell trajectory at time t to another at time t+1) and the other half was false deaths (the tracking did not find the position of the cell at time t+1, and cell trajectory ended). 



Automated validation : the salt and pepper strategy

""
Validation with the above sampling procedure remained a time consuming task. We however showed that similar data sets i.e. corresponding to the same developmental period, labelling and imaging conditions, provided similar results. Nevertheless, we looked for an automated validation strategy and came up with the following. Transplanting about one hundred cells between high stage and sphere stage [Kimmel, 1995 #11] from a donor embryo with double stained nuclei (H2B/mcherry through RNA injection into H2A/eGFP transgenic embryos) into a transgenic host with green nuclei [Pauls, 2001 #236] provided means to both assess our tracking performances in the context of the whole embryo cells tracking (the salt channel) and in the context of a small subpopulation of labelled nuclei (the pepper channel). We were able to show that tracking accuracy in the pepper channel is as good as the hand-made gold standard (gs) and that the automated comparison with either the gold standard or the automatically tracked pepper channel (lineage tree p) provided the same measurement for the “salt channel” tracking accuracy (Figure 5). Mappings mgs,p; mgs,s and ms,p (explain more)   identifying the same cell in both dataset were created. These mappings minimized the distance between matching cells. Cell i is catgoarized as false positive if i is in the set gs / Range(m[gs,p]) (Figure 4 b?), false negative if i is in p / Domain(m[gs,p]) (Figure 4 c?). An error in cell detection is defined as range(m[gs,p]) Δ domain(m[gs,p]).  We conclude about a tracking error if f(it+1)=jt and f(m(it+1)) != m(jt) where f is the predecessor function (Figure 4 d?).

There was no significant difference in errors’ categorization plots when the mappings Mgs,s and Ms,p were compared. The errors characterization done with two different evaluation methods, salt & paper and manually (figure 4), applied to the reconstruction of two individuals, showed similar numbers. The salt & pepper strategy allowed the fully automated evaluation of the reconstruction accuracy and errors’ categorization without manual intervention.

Provided that imaging labelling procedures and imaging set up allowed a third colour and channel, the salt and pepper strategy might be performed still preserving the membrane channel and the possibility to get measurements for cell shape. 
""





</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_1_3_'>
<h3>7.1.3. Post Processing and Exploitation  </h3>
 
 

<p>
	<strong>Spatio-temporal cartography of the cell kinematics</strong>
</p>




 <p>
 	Thèse Benoit -> Lombardot, B., 2010. Description intégrative des cinématiques de la structure embryonnaire : morphogenèse précoce du poisson zébré. Ecole Polytechnique EDX Paris. <a name='Lombardot:2010vd' class='ref'></a>
 </p>



<p>
	On présente rapidement les mesures utilisées par Benoît M U ... en ensuite les cartes. 
</p>



<p>

	Ajouter mesures
</p>



 <figure id='these_lombardot_p93_texture_tensor'>
 	<img src="../../images/Reconstruction/these_lombardot/p93_texture_tensor.png" width="700">
 	<figcaption>Le tenseur de texture de la cellule i (jaune) peut ^etre vu comme une
caracterisation geometrique de la forme du voisinage de i (A). Le tenseur de texture
(vert) est proche de la covariance du voisinage de i (rouge) (B). Le tenseur de texture
peut aussi ^etre interprete comme la covariance du voisinage symetrise de i. adapted from XXXXXX</figcaption>
 </figure>



 <figure id='these_lombardot_p94_texture_tensor_average'>
 	<img src="../../images/Reconstruction/these_lombardot/p94_texture_tensor_average.png" width="700">
 	<figcaption>En general, le tenseur de texture n'est pas deni pour une cellule mais pour
une collection de cellules C (cellule dans le cercle jaune). Comme pour une representation
"coarse grained" du milieu le tenseur de texture a la position X est deni par la moyenne
des tenseurs de texture de chaque cellule de la region de mesure (cercle jaune) (a droite).
En representant tous les liens impliques dans la mesure du tenseur de texture au point X
(M(X)), on comprend bien que M(X) est une caracterisation moyenne de la forme des
voisinages cellulaires dans la region de mesure autour de X.</figcaption>
 </figure>



 <figure id='these_lombardot_p98_texture_tensor_derivative'>
 	<img src="../../images/Reconstruction/these_lombardot/p98_texture_tensor_derivative.png" width="700">
 	<figcaption>La derivee de M traduit la description intuitive de l'evolution du voisinage
d'une cellule. Le tenseur de texture de la cellule i (jaune) a t + dt est egale au tenseur
de texture de i a t (M(t)) auquel trois contributions sont ajoutees : la premiere traduit
la deformation continue des positions de cellules restant dans le voisinage de i (cellules
bleues), la seconde traduit l'entree de cellules dans le voisinage (cellules rouges), enn la
derniere contribution correspond aux cellules sortant du voisinage (cellules vertes). Ces
trois contributions correspondent respectivement aux tenseurs B(t), n_ +M+ et 􀀀n_ 􀀀M􀀀.
Cette interpretation intuitive n'est valide que si le nombre de liens apparus est egal au
nombre de liens disparus.</figcaption>
 </figure>



 <figure id='these_lombardot_p166_velocity_map'>
 	<img src="../../images/Reconstruction/these_lombardot/p166_velocity_map.png" width="700">
 	<figcaption>La gure montre l'evolution du champ de vitesse et de sa norme dans le jeu
de donnees 080322a. C'est une vue laterale. Le triedre rappelle l'orientation de la carte sur
chaque image : l'axe vert pointe le c^ote dorsal et l'axe rouge le p^ole animal. Sur la vignette
A les donnees sont representees sur le modele geometrique de l'embryon (sphere + repere).
L'echelle en bas a gauche de chaque vignette represente une longueur de 100m. Sur les
gures D, E et F on a superpose la topologie du champ de vitesse : point singulier (rouge),
separatrices (bleu), lignes de courant (vert). La vignette B ne presente pas de moyenne
spatiale du champ de deplacement. Les donnees sont presentees de l'interieur du vitellus.
On peut observer les cellules de l'hypoblaste lateral pres de la marge.</figcaption>
 </figure>



 <figure id='these_lombardot_p172_velocity_map_drawings'>
 	<img src="../../images/Reconstruction/these_lombardot/p172_velocity_map_drawings.png" width="700">
 	<figcaption>Les champs de vitesse peuvent ^etre caracterises par une topologie simple.
Les vignettes presentent la topologie du systeme au cours du temps. Chaque schema
represente l'embryon en vue laterale par un cercle. Le c^ote dorsale est a droite, ventrale
a gauche, les p^oles animal et vegetatif respectivement en haut et en bas. Etant donnee
la symetrie bilaterale du systeme cette vue est susante pour caracteriser l'ensemble du
champ de vitesse. Les separatrices sont representees par des lignes rouges, les singularites
par des points rouges. On a trace quelques lignes de courant en vert pour illustrer la
coordination des mouvements de matiere associes a la topologie tracee.</figcaption>
 </figure>



 <figure id='these_lombardot_p180_deformation_rate'>
 	<img src="../../images/Reconstruction/these_lombardot/p180_deformation_rate.png" width="700">
 	<figcaption>La gure presente a 7,5 hpf les champs de directions propres associes
aux taux de deformation dans les tissus. Les valeurs propres vont croissantes de gauche
a droite. La carte de couleur indique le signe des valeurs propres et l'orientation des
directions propres par rapport au tissu (voir la gure 4.5). L'orientation des donnees est
rappelee par le repere (bleu, vert, rouge) : laterale (080322a), animale (071226a), dorsale
(070128c). Sur la vue laterale de la premiere valeur propre, la ligne pointillee marque la
separation entre les regions A et B denis dans la description. Les regions A, B, C, D, E
sont 
echees sur les dierentes vues associees a la premiere valeur propre lorsqu'elles sont
visibles.</figcaption>
 </figure>




 <figure id='these_lombardot_p200_neighbor_shape_U'>
 	<img src="../../images/Reconstruction/these_lombardot/p200_neighbor_shape_U.png" width="700">
 	<figcaption>La gure montre l'evolution des cartes moyennes de la forme des voisinages
cellulaires en vue laterale. Ces cartes sont construites a partir du jeu de donnees 080322a.</figcaption>
 </figure>




 <figure id='these_lombardot_p205_lost_contact'>
 	<img src="../../images/Reconstruction/these_lombardot/p205_lost_contact.png" width="700">
 	<figcaption>La gure montre en vue animale (071226a) des cartes de direction de
disparition des contacts (1ere valeur propre de P􀀀). Le repere indique l'orientation de
l'embryon : c^ote dorsal pointe par la 
eche verte, p^ole animal pointe par la 
eche rouge.
En premiere ligne on a une coupe presentant la carte dans le plan de symetrie bilaterale
de l'embryon. Sur la seconde ligne on a une vue du p^ole animal sur la carte.</figcaption>
 </figure>




 <figure id='these_lombardot_p209_new_contact'>
 	<img src="../../images/Reconstruction/these_lombardot/p209_new_contact.png" width="700">
 	<figcaption>Les cartes presentent la direction principale d'apparition des contacts (1ere
valeur propre du tenseur P+). Une partie de la regionalisation observee correle avec les
directions de contraction du champ de deformation mesoscopique (1ere valeur propre).</figcaption>
 </figure>




 <figure id='these_lombardot_p211_mix_tensor'>
 	<img src="../../images/Reconstruction/these_lombardot/p211_mix_tensor.png" width="700">
 	<figcaption>La gure montre l'evolution de la principale direction propre du tenseur de
melange dans une vue animale (071226a). C'est la direction dans laquelle les translations
de cellules vis-a-vis de leur voisinage sont les plus importantes en moyenne. La partie
superieure de chaque vignette montre une section de la carte dans le plan de symetrie
bilaterale de l'embryon. La partie inferieure de chaque vignette est une vue du p^ole animal.</figcaption>
 </figure>





 <figure id='these_lombardot_p237_deformation_rate_comparison_zoep'>
 	<img src="../../images/Reconstruction/these_lombardot/p237_deformation_rate_comparison_zoep.png" width="700">
 	<figcaption>La gure illustre les dierences macroscopiques observees dans le champ
de deformation continue des tissus en vue animale pour un embryon sauvage (WT) et
un mutant zoep. Le jeu de donnees WT est 071226a, celui du mutant zoep est 081018a.
La structuration du champ de deformation est dierente des 7,5 hpf. Apres 8,5 hpf il se
forme chez le WT une region avec des lignes de deformation paralleles au plan median
(region anterieure aux pointilles). Cette region de part et d'autre de laquelle se forment
les vesicules optiques n'existe pas chez le mutant zoep.</figcaption>
 </figure>





<p>
	<strong>Orientation of mitosis axis</strong>
</p>


<p>
	...
</p>



</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_1_4_'>
<h3>7.1.4. 4D Atlas of Gene Expression  </h3>
 
 
	
<p>
	papier Carlos ?
</p>



</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_2_'>
<h2>7.2. Reconstructing In Toto Deformation Fields </h2>
 
 


	<figure id='washington_workflow'>
		<img src="../../images/Reconstruction/washington/workflow_corrected.png" width="900">
		<figcaption><strong>Global diagram of the reconstruction workflow of the <i>in toto</i> deformation fields. </strong> </figcaption>
	</figure>


<p>
	In this section, we describe an additional module of the Bioemergences workflow, which we developed specially for our study (Fig. <a name='washington_workflow' class='fig'></a>). It takes in input a series of raw microscope images, produces in output a global &quot;deformation field&quot;, and is composed of six image processing components (explained below). 
</p>

<p>
	The rationale is as follows. The biphotonic microscopy devices do not allow the capture of the entire zebrafish embryo because of optical loss of signal and image deformation in the depth of the tissue. One solution is to take advantage of the spherical shape of the embryo during early gastrulation and rotate it to capture multiple views, thus covering the whole volume without major image degradation. The problem with this solution, however, is that the total imaging time is greater since we need consecutive angles of view to capture superposed region of the embryo, and cells move slightly in the meantime. Fortunately, a recent microscopic technology, the Digital Scanning Light Microscope (DSLM), increases the speed of scanning thanks to its ability to illuminate and record the signal emitted by the embryo plane by plane and not only point by point. Faster recording allows a better comparison and match of the 3D volumes to extract and reconstruct the dynamics of the cell trajectories described in the previous section.
</p>

<p>
	However, the quality of the signal is different than the one generated by the biphoton illuminated device used in the previous section, therefore the reconstruction strategies had to be adapted accordingly. We decided to set aside the reconstruction of the <i>in toto</i> lineage tree for later investigation, and instead work on a more accessible reconstruction: the <i>in toto</i> deformation fields of the embryo. This workflow is presented in detail in the subsections below. It was programmed using the OpenGL and ITK/VTK libraries.
</p>

<figure id='washington_5angles_before'>
	<img src="../../images/Reconstruction/washington/5angles_before_letter.png" width="900">
	<figcaption><strong>Original five views produced by the Digital Scanning Light Microscope (DSLM).</strong> Isosurfaces have been extracted from the DSLM raw data. Each view is obtained by rotating the embryo 72 degrees (one color per angle of view). A: Sphere stage. B: 40 percent epiboly. C: 75 percent epiboly. D: 90 percent epiboly. E: 8-somite stage. F: 12-somite stage.</figcaption>
</figure>




</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_2_1_'>
<h3>7.2.1. Manual Registration  </h3>
 
 


	
	<figure id='washington_workflow_manual'>
		<img src="../../images/Reconstruction/washington/workflow_manual_corrected.png" width="600">
		<figcaption><strong>The manual registration component.</strong> This process aims at inferring the parameters required for a spatial registration of the five angles of view. </figcaption>
	</figure>




	<p>
		We explain here the manual registration component (Fig. <a name='washington_workflow_manual' class='fig'></a>). The DSLM allows rapid imaging of the embryo and, coupled with a rotating device, recording multiple views of the same embryo obtained from different angles (Fig. <a name='washington_SPIM' class='fig'></a>). We have been using an exact replicate of the DSLM described in <a name='Keller:2008km' class='ref'></a>. In this whole section 7.2, we use the following conventions to name the axes of each view: the axis from the camera to the embryo is the z-axis or &quot;depth&quot;, the horizontal axis of the light plane is the x-axis, and the vertical axis of the light plane is the y-axis.
	</p>


	<figure id='washington_SPIM'>
		<img src="../../images/Reconstruction/washington/SPIM_corrected.png" width="900">
		<figcaption><strong>Digital Scanning Light Microscope (DSLM) setup.</strong> Diagram after P.J. Keller, E. Stelzer et al. <a name='Keller:2008km' class='ref'></a>, adapted by V. Gurchenkov.</figcaption>
	</figure>





	<p>
		Reconstructing an <i>in toto</i> view of the embryo requires the spatial fusion of the five views within a reference frame. While sophisticated methods of non-rigid spatial registration based on raw image processing and &quot;wavelets&quot; are currently being developed by our team <a name='RubioGuivernau:2012dx' class='ref'></a>, we decided to design our own. Here, we rely on the knowledge of the rotating motion of the microscope plate to geometrically infer a rigid spatial registration. Moreover, we assume that the registration parameters remain constant throughout the imaging process since the embryo is not moving in the agarose chamber and the rotating plate executes precise and controlled displacements. Thus we use a single time step to calculate the parameters, which will be generalized to all other time steps.
	</p>




	<p>
		The embryo is mounted onto a cylindrical agarose tube. The tube is fixed on a table that can rotate around the cylinder's central axis and translate it. As the embryo never passes exactly through that central axis, the software controlling the motion of the table must be calibrated before the time-lapse imaging starts. The calibration phase is equivalent to the following process (Fig. <a name='washington_manual_spim' class='fig'></a>): the operator must center the embryo in the light plane. Once the position is validated, the software performs a 90 degree rotation and the operator must center again the embryo in the light plane. Based on the translation vector created by the operator's action, the software computes the relative position of the embryo with respect to the cylinder's center. However, the computed relative position is only an approximation of the real relative position because the operator's estimation is never exact. This is due to the fact that s/he only sees a projection along the z-axis, and as the embryo rotates, the aspect on the screen changes.
	</p>



	<figure id='washington_manual_spim'>
		<img src="../../images/Reconstruction/washington/manual_spim.png" width="900">
		<figcaption><strong>Schema of the calibration phase. </strong> Left: The operator centers the embryo in the light plane. Right: To decipher the relative position of the embryo with respect to the center of rotation, the software performs a 90 degree rotation, then the operator must translate the embryo back into the light plane (blue vector). </figcaption>
	</figure>



	<p>
		We have programmed a graphical user interface to manually correct the estimation error and determine the optimal relative position (Fig. <a name='washington_manual_registration_selection' class='fig'></a>). The two parameters controlled by the user are the coordinates of the correction vector. As the user moves this vector, s/he can directly observe the motion of the five views computed in real time, hence determine the optimal correction vector. For this task, the nuclei of the margin (yolk syncytial layer), which are separated from the mass of the deep cells, offer useful landmarks for a precise determination of these vector coordinates, i.e. the best match (Fig. <a name='washington_manual_registration_final' class='fig'></a>).
	</p>


	<figure id='washington_manual_registration_selection'>
		<img src="../../images/Reconstruction/washington/manual_registration_selection.png" width="900">
		<figcaption><strong>Graphical user interface designed to infer the estimation of the center of rotation.</strong> The red line passes through the center of rotation determined by the calibration phase. The brown line passes through the corrected center of rotation. As the user moves the brown line, the five angles of views are repositioned in real time, allowing to infer the best estimation of the center of rotation.</figcaption>
	</figure>

	<figure id='washington_manual_registration_final'>
		<img src="../../images/Reconstruction/washington/manual_registration_final.png" width="900">
		<figcaption><strong>Best match for the rigid manual registration. </strong> The marginal yolk nuclei visible on top of the embryo are used to estimate the correct registration parameters. </figcaption>
	</figure>

	<p>
		After the five views have been registered for one time step, the same registration parameters will be applied to the whole time lapse imaging data in the other components of the workflow (Fig. <a name='washington_workflow' class='fig'></a>). However, as it can be noticed in Fig. <a name='washington_5angles_after' class='fig'></a>, the optical deformation induced by the perturbation of the light path through the embryonic tissue causes the result to be rather suboptimal. Other components of the workflow (Voxel Quality Evaluation and Blending Function) will take care of removing these defects in each view.
	</p>



	<figure id='washington_5angles_after'>
		<img src="../../images/Reconstruction/washington/5angles_after_letter.png" width="900">
		<figcaption><strong>The five DSLM views superimposed by rigid registration.</strong> These views are identical to the ones shown in Fig. <a name='washington_5angles_before' class='fig'></a>. A: Sphere stage. B: 40 percent epiboly. C: 75 percent epiboly. D: 90 percent epiboly. E: 8-somite stage. F: 12-somite stage.</figcaption>
	</figure>





</div>
 
 
 
 
 
 
 
<div id='7_2_2_'>
<h3>7.2.2. Voxel Quality Evaluation  </h3>
 
 


	<figure id='washington_workflow_vqe'>
		<img src="../../images/Reconstruction/washington/workflow_vqe_corrected.png" width="600">
		<figcaption><strong>The voxel quality component.</strong> This process aims at quantitatively evaluating the local image quality for each angle of view. </figcaption>
	</figure>
 
	
	<p>
		The voxel quality component runs independently from the manual registration component. Its goal is to determine what spatial region of the volume shows the best signal quality, in each view and at every time step. The actual output of this component is to attribute a quality score to each voxel of the volume. The rationale here is that the farther the light travels through the embryo, the more degraded the signal is. The light beam through the embryo is split into two orthogonal beams: the continuation of the original light beam from the laser, and a new light beam emitted by the fluorescent molecules of the embryo toward the camera. For the voxel quality component, the relevant part of the light beam is inside the embryo. We performed a geometrical computation to determine the length of the light beam from each voxel of the volume to the physical border of the embryo. For the evaluation, we first calculated the border and then the beam length. It can be summarized as follows:
	</p>

	<ul>
		<li>1. To eliminate the low amplitude background noise, the averaged voxel intensity of the volume was computed to determine the isosurface of the nuclei's shapes (Fig. <a name='washington_vqe' class='fig'></a>A). This isosurface is a mesh of triangles. </li>
		<li>2. After a random decimation of the isosurface triangles to decrease the number of vertices, small spheres were centered on these vertices and resized in order to produce a contiguous embryo surface without holes. Then, a binary mask of the image was extracted to store the outer shape of the embryo.</li>
		<li>3. We eliminated spurious signal traces (blue arrows in Fig. <a name='washington_vqe' class='fig'></a>B) that may have subsisted outside of the embryo shape by keeping only the closed region that had the largest volume.</li>
		<li>4. The spheres creating an embryo larger than desired, we eroded the volume by an amount equivalent to the radius of the spheres to obtain the actual physical border of the embryo. The reduced volume is stored as a binary data array with value 1 (resp. 0) for voxels inside (resp. outside) the border (Fig. <a name='washington_vqe' class='fig'></a>C).</li>
		<li>5. Since each view is oriented with its z-axis along the camera path and x-axis along the illumination plane, it is now straightforward to compute both distances from the border by counting the number of 1's along those axes (Fig. <a name='washington_vqe' class='fig'></a>D,E). The voxels that are outside of the border receive an eliminatory score as they cannot be used for the registration task (Fig. <a name='washington_vqe' class='fig'></a>F).</li>
		<li>6. Finally, we also eliminated the voxels located beyond a cutoff distance (about 200\(\mu\)m) in both directions as they belonged to parts of the image that were too degraded to be used (Fig. <a name='washington_vqe' class='fig'></a>G,H,I).</li>
	</ul>
	



	<figure id='washington_vqe'>
		<img src="../../images/Reconstruction/washington/vqe_letter.png" width="900">
		<figcaption><strong>The various steps of the voxel quality evaluation component.</strong> See text.</figcaption>
	</figure>



</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_2_3_'>
<h3>7.2.3. Blending Function  </h3>
 
 
 


	<figure id='washington_workflow_blend'>
		<img src="../../images/Reconstruction/washington/workflow_blend_corrected.png" width="600">
		<figcaption><strong>The blending function component.</strong> This component builds a function determining what original angle of view has the best quality score for each voxel in the registered space.</figcaption>
	</figure>

	

	<p>
		The objective of this task is to use both the manual registration component and the voxel quality evaluation component described in the previous subsections to build a blending function. For each registered voxel, this function selects among all five views which one will be used in the <i>in toto</i> reconstruction. In other words, the blending function produces a 3D integer volume in which each voxel contains the id of the most relevant angle of view according to its quality evaluation. Knowing the coordinates of the center of rotation, we were able to precisely position all five views within a common reference space (Fig. <a name='Reconstruction_5angles_3Dvolume' class='fig'></a>, left). In this space, we visited each voxel and checked for each projected angle of view whether it was located inside the embryo border evaluated in the previous component (Fig. <a name='Reconstruction_5angles_3Dvolume' class='fig'></a>, right). Then, we attributed to this voxel the id of the angle of view (from 1 to 5) under which it had the best quality, or 0 if it was lying outside of all borders. This produced the integer mask shown in Fig. <a name='washington_blendingfunction' class='fig'></a>. Note that instead of the maximum voxel quality, we could also have used a weighted average of all angles. An output of the blending function applied to the raw voxel intensities is shown in Fig. <a name='washington_blendingfunction_raw' class='fig'></a>, providing a first example of <i>in toto</i> registration of the embryo. In our workflow, however, we applied the blending function to the &quot;deformation fields&quot; calculated in the next component.
	</p>



	<figure id='Reconstruction_5angles_3Dvolume'>
		<img src="../../images/Reconstruction/5angles_fusion.png" width="900">
		<figcaption><strong>Relative positions of the five views in the registered space.</strong> Left: The five cubes represent the original reference frames of the five different views. Right: A 2D illustration of the sequential visit of the registered voxels.</figcaption>
	</figure>
	


	
	<figure id='washington_blendingfunction'>
		<img src="../../images/Reconstruction/washington/blendingfunction.png" width="900">
		<figcaption><strong>Five-valued blending function mask.</strong> In the registered space, the color of each voxel represents the id of the original angle of view under which it has the best quality. </figcaption>
	</figure>




	<figure id='washington_blendingfunction_raw'>
		<img src="../../images/Reconstruction/washington/blendingfunction_raw.png" width="900">
		<figcaption><strong>Output of the blending function applied to the raw voxel intensities.</strong> </figcaption>
	</figure>




</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_2_4_'>
<h3>7.2.4. Deformation Fields  </h3>

	<figure id='washington_workflow_deffield'>
		<img src="../../images/Reconstruction/washington/workflow_deffield_corrected.png" width="600">
		<figcaption><strong>The deformation field component.</strong> This component produces a 3D vector field that indicates the direction and amplitude of local displacements in each voxel.</figcaption>
	</figure>


	<p>
		The objective of the deformation field component is to reconstruct the spatial deformation that relates two 3D volumes produced by the DSLM at consecutive time steps under the same angle of view (one of the five; Fig. <a name='washington_vueGlobal_frame' class='fig'></a>), which we denote here by V and V'. The notion of image deformation is close to the notion of optical flow as its goal is to compute the image motion between consecutive images. We employed a non-rigid registration method called &quot;Demons Registration&quot;: it acts as if voxel-sized &quot;demons&quot; were pulling and pushing the voxel of volume V' along the local gradient of voxel intensity in volume V. This method is iterative and the demons run for a predefined number of steps, or until some matching criterion is reached. The implementation that we used here is based on Thirion's algorithm <a name='Thirion:1998hg' class='ref'></a>. Fig. <a name='washington_vueDeffield2ts' class='fig'></a> shows the superposition of the isosurfaces of V (in white) and V' (in yellow) at the marginal region of the zebrafish embryo. As the deformation field was computed in the entire images of V and V', and not only at the position of the nuclei, we had to filter out spurious vectors from the resulting field. The vectors belonging to voxels located inside the nucleus envelopes (corresponding to another isosurface obtained by thresholding the voxels at twice the mean volume intensity) were retained, while the others were nullified. We calculated the deformation fields for each angle of view separately. The next component takes care of blending them together.
	</p>






	<figure id='washington_vueGlobal_frame'>
		<img src="../../images/Reconstruction/washington/deffield/vueGlobal_frame.png" width="900">
		<figcaption><strong>One volume V under one of the five angles of view.</strong> The white grains represent the envelopes of the nuclei calculated by thresholding the voxels at twice the mean intensity of V.</figcaption>
	</figure>

	<figure id='washington_vueDeffield2ts'>
		<img src="../../images/Reconstruction/washington/deffield/vueDeffield2ts.png" width="900">
		<figcaption><strong>Deformation field between two volumes V and V' at consecutive time steps.</strong> The white isosurface represents V, the yellow isosurface represents V'. Not all deformation arrows are displayed.</figcaption>
	</figure>




</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
<div id='7_2_5_'>
<h3>7.2.5. Blended Deformation Fields  </h3>
 



	<figure id='washington_workflow_blend_deffield'>
		<img src="../../images/Reconstruction/washington/workflow_blend_deffield_corrected.png" width="600">
		<figcaption><strong>The blended deformation field component.</strong> This component combines the blending function with the 3D deformation vector field.</figcaption>
	</figure>


	<p>
		Finally, the blending function obtained in Section 7.2.3 is applied to the five deformation vector fields obtained in Section 7.2.4. Each voxel of the reference volume simply receives the deformation vector from the angle of view that has the best quality evaluation score.
	</p>

	<figure id='washington_blendingfunction_deffield'>
		<img src="../../images/Reconstruction/washington/blendingfunction_deffield.png" width="900">
		<figcaption><strong>Blended deformation field at a given time step viewed from four different angles.</strong> (Note that these angles do not correspond to the original angles of view.)</figcaption>
	</figure>


	<figure id='washington_blendingfunction_deffield_multits'>
		<img src="../../images/Reconstruction/washington/blendingfunction_deffield_multits.png" width="900">
		<figcaption><strong>Blended deformation field at successive time steps viewed from the same angle.</strong> </figcaption>
	</figure>



 
 
</div>
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
