
<figure id='homonculi_Hartsoecker_1695'>
	<img src="../../images/Development_Review/homonculi_Hartsoecker_1695.png" width="500">
	<figcaption>Drawing of the homonculi observed in sperm by Nicolaas Hartsoecker (1695)</figcaption>
</figure>
<figure id='Development_Review_dog_segmentation_kolliker_1861'>
	<img src="../../images/Development_Review/dog_segmentation_kolliker_1861.png" width="700">
	<figcaption>Egg segmentation from a dog's oviduct, surrounded by the zona pellucida and spermatozoa as represented by Albert KÃ¶lliker in 1861 <a name='Kolliker:1861uj' class='ref'></a>.</figcaption>
</figure>
<figure id='Development_Review_waddington_epigenetic_nature_mod_small_intro'>
	<img src="../../images/Development_Review/waddington_epigenetic_nature_mod_small.png" width="800">
	<figcaption><strong>Waddington's epigenetic landscape.</strong> A: The ball represents a cell evolving in the epigenetic landscape. Its fate is determined by the canals in which the ball is rolling. B: A view of the landscape's behind the scene. The landscape relief is dynamically controlled by hidden wires that symbolize genes' expression and interactions. Image and caption adapted from Slack <a name='Slack:2002kg' class='ref'></a>.</figcaption>
</figure>
<figure id='experimental_science_schematic'>
	<img src="../../images/experimental_science/experimental_science_raw_v2.png" width="950">
	<figcaption><strong>The observation-hypothesis-experiment loop of experimental science. </strong> It involves two main actors: the individual and reality (environment); and three main processes: perception, hypothesis generation (modeling), and experimentation.</figcaption>
</figure>
<figure id='experimental_science_schematic_communauty'>
	<img src="../../images/experimental_science/experimental_science_communauty2.png" width="950">
	<figcaption><strong>The collective effort of experimental science.</strong> Each member of the scientific community may send or receive scientific work, which are verbal or written descriptions of parts or the whole observation-hypothesis-experiment loop.</figcaption>
</figure>
<figure id='experimental_science_perception_augmented'>
	<img src="../../images/experimental_science/experimental_science_perception_augmented2.png" width="950">
	<figcaption><strong>Augmented perception with interfacing tool (optical microscope) and a reconstruction workflow.</strong></figcaption>
</figure>
<figure id='schema_raw_reconstructed_embryo_macroscopic'>
	<img src="../../images/experimental_science/schema_raw_reconstructed_embryo_macroscopic.png" width="1000">
	<figcaption><strong>The three major steps of the reconstruction in the BioEmergences platform: raw data, reconstructed embryo and macroscopic measures. </strong> Left: raw data of the zebrafish developing head (nuclei labeled in green and cell membrane in red). Center: visualization of the reconstructed embryo with the MoveIt software. Small dots are the cell centers and arrows gives the future cell displacement. Colors indicates the velocities of the cell. The central image illustrates the extremely high dimensionality of the reconstructed embryo. Right: Macroscopic measures of the displacement field in the developing embryo (from B. Lombardot PhD manuscript <a name='Lombardot:2010vd' class='ref'></a>). Images realized by the BioEmergences team.</figcaption>
</figure>
<figure id='experimental_science_multi_initialstate'>
	<img src="../../images/experimental_science/multi_initialstate_fusion.png" width="1000">
	<figcaption>Simple illustration of a dynamical system: if a ball is hit by an object, it will move in space following a parabolic curve until it lands on the ground. A classical mechanics model would consider the curve (i.e. the temporal evolution of the position) as the <q>phenotype</q>, which is determined only from an initial known position and the velocity of the ball. Left: parameter space, Right: phenotypic space.</figcaption>
</figure>
<figure id='experimental_science_hypotheses_augmented'>
	<img src="../../images/experimental_science/experimental_science_hypotheses_augmented2.png" width="950">
	<figcaption><strong>Augmentation of hypothesis generation through theoretical model and computer simulation.</strong></figcaption>
</figure>
<figure id='experimental_science_larger_augmented'>
	<img src="../../images/experimental_science/experimental_science_larger_augmented.png" width="900">
	<figcaption><strong>Integrative schematic of the tool-augmented observation-hypothesis-experiment loop.</strong></figcaption>
</figure>
<figure id='experimental_science_phase_diagram2'>
	<img src="../../images/experimental_science/phase_diagram2.png" width="700">
	<figcaption><strong> Generic evaluation of the performance of a model through quantitative evaluation of its parameter space.</strong> To each element of the parameter space (left column), a deterministic dynamical system model associates a trajectory that may be represented by a single point in the extensive microscopic measurement space (this is the state space but we want to emphasize the complexity and the high dimensionality of this space when the model is agent-based, center column). The macroscopic measurement point are obtained by reconstructing and measuring the observations of interest in the complex trajectory space. Row 1 illustrates the passage from the parameter space to the extensive microscopic measurement space (red arrows). As shown by the arrow pointing to the white dot, there is no guaranty that every point in the parameter space produces a <q>viable</q> trajectory. Quantitative assessment of the performance of the trajectory are determined according to adapted criteria (comparison with a target measure Fig. <a name='experimental_science_phase_diagram_distance' class='fig'></a>, clustering). A <q>color</q> is attributed to the macroscopic measure points to symbolize this evaluation (Row 2). The color is easily propagated back to the parameter space and allows a quantitative assessment of the performance of every trajectory (Row 3, 4, 5). White regions represent parts of the parameter space that are not viable (Row 5).</figcaption>
</figure>
<figure id='experimental_science_phase_diagram_distance'>
	<img src="../../images/experimental_science/phase_diagram_distance.png" width="800">
	<figcaption><strong>Fitness landscape according to a target macroscopic measure.</strong> This schema is a particular example illustrating the case of a model evaluated by the comparison a macroscopic measure (black cross). The score of each trajectory is attributed as a function of the distance between the simulated macroscopic measures and the target macroscopic measure.</figcaption>
</figure>
<figure id='experimental_science_experimental_science_cleaner'>
	<img src="../../images/experimental_science/experimental_science_cleaner.png" width="950">
	<figcaption><strong>Summary of the methodological workflow adopted for this project.</strong> The top left \(\Delta\) symbolizes the theoretical fitness (TF) and the top right \(\Delta\) the experimental reconstruction fitness (ERF).</figcaption>
</figure>
<figure id='biomechanics_sota_jacobson1976'>
	<img src="../../images/biomechanics_sota/jacobson1976.png" width="500">
	<figcaption><strong>One of the earliest computer simulations of spatial transformation during vertebrate development.</strong> <q>A. Appearance of the neural plate immediately after isolation without notochord at stage 13. B. Shape attained in the same explant when controls had reached mid-neurula stage 15. C. Results of a computer simulation using shrinkage alone, which would be compared to B</q>. Images and caption from Jacobson and Gordon, 1976 <a name='Jacobson:1976cc' class='ref'></a>.</figcaption>
</figure>
<figure id='biomechanics_sota_young_1984'>
	<img src="../../images/biomechanics_sota/young_1984.png" width="600">
	<figcaption><strong>Patterns produced with the activator-inhibitor model.</strong> As inhibition is decreased (left to right), the spot pattern connects up into a pattern of stripes. Images and caption from Young, 1984 <a name='Young:1984hd' class='ref'></a>.</figcaption>
</figure>
<figure id='gradient_tabata_2004'>
	<img src="../../images/Development_Review/theoretical_model/gradient/tabata_2004.png" width="400">
	<figcaption><strong>Models for morphogen transport.</strong> (A) A model for cytonemes. Cells at the periphery of the imaginal disc extend long processes, cytonemes, towards the AP border, where Dpp is expressed (light blue). (B) A model for argosomes. The basolateral membranes of imaginal disc cells vesiculate and travel throughout the disc epithelium. Image and caption from Tabata and Takei (2004) <a name='Tabata:2004bp' class='ref'></a></figcaption> 
</figure>
<figure id='gradient_ben_zvi_2011'>
	<img src="../../images/Development_Review/theoretical_model/gradient/ben_zvi_2011.png" width="700">
	<figcaption><strong>Scaling of morphogen gradients.</strong> (d) The ratio (blue) between two gradients emanating from opposing edges of the field (green and red) can provide scaling of a signaling gradient. Left panel: wild type field; right panel: larger field. Image and caption excerpt from <a name='BenZvi:2011ky' class='ref'></a></figcaption> 
</figure>
<figure id='biomechanics_sota_epithelium'>
	<img src="../../images/biomechanics_sota/epithelium.png" width="900">
	<figcaption><strong>Selected illustrations from the literature of epithelial cell shaping and division patterns.</strong> A: Comparison of the distributions of polygonal epithelial cells obtained from simulation (yellow), the <i>Drosophila</i> wing disc (pink), the <i>Xenopus</i> tail epidermis (green) and the <i>Hydra</i> epidermis (blue) <a name='Gibson:2006gia' class='ref'></a>. B: Schematized Markov chain model expressing the proliferation dynamics of polygonal cells and used in the simulation of A <a name='Gibson:2006gia' class='ref'></a>. C: Typical cell division event in a vertex-based model <a name='Farhadifar:2007vj' class='ref'></a>.</figcaption>
</figure>
<figure id='leveloforganization_circle'>
	<img src="../../images/levels_of_organization/leveloforganization_circle2.png" width="900">
	<figcaption>Biology is studied at many levels of organization.</figcaption>
</figure>
<figure id='leveloforganization_micro_macro'>
	<img src="../../images/levels_of_organization/micro_macro/micro_macro_julien.png" width="800">
	<figcaption><strong>The hierarchy of level of organization of physical matter.</strong> At each level of organization, entities are <i>complex systems</i> composed of a great number of small, repeated elements. A: Level view. B: Integrated view.</figcaption>
</figure>
<figure id='theoretical_model_review1'>
	<img src="../../images/Development_Review/theoretical_model/review3_part1_higher.png" width="950">
	<figcaption><strong>Summary table of the developmental models and principles reviewed in Section 2.</strong> (Part I)</figcaption>
</figure>
<figure id='theoretical_model_review2'>
	<img src="../../images/Development_Review/theoretical_model/review3_part2_higher.png" width="950">
	<figcaption><strong>Summary table of the developmental models and principles reviewed in Section 2.</strong> (Part II)</figcaption>
</figure>
<figure id='schematic_mecagen'>
	<img src="../../images/MECAGEN/schematic/schemaMECAGEN.png" width="950">
	<figcaption><strong>General diagram of the MECAGEN project.</strong> a: Biomechanical processes, including a <q>passive</q> and <q>active</q> forces (see summary of Chapter 3 above). b: Molecular/genetic regulation and signaling processes, including gene regulation networks, intracellular protein concentrations, extracellular ligands secretion, diffusion and binding.</figcaption>
</figure>
<figure id='experimental_science_experimental_science_cleaner_focus_theo'>
	<img src="../../images/experimental_science/experimental_science_cleaner_focus_theo.png" width="750">
	<figcaption><strong>Situation of Chapter 3 in the methodological workflow.</strong></figcaption>
</figure>
<figure id='biomechanics_sota_general_type_of_models'>
	<img src="../../images/biomechanics_sota/general_type_of_models.png" width="700">
	<figcaption>The schematic categorization of the models adopted in this review.</figcaption>
</figure>
<figure id='solids_1000px-Stress_in_a_continuum.png'>
	<img src="../../images/biomechanics_sota/solids/1000px-Stress_in_a_continuum.png" width="450">
	<img src="../../images/biomechanics_sota/solids/1000px-Plane_strain.png" width="450">
	<figcaption><strong>Stress and Strain.</strong> Left. Stress from a loaded deformable material body assumed as a continuum from <a href="http://en.wikipedia.org/wiki/Stress_(mechanics)#Cauchy_stress_tensor">Wikipedia</a>. Right. Plane strain state in a continuum from <a href="http://en.wikipedia.org/wiki/Plane_strain#Plane_strain">Wikipedia</a>.
 </figcaption>
</figure>
<figure id='CPM_onlattice_potts.png'>
	<img src="../../images/CPM/onlattice_potts.png" width="600">
	<figcaption>Cellular Potts Model on-lattice space.</figcaption>
</figure>
<figure id='MECAGEN_selfassembly3D'>
	<img src="../../images/MECAGEN/selfassembly3D.png" width="600">
	<figcaption><strong>One of our early experiments of single-particle cell models exploring the biomechanics of the MECAGEN project.</strong> Each cell is represented by a small sphere and interacts with its nearest neighbors in a Delaunay tessellation through elastic forces (see Section 3.2).</figcaption>
</figure>
<figure id='beforeMECAGEN_selfassembly3D'>
	<video src="../../videos/video_modeling/beforeMECAGEN/selfassembly3D.ogg" width="800" controls></video>
	<figcaption><strong>Video of single-particle cell division.</strong> This corresponds to Fig. <a name='MECAGEN_selfassembly3D' class='fig'></a>.</figcaption>
</figure>
<figure id='MECAGEN_multi_two_models'>
	<img src="../../images/MECAGEN/multi_two_models.png" width="800">
	<figcaption><strong>Two of our early experiments of multiparticle cell models exploring the biomechanics of the MECAGEN project.</strong> Left: Each cell is a sac filled with a great number of subcellular elements, appearing as vertices of a triangular mesh on the cell membrane. Green edges schematize intercellular adhesion forces (Movie <a name='beforeMECAGEN_MemInECMmitosis' class='fig'></a>). Right: Similar experiment with simpler empty polyhedric cells.</figcaption>
</figure>
<figure id='beforeMECAGEN_MemInECMmitosis'>
	<video src="../../videos/video_modeling/beforeMECAGEN/MemInECMmitosis.ogg" width="800" controls></video>
	<figcaption><strong>Video of multiparticle cell division and growth.</strong> This corresponds to Fig. <a name='MECAGEN_multi_two_models' class='fig'></a>, Left.</figcaption>
</figure>
<figure id='sandersius_newman_4_images'>
	<img src="../../images/biomechanics_sota/sandersius_newman/4_images.png" width="750">
	<figcaption><strong>Illustrations of the Subcellular Element Model (ScEM).</strong> Top left: Schematic diagram showing two cells, \(i\) and \(j\), and a subset of the intra- (solid lines) and intercellular (dashed lines) interactions between their sub-elements (open circles in \(i\), filled circles in \(j\)). Image and caption adapted from <a name='Newman:2005ux' class='ref'></a>. Top Right: Visualization of two interacting cells simulated using the ScEM and comprising about 500 elements each. Image and caption from <a name='Sandersius:2011ih' class='ref'></a>. Bottom left: Visualization of an ScEM simulation of a single cell being stretched by a constant unilateral applied force. Image and caption from <a name='Sandersius:2011ih' class='ref'></a>. Bottom right: A spherical cluster of about 110 cells with about 370 elements per cell. Image and caption from <a name='Sandersius:2008ia' class='ref'></a>. </figcaption>
</figure>
<figure id='spatial_neighb_field_volume_ratio'>
	<img src="../../images/MECAGEN/spatial_neighb/field_volume_ratio.png" width="600">
	<figcaption><strong>Ratio of the physical volume of a particle over the volume of its local interaction field.</strong> Atoms and planet have an interaction range that is much larger that their physical envelope whereas cells can only mechanically interact by physically touching their neighbors.</figcaption>
</figure>
<figure id='spatial_neighb_voronoi'>
	<img src="../../images/MECAGEN/spatial_neighb/voronoi.png" width="500">
	<figcaption>2D Voronoi diagram generated with the <a href="http://www.liefke.com/hartmut/cis677/">Voronoi sweeping algorithm applet</a> by H. Liefke.</figcaption>
</figure>
<figure id='spatial_neighb_Delaunay_circumcircles_centers'>
	<img src="../../images/MECAGEN/spatial_neighb/Delaunay_circumcircles_centers.png" width="300">
	<figcaption><strong>A Delaunay triangulation in the 2D plane.</strong> Black dots are the particles, black lines their Delaunay neighborhood links, and red dots the circumcircles' centers. No circumcircle contains a particle in its interior. Image from <a href="http://en.wikipedia.org/wiki/Delaunay_triangulation">http://en.wikipedia.org/wiki/Delaunay_triangulation</a></figcaption>
</figure>
<figure id='Contact_mechanics_two_cells_contact'>
	<img src="../../images/Contact_mechanics/two_cells_contact.png" width="500">
	<figcaption><strong>Typical profile of the contact radius as a function of the particles' distance in contact mechanics (with adhesion).</strong> In abscissa: distance between the particles; in ordinate: contact radius. As the adhesive energy increases or the elastic modulii decrease, the equilibrium contact surface area tends toward the upper left part of the curve. </figcaption>
</figure>
<figure id='Contact_mechanics_multi_cells_contact'>
	<img src="../../images/Contact_mechanics/multi_cells_contact.png" width="500">
	<figcaption><strong>Multicellular limits of contact mechanics.</strong> As the adhesion energy increases, the contact surface areas become larger and the volume of the central cell is not conserved (bottom right). Contact mechanics does not deal well with multiple surfaces of contact when <q>triple junction points</q> appear (from top right to bottom). </figcaption>
</figure>
<figure id='spatial_neighb_spherecontact_edge_surf_link_figure'>
	<img src="../../images/MECAGEN/spatial_neighb/spherecontact_edge_surf_link_figure.png" alt="" width="900">
	<figcaption>Output of the neighborhood algorithm. Left: A 2D swarm of cells characterized by various given positions and radii. Right: The neighborhood links (black edges) and contact surface areas (red edges) between cells calculated by the method described in this section.</figcaption>
</figure>
<figure id='spatial_neighb_surface_distance_why_all_figure'>
	<img src="../../images/MECAGEN/spatial_neighb/surface_distance_why_all_figure.png" alt="" width="900">
	<figcaption><strong>Schema of the particle-cell and its maximum distance of deformation. </strong> The smaller circle gives the radius of the spherical cell and the larger circle gives the maximum distance of deformation. </figcaption>
</figure>
<figure id='spatial_neighb_Fusion_light'>
	<img src="../../images/MECAGEN/spatial_neighb/surf_dist_movie_1677cells_gS11_filtered_SNAPSHOT/Fusion_light.png" alt="" width="600">
	<figcaption><strong>Estimation of the contact surface area in a weighted Delaunay/Voronoi simulation.</strong> An ellipsoid filled with 300 cells is artificially deformed (stretched and compressed) to generate various distance-surface measurement pairs. Blue cells indicate local compression (i.e. weighted Voronoi volume less than theoretical volume), red cells indicate local dilation.</figcaption>
</figure>
<figure id='potential_surf_dist_movie_1677cells_gS11_filtered'>
	<video src="../../videos/MECAGEN/potential/surf_dist_movie_1677cells_gS11_filtered.ogg" width="800" controls></video>
	<figcaption> <strong>Estimation of the contact surface area in a weighted Delaunay/Voronoi simulation.</strong> An ellipsoid filled with 300 cells is artificially deformed (stretched and compressed) to generate various distance-surface measurement pairs. Blue cells indicate local compression (i.e. weighted Voronoi volume less than theoretical volume), red cells indicate local dilation.</figcaption>
</figure>
<figure id='spatial_neighb_g10_g11_exp9_simple'>
	<img src="../../images/MECAGEN/spatial_neighb/g10_g11_exp9_simple.png" alt="" width="600">
	<figcaption><strong>Scatter plot of the contact surface area between neighboring Voronoi domains as a function of the distance between their centers.</strong> Each dot is a surface-distance pair of values obtained from two neighbors according to the weighted Voronoi algorithm. The color map reflects the density in plot space (white for highest density). The central yellow curve is the regression of the distribution (segmented curve calculated at discrete distance intervals).</figcaption>
</figure>
<figure id='spatial_neighb_zoomALL'>
	<img src="../../images/MECAGEN/spatial_neighb/zoomALL.png" alt="" width="800">
	<figcaption><strong>Heat map of the sum of squared errors of prediction (SSE) for various pairs of parameters \((a,c_{\mathrm{max}})\).</strong> Blue regions indicate the best fitness domains, with point \((a=1.3697 , c_{\mathrm{max}}=1.2414)\) representing the absolute best fit. In abscissa: \(c_{\mathrm{max}}\); in ordinate: \(a\); in color: \(\log(\log(\mathrm{SSE}))\) to accentuate the contrast.</figcaption>
</figure>
<figure id='spatial_neighb_multi_scatter_gp1_gp2_exp9_14'>
	<img src="../../images/MECAGEN/spatial_neighb/multi_scatter_gp1_gp2_exp9_14.png" alt="" width="800">
	<figcaption><strong>Comparison of the best fit with measures from various experiments. </strong> </figcaption>
</figure>
<figure id='spatial_neighb_gabriel'>
	<img src="../../images/MECAGEN/spatial_neighb/gabriel.png" alt="" width="400">
	<figcaption>An example of Gabriel graph on four nodes K, L, M and N. Only edges KL, LM and MN are valid because the disks whose diameters are these edges do not contain any other node (whereas, for example, the disk based on LN contains node M).</figcaption>
</figure>
<figure id='spatial_neighb_gabdelau'>
	<img src="../../images/MECAGEN/spatial_neighb/gabdelau.png" alt="" width="400">
	<figcaption>A 2D Gabriel graph is a subset of a Delaunay triangulation. Here, 5 solid lines represent the Delaunay edges. The Gabriel graph has only 4 of them because it excludes edge \(ab\). Visually, a Delaunay edge is also a Gabriel edge if it intersects the contact border (dashed lines) between the two cell domains (schema from <a name='Cazals:2004wc' class='ref'></a>)</figcaption>
</figure>
<figure id='spatial_neighb_hexagonal_compact'>
	<img src="../../images/MECAGEN/spatial_neighb/hexagonal_compact.png" alt="" width="500">
	<figcaption><strong>Perfect hexagonal lattice.</strong> In the 2D plane, the hexagonal lattice configuration realizes the densest packing of disks. </figcaption>
</figure>
<figure id='spatial_neighb_hexagonal_compact3D'>
	<img src="../../images/MECAGEN/spatial_neighb/hexagonal_compact3D.png" alt="" width="500">
	<figcaption><strong>Hexagonal close packing in 3D. </strong></figcaption>
</figure>
<figure id='potential_trapezo_all_figure'>
	<img src="../../images/MECAGEN/potential/trapezo_all_figure.png" alt="" width="900">
	<figcaption>Trapezo-rhombic dodecahedra (TRD) constitute the Voronoi domains of a maximally compact 3D packing of equally sized solid spheres.</figcaption>
</figure>
<figure id='potential_apf_sphere_figure'>
	<img src="../../images/MECAGEN/potential/apf_sphere_figure.png" alt="" width="800"> 
	<figcaption>The green sphere has the same volume as the trapezo-rhombic dodecahedron. The orange sphere has a radius \(c_{\mathrm{eq}}R_i\) tangential to each face of the TRD.</figcaption>
</figure>
<figure id='potential_surface_distance_why_potential'>
	<img src="../../images/MECAGEN/potential/surface_distance_why_potential.png" alt="" width="600">
	<figcaption><strong>Schema of the particle-cell, its equilibrium distance and its maximum distance of deformation. </strong> </figcaption>
</figure>
<figure id='potential_surface_distance_why_potential_rep_eq_adh'>
	<img src="../../images/MECAGEN/potential/surface_distance_why_potential_rep_eq_adh.png" alt="" width="950">
	<figcaption><strong>Neighbor particle-cells in attraction (left), equilibrium (middle) and repulsion (right) configuration.</strong></figcaption>
</figure>
<figure id='potential_potential_linear_force'>
	<img src="../../images/MECAGEN/potential/potential_linear_force.png" alt="" width="900">
	<figcaption><strong>Plot of the linear spring-like force \(\vec{F}^{P,\mathrm{lin}}\).</strong> The repulsion coefficient \(w_{\mathrm{rep}}\) is fixed, while the adhesion coefficient \(w_{\mathrm{adh}}\) can vary.</strong> Here, the equilibrium distance is \(r^{\mathrm{eq}} = 1\).</figcaption>
</figure>
<figure id='potential_potential_surface'>
	<img src="../../images/MECAGEN/potential/potential_surface.png" alt="" width="900">
	<figcaption><strong>Contact surface area \(A_{ij}\) as a function of intercellular distance \(r_{ij}\).</strong> Here, the sum of \(R_i\) and \(R_j\) is \(1\).</figcaption>
</figure>
<figure id='potential_potential_linear_force_scale'>
	<img src="../../images/MECAGEN/potential/potential_linear_force_scale.png" width="900">
	<figcaption><strong>Plot of the relaxation force \(\vec{F}^P\) with variable adhesion coefficient \(w_{\mathrm{adh}}\).</strong> </figcaption>
</figure>
<figure id='potential_potential_superpose'>
	<img src="../../images/MECAGEN/potential/potential_superpose.png" width="900">
	<figcaption><strong>Comparison of the relaxation force used in MECAGEN with various classical attraction-repulsion forces.</strong> (See legend inset.)</figcaption>
</figure>
<figure id='protrusion_multicell_white_rec_figure'>
	<img src="../../images/MECAGEN/protrusion/multicell_white_rec_figure.png" alt="" width="650">
	<figcaption><strong>Cellular protrusion illustrated at the subcellular level.</strong> The cell cytoskeleton is in blue, the cell membrane in grey, the thin orange edges are the regular adhesion bridges and the thick red edges are focal adhesion points, related to the protrusion. The gradient of color in the cell interior reflects the polarization axis. New focal adhesion points appear at the tip of the cell, move back, then disappear as the cell moves forward.</figcaption>
</figure>
<figure id='protrusion_protrusion_schematic_left'>
	<img src="../../images/MECAGEN/protrusion/protrusion_schematic_left.png" alt="" width="500">
	<figcaption><strong>Idealized representation of the protrusive activity in MECAGEN.</strong> Green arrows represent the cell interior flow and red arrows the cell neighborhood flow, as the central cell exerts a protrusion over its surroundings.</figcaption>
</figure>
<figure id='protrusion_protrusion_schematic_figure'>
	<img src="../../images/MECAGEN/protrusion/protrusion_schematic_figure2.png" alt="" width="950">
	<figcaption><strong>Schematic representation of active protrusive activity in MECAGEN.</strong> See text for details. This illustrates the formalization of the idealized mechanism of Fig. <a name='protrusion_protrusion_schematic_left' class='fig'></a> in the particle-based framework. (A,B) The polar domain (green slice) of cell \(i\), denoted by \(\mathcal{N}^{t+}_i\), contains two neighbor cells \(j\) and \(k\), over which \(i\) exerts a protrusive force. <q>Intrinsic</q> forces are shown in green and extrinsic forces in red. (A) Highlighting one pair of opposite forces. (B) Forces produced by \(i\)'s activity. (C) Forces produced by \(j\)'s activity if \(j\), too, happens to be protruding. (D) The net resulting <q>active</q> forces (not shown) are obtained by adding the net resulting '\(\mathrm{int}\)' and '\(\mathrm{ext}\)' arrows on each cell.</figcaption>
</figure>
<figure id='experimental_science_experimental_science_cleaner_focus_theo_2'>
	<img src="../../images/experimental_science/experimental_science_cleaner_focus_theo.png" width="750">
	<figcaption><strong>Situation of Chapter 4 in the methodological workflow.</strong></figcaption>
</figure>
<figure id='model_grns_zebra_grn_yuh'>
	<img src="../../images/model_grns/zebra_grn_yuh.png" width="600">
	<figcaption><strong>Example of BioTapestry representation of a developmental gene regulatory network (GRN) in the zebrafish.</strong> Figure from Chan et al. (2009) <a name='Chan:2009er' class='ref'></a>.</figcaption>
</figure>
<figure id='Case_theo_grn_generic_grn_cell_shape'>
	<img src="../../images/Cases_Studies/Case_theo_grn/generic_grn_cellshape.png" width="900">
	<figcaption><strong>Example of abstract gene regulatory network (GRN) produced for this study using the BioTapestry software.</strong> This GRN illustrates the typical molecular/genetic regulation and signaling components used in MECAGEN: protein-protein interactions (<q>PPI</q>), genes and their cis-regulatory elements (<q>Gene X</q>), signal secretion module (<q>Secretor X</q>), signal transduction module (<q>Receptor X</q>). The solid arrows in the cell interior represent intracellular proteins; the dashed arrows outside the cell membrane represent extracellular ligands. Practically, the topology and parameters of the GRN are all specified by the user of BioTapestry, then an XML file is saved and can serve as an input into the model.</figcaption>
</figure>
<figure id='model_grns_diffusion_schematic'>
	<img src="../../images/model_grns/diffuse_volume_augmented_crop.png" width="800">
	<figcaption><strong>Schema of the flux of ligand.</strong> Here, \(\vec{J}\) represents \(\vec{J}_{a,ij} = \vec{J}_{a,ji}\), the flux of ligand \(Q_a\) between the cellular volumes occupied by neighboring particles \(i\) and \(j\).</strong></figcaption>
</figure>
<figure id='double_negative_gate_dng_grn'>
	<img src="../../images/Cases_Studies/Case_theo_grn/double_negative_gate/dng_grn.png" width="900">
	<figcaption><strong><q>Double negative gate</q> GRN subcircuit.</strong> We generated this network map with the BioTapestry software from the input file of our experimental model.</figcaption>
</figure>
<figure id='double_negative_gate_dng_grn_plot2'>
	<img src="../../images/Cases_Studies/Case_theo_grn/double_negative_gate/double_negative_gate_plot2_letter.png" width="900">
	<figcaption><strong>Evolution of the proteins concentration in the region where X is expressed.</strong> See text for the explanation of the curves' profile. The tagged vertical bars A, B, C, and D indicate the timing of the snapshots shown in Fig. <a name='double_negative_gate_double_negative_gate_spatial' class='fig'></a>. The curve of Target1 (pink) is hidden by the curve of Target2 (blue) as their dynamics is exactly the same.</figcaption>
</figure>
<figure id='double_negative_gate_double_negative_gate_spatial'>
	<img src="../../images/Cases_Studies/Case_theo_grn/double_negative_gate/double_negative_gate_spatial.png" width="900">
	<figcaption><strong>Spatial evolution of the protein concentration in the double negative gate experiment.</strong> The letters correspond to the bars of Fig. <a name='double_negative_gate_dng_grn_plot2' class='fig'></a>. Each image is composed of 6 simultaneous views of the simulated cell population. Colors represent protein types and shades represent their concentrations (dark for low, bright for high). The top left corner of each image is the region where protein X is artificially secreted.</figcaption>
</figure>
<figure id='MECAGEN_Case_theo_grn_dng_x4'>
	<video src="../../videos/MECAGEN/Case_theo_grn/dng_x4.ogg" width="800" controls></video>
	<figcaption><strong>Real-time representation of the simulation in the double negative gate experiment.</strong> Each image is composed of 6 simultaneous views of the same simulated cell population. Colors represent protein types and shades represent their concentrations (dark for low, bright for high).</figcaption>
</figure>
<figure id='experimental_science_experimental_science_cleaner_focus_theo_3'>
	<img src="../../images/experimental_science/experimental_science_cleaner_focus_theo.png" width="750">
	<figcaption><strong>Situation of Chapter 5 in the methodological workflow.</strong></figcaption>
</figure>
<figure id='waddingtonian_timeline_timeline'>
	<img src="../../images/MECAGEN/waddingtonian_timeline/timeline.png" width="800">
	<figcaption><strong>Waddingtonian timeline of cell types \(\{\mathcal{T}(i, t)\}\).</strong> The grey bar on top is the time axis (oriented left to right), segmented into stages \(S = [t_S, t_{S+1})\). (The thin white time tick is the current time step of the simulation and the thick red time ticks mark backup &amp; restore points of the entire embryo state.) The colored horizontal branches symbolize the types that cells can potentially take, and correspond to the grooves of a Waddingtonian landscape (Fig. <a name='Development_Review_waddington_epigenetic_nature_mod_small_intro' class='fig'></a>). Here, we can see 5 stages in the timeline corresponding respectively to the onset of 1 type (red), 2 types (red and blue), ..., and 5 types (red, cyan, green, blue, yellow). Differentiations (type transitions) may happen during each stage, for example shown here in Stage 3: the three thin dashed arrows correspond to the differentiation table of Fig. <a name='waddingtonian_timeline_differentiation_aug' class='fig'></a> below, indicating that type red may become green, green may become red, and blue may become green. In this WTS graphical interface, stages can also be selected manually to specify their parameters (as will be shown in the next figures): here, Stage 5 is selected by clicking on \(t_5\), which is represented by gray circles on all the type nodes.</figcaption>
</figure>
<figure id='waddingtonian_timeline_differentiation_aug'>
	<img src="../../images/MECAGEN/waddingtonian_timeline/differentiation_aug.png" width="500">
	<figcaption><strong>Visualization of the differentiation (type-transition) operator \(\mathscr{D}\).</strong> (A) This particular table represents the possible transitions between cell types during Stage 3 above, thus can be denoted \(\mathscr{D}_3\). It contains three differentiation modules displayed in one column, one module by origin type. In each module, the current cell type \(\mathcal{T}(i, t)\) is represented by the frame color, the target cell type by the color (or id) of the bottom-right block, and the ligand-threshold-sign triplet \((Q_a, \theta_a, \epsilon_a)\) by the top-left block's color (or id), bottom-left block's value, and top-left block's value. Each cell changes its type depending on whether \(\epsilon_a q_{a,i} \geq \epsilon_a \theta_a\). (B) The generic template of one differentation module inside one origin type.</figcaption>
</figure>
<figure id='waddingtonian_timeline_sources_sinks'>
	<img src="../../images/MECAGEN/waddingtonian_timeline/sources_sinks.png" width="700">
	<figcaption><strong>Visualization of the ligand diffusion parameters \(\mathscr{L}\).</strong> This is the particular table at Stage 3, thus can be denoted \(\mathscr{L}_3\). Left column: Cells as sources. Each module corresponds to a current cell type (denoted by its frame color) and is composed of one or several pairs of boxes displayed vertically. The top box indicates the color (or id) of the secreted ligand \(Q_a\). The bottom box indicates the rate \(s_a\) of ligand secreted by each cell belonging to the cell type (defined in Section 4.2.1). Thus, red cell types 0 secrete both ligand types 2 (dark green) and 1 (purple), with rates respectively 2000 and 200. Right column: Cells as sinks. Each module only specifies the type (color or id) of the ligand that the cells can absorb, if any (red and green cell types absorb none). No particular quantity is specified as we assume here that all the ligand is absorbed by the cells acting as sinks.</figcaption>
</figure>
<figure id='waddingtonian_timeline_adhesion'>
	<img src="../../images/MECAGEN/waddingtonian_timeline/adhesion.png" width="500" height="450">
	<figcaption><strong>Visualization of the passive adhesion coefficients \(\mathscr{P}\).</strong> This is the particular table at Stage 5, thus can be denoted \(\mathscr{P}_5\). The matrix is symmetrical so only half of the coefficients are displayed. The outer frame color represents cell types by row, and the inner color cell types by column. Here, all values of \(w^{\mathrm{adh}}_{\mathcal{TT'}}\) are uniformly set to 1.00.
</figure>
<figure id='waddingtonian_timeline_protrusion'>
	<img src="../../images/MECAGEN/waddingtonian_timeline/protrusion2.png" width="550">
	<figcaption><strong>Visualization of the active protrusion coefficients \(\mathscr{A}\).</strong> This is the particular table at Stage 2, thus can be denoted \(\mathscr{A}_2\). (A) Two cell types are involved here: red and green. In each cell type's module, protrusion is represented by four parameters. Top-left: target cell type; top-right: the chosen polarization axis id \(d\); bottom-left: the intensity of the protruding force \(f^A\); bottom-right: the type of protrusion, either monopolar\(+\), monopolar\(-\) or bipolar. (B) The generic template of one active protrusion module inside one origin type.</figcaption>
</figure>
<figure id='Case_theo_sorting_nosource_explanation'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_nosource_explanation.png" width="180" height="200">
	<figcaption><strong>Experiment I passive adhesion table \(\mathscr{P}\).</strong> See definition in Section 5.2.3 and text for details. In this case, homotypic adhesion is strong (5.0), heterotypic adhesion is weak (1.0).</figcaption> 
</figure>
<figure id='Case_theo_sorting_nosource_protrusion'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_nosource_protrusion.png" width="700">
	<figcaption><strong>Experiment I active protrusion table \(\mathscr{A}\).</strong> See definition in Section 5.2.4 and text for details.</figcaption>
</figure>
<figure id='Case_theo_sorting_nosource_allmigr_DAH_image'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_nosource_allmigr_DAH.png" width="900">
	<figcaption><strong>Experiment I: Strong homotypic adhesion and random protrusion axis.</strong> See text for comments. The homotypic adhesion coefficient (\(w_{\mathrm{adh}} = 5\)) is higher than the heterotic coefficient (\(w_{\mathrm{adh}} = 1\)). All cells have bipolar protrusive activities with random polarization axes (mode (d) in Section 5.1). The rate of cluster size growth is decreasing over time. The time is provided through the number of simulation time steps.</figcaption>
</figure>
<figure id='Case_theo_sorting_nosource_allmigr_DAH'>
	<video src="../../videos/MECAGEN/Case_theo/sorting_nosource_allmigr_DAH.ogg" width="800" controls></video>
	<figcaption><strong>Experiment I: Strong homotypic adhesion and random protrusion axis.</strong> For caption, see Fig. <a name='Case_theo_sorting_nosource_allmigr_DAH_image' class='fig'></a>.</figcaption>
</figure>
<figure id='sorting_sourcecenter_explanation_sources_sinks_new'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_sourcecenter_explanation/sources_sinks_new.png" width="700">
	<figcaption><strong>Experiment II ligand diffusion table \(\mathscr{L}\).</strong> See definition in Section 5.2.2 and text for details. In summary, both cell types secrete both ligand types \(Q_1\) and \(Q_2\) with the same secretion rates \(s_1 = s_2 = 200\) simulation units. They do this, however, only if they enter one of the source regions on either vertical border of the domain.</figcaption>
</figure>
<figure id='Case_theo_sorting_source_adhesion'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_source_adhesion.png" width="180" height="200">
	<figcaption><strong>Experiment II passive adhesion table \(\mathscr{P}\).</strong> See definition in Section 5.2.3 and text for details.</figcaption>
</figure>
<figure id='Case_theo_sorting_sourceext_protrusion'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_sourceext_protrusion.png" width="700">
	<figcaption><strong>Experiment II active protrusion table \(\mathscr{A}\).</strong> See definition in Section 5.2.4 and text for details. In summary, protrusion behavior here is heterotypic, i.e. cells act only upon neighboring cells of the other type.</figcaption>
</figure>
<figure id='Case_theo_sorting_sourceext_extmigr_image'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_sourceext_extmigr.png" width="900">
	<figcaption><strong>Experiment II: Ligand-based heterotypic protrusion, planar diffusion sources.</strong> See text for comments. Two hidden ligands are diffusing from the left and right borders of the cell bilayer. Green cells' polarization axes are oriented toward the right source and red cells' axes toward the left source. Each cell type is exerting monopolar protrusion over the other cell type (heterotypic contacts). </figcaption>
</figure>
<figure id='Case_theo_sorting_sourceext_extmigr'>
	<video src="../../videos/MECAGEN/Case_theo/sorting_sourceext_extmigr.ogg" width="800" controls></video>
	<figcaption><strong>Experiment II: Ligand-based heterotypic protrusion, planar diffusion sources.</strong> For caption, see Fig. <a name='Case_theo_sorting_sourceext_extmigr_image' class='fig'></a>.</figcaption>
</figure>
<figure id='sorting_sourcecenter_explanation_sources_sinks_radial'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_sourcecenter_explanation/sources_sinks_radial.png" width="550">
	<figcaption><strong>Experiment III ligand diffusion table \(\mathscr{L}\).</strong> See definition in Section 5.2.2 and text for details. The differences with Experiment II's table \(\mathscr{L}\) is a unique source of ligand \(Q_1\) and a central region of secretion creating a (quasi) radial gradient.</figcaption>
</figure>
<figure id='Case_theo_sorting_sourcecenter_protrusion'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_sourcecenter_protrusion.png" width="700">
	<figcaption><strong>Experiment III active protrusion table \(\mathscr{A}\).</strong> See definition in Section 5.2.4 and text for details. The differences with Experiment II's table \(\mathscr{A}\) is that the polarization mode relies on only one type of ligand \(Q_1\) and the green cells' monopolar protrusion is oriented in the <q>negative</q> direction.</figcaption>
</figure>
<figure id='Case_theo_sorting_sourcecenter_extmigr_image'>
	<img src="../../images/Cases_Studies/Case_theo/sorting_sourcecenter_extmigr.png" width="900">
	<figcaption><strong>Experiment III: Ligand-based heterotypic protrusion, radial source.</strong> See text for comments. A single ligand is diffusing from the center of the cell bilayer. All cells' polarization axes are roughly radial, the red ones oriented toward the center (uphill along the gradient) and the green ones away from the center (downhill). As in Experiment II, each cell type is exerting monopolar protrusion over the other cell type (heterotypic contacts).</figcaption>
</figure>
<figure id='Case_theo_sorting_sourcecenter_extmigr'>
	<video src="../../videos/MECAGEN/Case_theo/sorting_sourcecenter_extmigr.ogg" width="800" controls></video>
	<figcaption><strong>Experiment III: Ligand-based heterotypic protrusion, radial source.</strong> For caption, see Fig. <a name='Case_theo_sorting_sourcecenter_extmigr_image' class='fig'></a>.</figcaption>
</figure>
<figure id='Case_theo_individual_migration2'>
	<img src="../../images/Cases_Studies/Case_theo/individual_migration2.png" width="500">
	<figcaption><strong>Experiment IV: individual cell migration.</strong> Left: Simulation involving a three-type cell population: a few cells secrete a ligand (the red color represents its thresholded concentration), the green cells are exerting monopolar protrusion over all their neighbors oriented uphill along the gradient, and the gray cells are inactive. Right: Microscope images of germ cells (green) migrating toward central cells expressing SDF-1a (red), which serves as a guidance cue. Images from the right column adapted from the E. Raz Lab's website <a href="http://zmbe.uni-muenster.de/institutes/izb/izbres.htm" target="_blank">http://zmbe.uni-muenster.de/institutes/izb/izbres.htm</a></figcaption>
</figure>
<figure id='Case_theo_migr_raz_simu'>
	<video src="../../videos/MECAGEN/migrRaz.ogg" width="800" controls></video>
	<figcaption><strong>Experiment IV: simulation of individual cell migration.</strong> For caption, see Fig. <a name='Case_theo_individual_migration2' class='fig'></a>, left column.</figcaption>
</figure>
<figure id='Case_theo_migr_raz_live'>
	<video src="../../videos/video_modeling/migrationRaz/sdf.ogg" width="500" controls></video>
	<figcaption><strong>Experiment IV: recording of individual cell migration.</strong> Microscope movie from the E. Raz Lab's website <a href="http://zmbe.uni-muenster.de/institutes/izb/izbres.htm" target="_blank">http://zmbe.uni-muenster.de/institutes/izb/izbres.htm</a>. For caption, see Fig. <a name='Case_theo_individual_migration2' class='fig'></a>, right column.</figcaption>
</figure>
<figure id='zebrafish_10h_review_kimmel_drawings_augmented'>
	<img src="../../images/zebrafish_10h_review/kimmel_drawings_augmented.png" width="600">
	<figcaption><strong>The zebrafish early developmental stages. </strong> Adapted from Kimmel <a name='Kimmel:1995kn' class='ref'></a>. From the 1-cell stage until the end of gastrulation, lateral views with animal pole to the top and dorsal side, identified by the shield stage, to the right. The enveloping layer (EVL) is in yellow. The deep cells (in blue) lead through gastrulation to the hypoblast (in purple) and the epiblast (in dark blue).  The whole spatio-temporal sequence is expected to last 10 hours at 28.5 degree celsius.</figcaption>
</figure>
<figure id='zebrafish_10h_review_krezel_1994'>
	<img src="../../images/zebrafish_10h_review/krezel_1994.png" width="600">
	<figcaption><strong>Schematic illustration of the changes in the organization of the cortical cytoplasm of the yolk cell in relation to other cell types in zebrafish embryo during epiboly in normal embryos.</strong> The organization of microtubule networks in the YSL and YCL observed in this study is indicated by thin lines. Only part of the blastoderm is shown to reveal the morphology of the yolk cell. The relative sizes of elements are not proportional. (A) The late blastula just before the onset of epiboly (sphere stage, 4.0 h). The blastoderm, composed of the internal deep cells and the superficial enveloping layer (EVL), is positioned atop of the syncytial yolk cell. The animal surface of the yolk cell underlying the blastoderm is flat. Most of the yolk syncytial nuclei (YSN) are in the external yolk syncytial layer (external YSL) vegetal to the blastoderm. The microtubules of the external YSL form a network. The organization of microtubules in the internal YSL at this stage of development is not known. The microtubules of the a nuclear yolk cytoplasmic layer (YCL) radiate from the organizing centers associated with the vegetal-most YSN and are aligned along the animal-vegetal axis. (B) 30 percent epiboly (4.7h). The blastoderm covers 30 percent of the yolk cell that bulged toward the animal pole taking on a dome shape. The external YSL has contracted and exhibits densely packed YSN and a dense network of microtubules. The external YSL is partially covered by the expanding vegetally blastoderm. (C) 50 percent epiboly (5.2h). The blastoderm arrives at 50 percent of the yolk cell latitude and covers almost completely the YSN which is also migrating vegetally and the YSL microtubule network. Only the YCL with its array of the animal-vegetal microtubules is visible vegetally to the blastoderm. (D) 60 percent epiboly (6.5h). Deep cells cover 60 percent of the yolk cell. The YSN nuclei are now visible in front of the blastoderm and lead the epibolic movement. The YSN are often stretched along the animal vegetal axis. The EVL rim is closer to the vegetal pole than the margin of deep cells. The YCL is diminished. Figure and caption from Solnica-Krezel and Driever (1994) <a name='SolnicaKrezel:1994wl' class='ref'></a> </figcaption>
</figure>
<figure id='zebrafish_10h_review_yuen_2012'>
	<img src="../../images/zebrafish_10h_review/yuen_2012.png" width="800">
	<figcaption><strong>Characterization of Ca(2+) signaling in the external yolk syncytial layer during the late blastula and early gastrula periods of zebrafish development. </strong> Timeline to show the major events that occur during: the appearance of the YSL; the division, distribution and movement of the E-YSN; the movement of the I-YSN; and the reported Ca2+ signals generated at or near to the blastoderm margin, during the blastula and early gastrula periods. New data regarding E-YSN clustering, changes in inter-nuclear distance in the E-YSL over time and the generation of E-YSL Ca2+ waves are also shown. MBT, midblastula transition; AP, animal pole; VP, vegetal pole. These features correlating events at different scales does not bring us closer however to an integrated model of zebrafish blastulation and gastrulation. But it indeed provides some hints in the ingredients that the model should take into account. Figure and caption from Yuen et al., 2012 <a name='Yuen:2012fr' class='ref'></a>
</figcaption>
</figure>
<figure id='videos_070927b'>
  <video src="../../videos/070927b.ogg" width="800" controls></video> 
  <figcaption><strong>Time-lapse sequence showing the tail bud closure at the vegetal pole.</strong> The left part of the video shows the projection of a slice delimited by the two planes visible on the right part. On the top left, internalization of the epiblast seems to occur, after the end of internalization described in the literature.</figcaption>
</figure>
<figure id='Wnt11_functions_in_gastrulation_by_controlling_cell_cohesion_through_Rab5c_and_E_cadherin_movieWT'>
  <video src="../../videos/Wnt11_functions_in_gastrulation_by_controlling_cell_cohesion_through_Rab5c_and_E_cadherin/movieWT.ogg" width="500" controls></video> 
  <figcaption><strong>Time-lapse sequence showing the movement of ectodermal precursor (epiblast) cells and mesendodermal precursor (hypoblast) cells in the shield region of wild-type embryos for 90 min at the onset of gastrulation.</strong> Video from <a name='Ulrich:2005jz' class='ref'></a></figcaption>
</figure>
<figure id='Zebra_Involution_Montero_2004'>
  <video src="../../videos/in_vivo/Zebra/Involution_Montero_2004.ogg" width="500" controls></video>
  <figcaption><strong>Time-lapse video of cell movements at the blastoderm margin in the shield region, from the shield stage.</strong> The video was recorded for a total of 123min at 2.5-min time intervals. Sagittal views. Frame size: 206\(\times\)206\(\mu\)m. Video from <a name='Montero:2004hh' class='ref'></a></figcaption>
</figure>
<figure id='6_Sdf1_Cxcr4_signaling_controls_the_dorsal_migration_of_endodermal_cells_during_zebrafish_gastrulation_movie1'>
  <video src="../../videos/Sdf1_Cxcr4_signaling_controls_the_dorsal_migration_of_endodermal_cells_during_zebrafish_gastrulation/movie1.ogg" width="800" controls></video> 
  <figcaption><strong>Time-lapse movie of migrating endodermal cells in a Tg(sox17:EGFP) transgenic embryo from 65 percent to 90 percent epiboly stages.</strong> Lateral views with dorsal to the right. Video and caption from <a name='Mizoguchi:2008di' class='ref'></a>.</figcaption>
</figure>
<figure id='experimental_science_experimental_science_cleaner_focus_reconstruction_live'>
	<img src="../../images/experimental_science/experimental_science_cleaner_focus_reconstruction_live.png" width="750">
	<figcaption><strong>Situation of Chapter 7 in the methodological workflow.</strong></figcaption>
</figure>
<figure id='bioemergences_BioemergencesWorkflow'>
	<img src="../../images/Reconstruction/bioemergences/BioemergencesWorkflow.png" width="700">
	<figcaption><strong>The Bioemergences platform.</strong></figcaption>
</figure>
<figure id='bioemergences_FigureWorkflow_Workflow2_lineage_only'>
	<img src="../../images/Reconstruction/bioemergences/FigureWorkflow/Workflow2_lineage_only_2.png" width="800">
	<figcaption><strong>The Bioemergences workflow.</strong> Raw images of nuclei and membranes (cyan layer in the diagram) are first filtered by an edge-preserving smoothing method. Cell positions are then extracted from the local maxima of this simplified images versions, and assimilated to the nuclei's positions. They are used to initialize both the segmentation and cell tracking tasks (green layers). The final output is composed of the cell lineage tree and the nuclei/membranes segmentation (red layer).</figcaption> 
</figure>
<figure id='segmentation_images_kanizsa_segmentation1'>
	<img src="../../images/Reconstruction/bioemergences/segmentation/images_kanizsa_segmentation1.png" width="700">
	<figcaption>
		<strong>Subjective Surfaces Method.</strong> Top and middle rows: 2D example summarizing the main steps of the segmentation algorithm, illustrated on the Kanizsa triangle (from left to right): original image depicting a triangle with missing <q>subjective</q> contours <a name='Sarti:2000jw' class='ref'></a>; construction of a surface located at the center of the triangle; evolution of the surface to minimize its area according to the model equation (two images); steady state condition giving a piecewise constant solution; choice of level set for the surface representing the contour of the triangle. Bottom: Segmentation of a membrane with missing boundary <a name='Zanella:2007vh' class='ref'></a>. Raw data displayed as an orthoslice in the x,y plane. Incomplete membrane contour highlighted with a red circle. Segmented membrane represented as an isosurface in blue. 
	</figcaption>
</figure>
<figure id='bioemergences_movie_1_5_fast'>
	<video src="../../videos/Reconstruction/bioemergences/movie_1_5_fast.ogg" width="900" controls></video>
	<figcaption>
		<strong>Cell tracking of dataset 070418a.</strong> Color code from yellow to blue through grey indicates the orientation of cell displacement (projection of the speed vector on the y axis). Yellow indicates migration from caudal to rostral, blue indicates migration from rostral to caudal, grey indicates null or mediolateral displacement. Each nucleus center is represented by a small cube with a streamline showing its trajectory for the next 12 time steps. Mov-IT Visualisation Platform. (Large movie, 73.3 MB).
	</figcaption>
</figure>
<figure id='bioemergences_movie_4_1_fatemap'>
	<video src="../../videos/Reconstruction/bioemergences/movie_4_1_fatemap.ogg" width="900" controls></video>
	<figcaption><strong>Cell trajectories displayed as streamlines and colormap according to the zebrafish fate map at somite stage.</strong> Dataset 070418a, reconstructed embryo from animal pole view. Colormap as in the following. Yellow, hypoblast including prechordal plate and notochord; red, presumptive hypothalamus; pink, right eye; light pink, right optic stalk; purple, left eye; light purple, left optic stalk; green, ventral telencephalon; gold, dorsal telencephalon, placodes and neural crest; light blue, ventral midbrain; light brown, dorsal midbrain and dorsal diencephalon; blue, ventral hindbrain; dark brown, dorsal hindbrain; dark blue, ears. Mov-IT Visualisation Platform.</figcaption>
</figure>
<figure id='movie_4_4_cellpop'>
	<video src="../../videos/Reconstruction/bioemergences/movie_4_4_cellpop.ogg" width="900" controls></video>
	<figcaption>Relative movements of cells selected in the prechordal plate (yellow), presumptive hypothalamus (red) and presumptive ventral telencephalon (green). Dataset 070418a, raw data displayed as orthoslice. Lateral view, rostral to the right. Mov-IT Visualisation Platform.</figcaption>
</figure>
<figure id='these_lombardot_p166_velocity_map'>
 	<img src="../../images/Reconstruction/these_lombardot/p166_velocity_map.png" width="700">
 	<figcaption>Evolution of the averaged speed field with its norm between 4 and 12 hours of development in zebrafish embryo. Dataset 080322a, lateral view. Green axis points towards the dorsal side, red axis towards the animal pole. Colormap between light blue and red related to the norm of the averaged speed field. Singular points in red, separatrices in blue, flow pathways in green. Image and caption adapted from B. Lombardot (2010) <a name='Lombardot:2010vd' class='ref'></a>.</figcaption>
</figure>
<figure id='these_lombardot_p172_velocity_map_drawings'>
 	<img src="../../images/Reconstruction/these_lombardot/p172_velocity_map_drawings.png" width="700">
 	<figcaption>Simplified topology of the speed field in zebrafish embryo between 4 and 12 hours of development. Circles in each figure represent a schematization of the embryo in a lateral view. Dorsal side on the right, ventral on the left. Animal pole on the top, vegetal pole on the bottom. Separatrices represented as red lines, singularities as red dots. Image and caption adapted from B. Lombardot (2010) <a name='Lombardot:2010vd' class='ref'></a>.</figcaption>
</figure>
<figure id='washington_workflow'>
	<img src="../../images/Reconstruction/washington/workflow_corrected.png" width="900">
	<figcaption><strong>Global diagram of the reconstruction workflow of the <i>in toto</i> deformation fields. </strong> </figcaption>
</figure>
<figure id='washington_5angles_before'>
	<img src="../../images/Reconstruction/washington/5angles_before_letter.png" width="900">
	<figcaption><strong>Original five views produced by the Digital Scanning Light Microscope (DSLM).</strong> Isosurfaces have been extracted from the DSLM raw data. Each view is obtained by rotating the embryo 72 degrees (one color per angle of view). A: Sphere stage. B: 40 percent epiboly. C: 75 percent epiboly. D: 90 percent epiboly. E: 8-somite stage. F: 12-somite stage.</figcaption>
</figure>
<figure id='washington_workflow_manual'>
	<img src="../../images/Reconstruction/washington/workflow_manual_corrected.png" width="600">
	<figcaption><strong>The manual registration component.</strong> This process aims at inferring the parameters required for a spatial registration of the five angles of view.</figcaption>
</figure>
<figure id='washington_SPIM'>
	<img src="../../images/Reconstruction/washington/SPIM_corrected.png" width="900">
	<figcaption><strong>Digital Scanning Light Microscope (DSLM) setup.</strong> Diagram after P.J. Keller, E. Stelzer et al. <a name='Keller:2008km' class='ref'></a>, adapted by V. Gurchenkov.</figcaption>
</figure>
<figure id='washington_manual_spim'>
	<img src="../../images/Reconstruction/washington/manual_spim.png" width="900">
	<figcaption><strong>Schema of the calibration phase. </strong> Left: The operator centers the embryo in the light plane. Right: To decipher the relative position of the embryo with respect to the center of rotation, the software performs a 90 degree rotation, then the operator must translate the embryo back into the light plane (blue vector). </figcaption>
</figure>
<figure id='washington_manual_registration_selection'>
	<img src="../../images/Reconstruction/washington/manual_registration_selection.png" width="900">
	<figcaption><strong>Graphical user interface designed to infer the estimation of the center of rotation.</strong> The red line passes through the center of rotation determined by the calibration phase. The brown line passes through the corrected center of rotation. As the user moves the brown line, the five angles of views are repositioned in real time, allowing to infer the best estimation of the center of rotation.</figcaption>
</figure>
<figure id='washington_manual_registration_final'>
	<img src="../../images/Reconstruction/washington/manual_registration_final.png" width="900">
	<figcaption><strong>Best match for the rigid manual registration.</strong> The marginal yolk nuclei visible on top of the embryo are used to estimate the correct registration parameters.</figcaption>
</figure>
<figure id='washington_5angles_after'>
	<img src="../../images/Reconstruction/washington/5angles_after_letter.png" width="900">
	<figcaption><strong>The five DSLM views superimposed by rigid registration.</strong> These views are identical to the ones shown in Fig. <a name='washington_5angles_before' class='fig'></a>. A: Sphere stage. B: 40 percent epiboly. C: 75 percent epiboly. D: 90 percent epiboly. E: 8-somite stage. F: 12-somite stage.</figcaption>
</figure>
<figure id='washington_workflow_vqe'>
	<img src="../../images/Reconstruction/washington/workflow_vqe_corrected.png" width="600">
	<figcaption><strong>The voxel quality component.</strong> This process aims at quantitatively evaluating the local image quality for each angle of view.</figcaption>
</figure>
<figure id='washington_vqe'>
	<img src="../../images/Reconstruction/washington/vqe_letter.png" width="900">
	<figcaption><strong>The various steps of the voxel quality evaluation component.</strong> See text.</figcaption>
</figure>
<figure id='washington_workflow_blend'>
	<img src="../../images/Reconstruction/washington/workflow_blend_corrected.png" width="600">
	<figcaption><strong>The blending function component.</strong> This component builds a function determining what original angle of view has the best quality score for each voxel in the registered space.</figcaption>
</figure>
<figure id='Reconstruction_5angles_3Dvolume'>
	<img src="../../images/Reconstruction/5angles_fusion.png" width="900">
	<figcaption><strong>Relative positions of the five views in the registered space.</strong> Left: The five cubes represent the original reference frames of the five different views. Right: A 2D illustration of the sequential visit of the registered voxels.</figcaption>
</figure>
<figure id='washington_blendingfunction'>
	<img src="../../images/Reconstruction/washington/blendingfunction.png" width="900">
	<figcaption><strong>Five-valued blending function mask.</strong> In the registered space, the color of each voxel represents the id of the original angle of view under which it has the best quality.</figcaption>
</figure>
<figure id='washington_blendingfunction_raw'>
	<img src="../../images/Reconstruction/washington/blendingfunction_raw.png" width="900">
	<figcaption><strong>Output of the blending function applied to the raw voxel intensities.</strong> </figcaption>
</figure>
<figure id='washington_workflow_deffield'>
	<img src="../../images/Reconstruction/washington/workflow_deffield_corrected.png" width="600">
	<figcaption><strong>The deformation field component.</strong> This component produces a 3D vector field that indicates the direction and amplitude of local displacements in each voxel.</figcaption>
</figure>
<figure id='washington_vueGlobal_frame'>
	<img src="../../images/Reconstruction/washington/deffield/vueGlobal_frame.png" width="900">
	<figcaption><strong>One volume V under one of the five angles of view.</strong> The white grains represent the envelopes of the nuclei calculated by thresholding the voxels at twice the mean intensity of V.</figcaption>
</figure>
<figure id='washington_vueDeffield2ts'>
	<img src="../../images/Reconstruction/washington/deffield/vueDeffield2ts.png" width="900">
	<figcaption><strong>Deformation field between two volumes V and V' at consecutive time steps.</strong> The white isosurface represents V, the yellow isosurface represents V'. Not all deformation arrows are displayed.</figcaption>
</figure>
<figure id='washington_workflow_blend_deffield'>
	<img src="../../images/Reconstruction/washington/workflow_blend_deffield_corrected.png" width="600">
	<figcaption><strong>The blended deformation field component.</strong> This component combines the blending function with the 3D deformation vector field.</figcaption>
</figure>
<figure id='washington_blendingfunction_deffield'>
	<img src="../../images/Reconstruction/washington/blendingfunction_deffield.png" width="750">
	<figcaption><strong>Blended deformation field at a given time step viewed from four different angles.</strong> (Note that these angles do not correspond to the original angles of view.)</figcaption>
</figure>
<figure id='washington_blendingfunction_deffield_multits'>
	<img src="../../images/Reconstruction/washington/blendingfunction_deffield_multits.png" width="750">
	<figcaption><strong>Blended deformation field at successive time steps viewed from the same angle.</strong> </figcaption>
</figure>
<figure id='experimental_science_experimental_science_cleaner_chap8'>
	<img src="../../images/experimental_science/experimental_science_cleaner.png" width="950">
	<figcaption><strong>Chapter 8 involves all the components of the methodological workflow introduced in Section 1.1.3.</strong></figcaption>
</figure>
<figure id='Case_0_Yolk_THG_thg'>
  <img src="../../images/Cases_Studies/Case_0_Yolk/THG/thg.png" width="900">
	<figcaption><strong><i>Third harmonic generation</i> (THG) imaging of the zebrafish embryo adapted from <a name='Olivier:2010jz' class='ref'></a></strong>. Snapshots from the movie <a name='Case_0_Yolk_1_THG_imaging_the_zebrafish_embryo_from_the_one_cell_stage' class='fig'></a> going from the one-cell stage to the 64-cell stage. Developmental timing (min) indicated top left. Scale bar (100 microns) indicated bottom left.</figcaption>
</figure>
<figure id='Case_0_Yolk_general_particles_yolk'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/general_particles_yolk.png" width="600">
	<figcaption><strong>Abstract schema of the particle types in the modeled embryo.</strong> Green particles represent the cells, yellow particles the external <q>yolk membrane</q> (ym), and red particles the inner lipid droplets or <q>yolk interior</q> (yi).</figcaption>
</figure>
<figure id='Case_0_Yolk_erratic_topology'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/erratic_topology.png" width="900">
	<figcaption><strong>Yolk membrane behavior depending on its particles' topological interactions.</strong> A: rank-1 interactions only. B: rank-1 + rank-2 interactions. </figcaption>
</figure>
<figure id='Case_0_Yolk_blender_all_gl_zoom'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/blender/all_gl_zoom_letter.png" width="900">
	<figcaption><strong>The structural topology of the yolk membrane is a geodesic dome.</strong> A: Icosahedron. B,C,D,E: iterative subdivision of the original icosahedron to obtain the geodesic dome. F: the right vertex is a regular geodesic vertex and the left vertex is one of the 12 original icosahedral vertices. Rank-1 neighbors link in red, rank-2 neighbors link in yellow.</figcaption>
</figure>
<figure id='121120_121120_raw_lateral_embryo1'>
	<video src="../../videos/MECAGEN/Case_0_Yolk/121120/121120_raw_lateral_embryo1.ogg" width="800" controls></video>
	<figcaption><strong>Experimental deformation of the egg yolk.</strong></figcaption>
</figure>
<figure id='121120_121120_raw_embryo1_m'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/121120/121120_raw_embryo1_m.png" width="950">
	<figcaption><strong>Experimental deformation of the egg yolk.</strong> 1-cell stage, lateral view, animal pole to the right. \(t_0\): Onset of the deformation. \(t_1\): Maximum deformation. Measurement is realized every 0.8 second. We measure the distance \(L\) between the area pointed by the tip of the bead and the opposite side (black line). Scalebar 500 microns. </figcaption>
</figure>
<figure id='Case_0_Yolk_plot_normalized'>
		<img src="../../images/Cases_Studies/Case_0_Yolk/plot_normalized_def.png"  width="800">
		<figcaption><strong>Plot of the normalized deformation length as a function of time.</strong> The shape recovery is asymptotic toward the normalized length at \(t_0\). Full recovery is nearly achieved by 8 seconds.</figcaption>
</figure>
<figure id='Case_0_Yolk_simulation_example'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/simulation_example2.png" width="700">
	<figcaption><strong>Snapshots of the simulated deformation and recovery.</strong> First and third rows: entire yolk from an external point of view. Second and fourth rows: another yolk sliced to reveal the yolk interior particles. A,B,C,D,E,F: compression phase. G,H,I,J,K,L: relaxation phase.  Note that these simulated experiments have been realized by manually operating the spherical bead, whereas in the study the manipulation is automated. Original movies are available in Figs. <a name='Case_0_Yolk_yolk_external' class='fig'></a> <a name='Case_0_Yolk_yolk_relaxation_cut' class='fig'></a>  </figcaption>
</figure>
<figure id='parameter_space_table_parameter_space_table'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/parameter_space_table/parameter_space_table.png" width="500">
	<figcaption><strong>Ranges, cardinalities and units of the four parameters explored in this study.</strong></figcaption>
</figure>
<figure id='Case_0_Yolk_all_curves_plot_fitness_ok'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/all_curves_plot_fitness_ok.png" width="800">
	<figcaption><strong>Plots of the simulated normalized deformation length as a function of time.</strong> The black thick curve represents the experimental live measurement. The simulated measurements are colored according to their fitness value (from red for the best fitness \(F = 0.1609\) to blue for the worst \(F = 7.1449\)). In ordinate, the normalized length of the yolk diameter is plotted. In abscissa, the time from the moment the bead is pulled away from the yolk surface.</figcaption>
</figure>
<figure id='4D_single_3D_plot'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/4D/single_3D_plot.png" width="800">
	<figcaption><strong>An example of 3D plot of the fitness landscape in parameter space.</strong> This figure aims at presenting the coordinate system used in Fig. <a name='4D_fusion_vertical_A' class='fig'></a> and Fig. <a name='4D_fusion_vertical_B' class='fig'></a>. Here, the repulsion coefficient \(\overline{w}^{\mathrm{y}}_{\mathrm{rep}}\) of the force exerted between yolk interior particles has a fixed value \(3.7\) \(10^{-6}\). The vertical axis is the coefficient \(c_r\), which determines the equilibrium surface of the yolk membrane. The depth axis is the adhesion coefficient, \(\overline{w}^{\mathrm{y}}_{\mathrm{adh}}\) which modulates the cohesion of the yolk interior particles. The width axis is the yolk membrane stiffness coefficient \(\overline{k}_{\mathrm{ym}}\). It should be noted that the values plotted on the depth (resp. width) axis actually correspond to \(w^{\mathrm{y}}_{\mathrm{adh}}\) (resp. \( k_{\mathrm{ym}}\)) and should be divided by \(\lambda_0\) to match the values of the table presented in Fig. <a name='parameter_space_table_parameter_space_table' class='fig'></a>.
	</figcaption>
</figure>
<figure id='4D_fusion_vertical_A'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/4D/fusion_vertical_A.png" width="700">
	<figcaption><strong>Fitness landscape of the yolk relaxation experiment.</strong> 
	The left column represents the fitness landscape in parameter space. The middle column shows the normalized yolk volume difference between the simulated phenotype and the targeted volume of radius \(R_{\mathrm{Y}}\). The right column is identical to the left one but keeping only the phenotypes that have a yolk volume less than 20 percent different from the targeted one. All the coordinates are described in Fig. <a name='4D_single_3D_plot' class='fig'></a>. Each row corresponds to a different value of the coefficient of repulsion between yolk interior particles \( \overline{w}^{\mathrm{y}}_{\mathrm{rep}} \). The value of \( \overline{w}^{\mathrm{y}}_{\mathrm{rep}} \) varies regularly between \(3.7\) \(10^{-6}\) for the top row to \(5.55\) \(10^{-5}\) for the bottom row.</figcaption>
</figure>
<figure id='4D_fusion_vertical_B'>
	<img src="../../images/Cases_Studies/Case_0_Yolk/4D/fusion_vertical_B.png" width="700">
	<figcaption><strong>Fitness landscape of the yolk relaxation experiment.</strong> For caption, see Fig. <a name='4D_fusion_vertical_A' class='fig'></a>.</figcaption>
</figure>
<figure id='Case_1_Division_Olivieretal'>
	<img src="../../images/Cases_Studies/Case_1_Division/Olivieretal_small.png" width="400">
	<figcaption><strong>Mean cell cycle duration (green) compared with cycle duration at the beginning (blue) and the end (red) of the division cycle, averaged over six embryos.</strong> Image and caption adapted from <a name='Olivier:2010jz' class='ref'></a>.</figcaption>
</figure>
<figure id='Case_1_Division_Olivieretal2'>
	<img src="../../images/Cases_Studies/Case_1_Division/Olivieretal2.png" width="400">
	<figcaption><strong>Scatter plot of mitosis time as a function of distance to the pseudo-wave origin.</strong> Image and caption adapted from <a name='Olivier:2010jz' class='ref'></a>.</figcaption>
</figure>
<figure id='kane_kimmel_1993_cell_cycle'>
	<img src="../../images/Cases_Studies/Case_1_Division/kane_kimmel_1993_cell_cycle.png" width="400">
	<figcaption><strong>Cell cycle lengthening and loss of synchrony at the midblastula transition.</strong> Adapted from <a name='Kane:1993wp' class='ref'></a>.</figcaption>
</figure>
<figure id='kane_kimmel_1993_cell_cycle_3'>
	<img src="../../images/Cases_Studies/Case_1_Division/kane_kimmel_1993_cell_cycle_3.png" width="800">
	<figcaption><strong>Correlation of cell volume and cell cycle length for cycles 10 to 13. </strong> Adapted from <a name='Kane:1993wp' class='ref'></a>.</figcaption>
</figure>
<figure id='kane_kimmel_1993_cell_cycle_2'>
	<img src="../../images/Cases_Studies/Case_1_Division/kane_kimmel_1993_cell_cycle_2.png" width="400">
	<figcaption><strong>Median cell cycle lengths during blastula stages.</strong> Adapted from <a name='Kane:1993wp' class='ref'></a>.</figcaption>
</figure>
<figure id='cell_number_plot_cell_number_plot'>
	<img src="../../images/Cases_Studies/Case_1_Division/cell_number_plot/cell_number_plot.png" width="800">
	<figcaption><strong>Theoretical cell number as a function of time of development in the geometric progression scenario.</strong> In ordinate, the number of simulated cells. In abscissa, time of development starting at 2h39. Three geometric progression scenario are plotted. Each of them starts from an initial simulated cell population comprising 512 cells with identical cell cycle length (see text), and identical common ratio (\(r_0 = 1.5\)). In red, the asynchrony is zero, all cells divide simultaneously at each cycle (\(w_r = 0\)). In yellow, cell cycles are desynchronizing reasonably at each cycle (\(w_r = 0.2\)), plateaus are still occurring in between mitosis cycles. In blue, a larger desynchronization (\(w_r = 0.4\)) leads to a progressive disappearance of the plateau as the cell population increases. </figcaption>
</figure>
<figure id='Case_1_Division_071222bF_vue_AP'>
	<video src="../../videos/MECAGEN/Case_1_Division/071222bF_vue_AP.ogg" width="800" controls></video>
	<figcaption><strong>Time-lapse movie of a developing zebrafish embryo.</strong> The embryo is imaged from the animal pole (slightly ventral). The movie starts at the 32-cell stage (Data set ID 071222bF, starting at 1h45min hpf) during the cleavage stages and goes on during the blastula and gastrula periods. By the end, only one layer of non-neural ectoderm remains between the yolk and the enveloping layer on the ventral side of the embryo. We need here the characteristics of the data, staining, voxel size timing.</figcaption>
</figure>
<figure id='Case_1_Division_071222bF_t_t2005_259_trimmed'>
	<video src="../../videos/MECAGEN/Case_1_Division/071222bF_t_t2005_259_trimmed.ogg" width="800" controls></video>
	<figcaption><strong>Manual tracking of a cell population through the Mov-It interface.</strong> The embryo shown in Fig. <a name='Case_1_Division_071222bF_vue_AP' class='fig'></a> has been processed through the Bioemergences workflow. The yellow cubes correspond to the cells that have been manually corrected (position and tracking).</figcaption>
</figure>
<figure id='Case_1_Division_071222bF_t_selection_1_all_T_121224_lineage_tree_no_evl'>
	<img src="../../images/Cases_Studies/Case_1_Division/071222bF_t_selection_1_all_T_121224_lineage_tree_no_evl.png" width="900">
	<figcaption><strong>Lineage tree of the selected cell population.</strong> The selected population consists in eight cells picked at the time of cell cycle 9. At cycle 9 they divide quasi-synchronously (one cell out of the eight divides one time step, i.e. 3 min, after the others). The selected cells contribute to the formation of the EVL. As this differentiated tissue is expected to have a specific cell cycle dynamics (see Case Study 8.4), EVL cells were removed from the selected cell population after division cycle 11. EVL cell lineage removed from the analyzed data is indicated by the grey branches. Cell cycle 11 occurs at the onset of the epiboly at the time of doming, prior to what is usually considered as gastrulation (6 hpf). The selected cells tracking was checked and corrected as far as possible in time.</figcaption>
</figure>
<figure id='Case_1_Division_071222bF_t_selection_1_all_T_121224_cellcyclelength_per_generation'>
	<img src="../../images/Cases_Studies/Case_1_Division/071222bF_t_selection_1_all_T_121224_cellcyclelength_per_generation.png" width="900">
	<figcaption><strong>Plot of the cell cycle length for cells selected among the deep cells, as a function of the timing of the divisions.</strong> Colors and numbers represent the iteration of the cell cycle as indicated on the right. A cell is plotted at the time of a division as a function of the timing of its mother's division. Timing is indicated in time steps. Data manually validated by Ingrid Colin with Mov-IT.</figcaption>
</figure>
<figure id='Case_1_Division_071222bF_t_t2005_672_generation_DC_only_trimmed'>
	<video src="../../videos/MECAGEN/Case_1_Division/071222bF_t_t2005_672_generation_DC_only_trimmed.ogg" width="800" controls></video>
	<figcaption><strong>Selected cell population from the eight original clones.</strong> The color marks the current generation of the cell from 9 (blue) to 15 (red).</figcaption>
</figure>
<figure id='071222bF_t_selection_1_all_T_121224_cellcyclelength_per_generation_MATLAB_mean'>
	<img src="../../images/Cases_Studies/Case_1_Division/071222bF_t_selection_1_all_T_121224_cellcyclelength_per_generation_MATLAB_mean.png" width="950">
	<figcaption><strong>Plot of the cell cycle length as a function of the cell cycle time for the cell population selected in the digital specimen.</strong> The blue color represents the \(10^{th}\) cell cycle, light blue, the \(11^{th}\), green, the \(12^{th}\), yellow, the \(13^{th}\), and orange, the \(15^{th}\). Small dots correspond to single cells at the time of mitosis and large dots are the mean of the division timing for the corresponding cell cycle. The data is the same as Fig. <a name='Case_1_Division_071222bF_t_selection_1_all_T_121224_cellcyclelength_per_generation' class='fig'></a> without cell cycle 15 and after elimination of the three outliers discussed above. This data is used for confronting live experimental and simulated data.</figcaption>
</figure>
<figure id='071222bF_t_selection_1_all_T_121224_cellcyclelength_per_generation_MATLAB_simu_arrow'>
	<img src="../../images/Cases_Studies/Case_1_Division/071222bF_t_selection_1_all_T_121224_cellcyclelength_per_generation_MATLAB_simu_arrow.png" width="950">
	<figcaption><strong>Plot of the cell cycle length as a function of the the cell cycle time for two simulated embryonic developments.</strong> In addition to the live cells plot, we provide two simulated distributions, one corresponding to a geometric progression, indicated by arrowheads (\(r_0 = 1.3\) and \(w_r = 0.4\), and one corresponding to an arithmetic progression, indicated by arrows (\(d_0 = 30 \mathrm{min}\) and \(w_d = 14 \mathrm{min}\)). For the sake of visualization, we picked here parameters that obviously did not provide the best fitness but kept the clouds separated.</figcaption>
</figure>
<figure id='Case_1_Division_parameter_space_table_parameter_space_table'>
	<img src="../../images/Cases_Studies/Case_1_Division/parameter_space_table/parameter_space_table.png" width="500">
	<figcaption><strong>Ranges, cardinalities and units of the four parameters explored in this study.</strong></figcaption>
</figure>
<figure id='071222bF_t_selection_1_all_T_121224_fitness_geo_ok'>
	<img src="../../images/Cases_Studies/Case_1_Division/071222bF_t_selection_1_all_T_121224_fitness_geo_ok.png" width="950">
	<figcaption><strong>Fitness landscapes of the <i>geometric</i> progression rule in parameter space.</strong> The left plot is the global fitness landscape. In the right part of the figure: the top left plot is the landscape of the mean of the cell cycle time's normalized objective function \(\overline{F^{\mathrm{geo}}_{\mu}}(T_{\mathrm{cc}})\), the top right plot is the landscape of the standard deviation of the cell cycle time's normalized objective function \(\overline{F^{\mathrm{geo}}_{\sigma}}(T_{\mathrm{cc}})\), the bottom left plot is the landscape of the mean of the cell cycle length normalized objective function \(\overline{F^{\mathrm{geo}}_{\mu}}(L_{\mathrm{cc}})\), the bottom right plot is the landscape of the standard deviation of the cell cycle length normalized objective function \(\overline{F^{\mathrm{geo}}_{\sigma}}(L_{\mathrm{cc}})\). The color map of all five plots follows the same color bar presented on the left. Each plot provides the cell cycle distribution center \(r_0\) on the abscissa axis and the distribution range coefficient \(w_r\) on the ordinate axis. Both parameters are explored in the range presented in Fig. <a name='Case_1_Division_parameter_space_table_parameter_space_table' class='fig'></a></figcaption>
</figure>
<figure id='071222bF_t_selection_1_all_T_121224_fitness_ari_ok'>
	<img src="../../images/Cases_Studies/Case_1_Division/071222bF_t_selection_1_all_T_121224_fitness_ari_ok.png" width="950">
	<figcaption><strong>Fitness landscapes of the <i>arithmetic</i> progression rule in parameter space.</strong> The left plot is the global fitness landscape. In the right part of the figure: the top left plot is the landscape of the mean of the cell cycle time's normalized objective function \(\overline{F^{\mathrm{ari}}_{\mu}}(T_{\mathrm{cc}})\), the top right plot is the landscape of the standard deviation of the cell cycle time's normalized objective function \(\overline{F^{\mathrm{ari}}_{\sigma}}(T_{\mathrm{cc}})\), the bottom left plot is the landscape of the mean of the cell cycle length normalized objective function \(\overline{F^{\mathrm{ari}}_{\mu}}(L_{\mathrm{cc}})\), the bottom right plot is the landscape of the standard deviation of the cell cycle length normalized objective function \(\overline{F^{\mathrm{ari}}_{\sigma}}(L_{\mathrm{cc}})\). The color map of all five plots follows the same color bar presented on the left. Each plot provides the cell cycle distribution center \(d_0\) on the abscissa axis and the distribution range coefficient \(w_d\) on the ordinate axis. Both parameters are explored in the range presented in Fig. <a name='Case_1_Division_parameter_space_table_parameter_space_table' class='fig'></a></figcaption>
</figure>
<figure id='071222bF_t_selection_1_all_T_121224_best_fitness_geo_c1_1_5_c2_0_4'>
	<img src="../../images/Cases_Studies/Case_1_Division/071222bF_t_selection_1_all_T_121224_best_fitness_geo_c1_1_5_c2_0_4.png" width="950">
	<figcaption><strong>Plot of the cell cycle length \(L_{\mathrm{cc}}\) as a function of the cell cycle time \(T_{\mathrm{cc}}\) for the simulated embryonic development with the best set of parameters for the <i>geometric</i> progression rule.</strong> This phenotype has also the best fitness of this study (\(r_0 = 1.5\), \(w_r = 0.4\)).</figcaption>
</figure>
<figure id='071222bF_t_selection_1_all_T_121224_best_fitness_ari_c1_140_c2_180'>
	<img src="../../images/Cases_Studies/Case_1_Division/071222bF_t_selection_1_all_T_121224_best_fitness_ari_c1_140_c2_180.png" width="950">
	<figcaption><strong>Plot of the cell cycle length \(L_{\mathrm{cc}}\) as a function of the cell cycle time \(T_{\mathrm{cc}}\) for the simulated cells generated with the best set of parameters for the <i>arithmetic</i> progression rule.</strong> This phenotype corresponds to the following parameters: \(d_0 = 14\,\mathrm{min}\), \(w_d = 18\,\mathrm{min}\).</figcaption></figcaption>
</figure>
<figure id='Case_2_Cleavage_kimmel_1995'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/kimmel_1995.png" width="950">
	<figcaption><strong>Cleavage and blastula period.</strong> A-F: Cleavage period. G-J: Blastula period. All face views except for B, which shows the embryo twisted about the animal-vegetal axis, roughly 45 degrees from the face view. A: 2-cell stage (0.75 h). B: 4-cell stage (1 h). C. 8-cell stage (1.25 h). D: 16-cell stage (1.5 h). E: 32-cell stage (1.75 h). F. 64-cell stage (2 h). G: 256-cell stage (2.5 h). H: high stage (3.3 h). I. transition between the high and oblong stages (3.5 h). J. transition between the oblong and sphere stages (3.8 h). Scale bar in F.: 250 \(\mu\)m. Image and caption adapted from Kimmel et al. (1995) <a name='Kimmel:1995kn' class='ref'></a></figcaption>
</figure>
<figure id='Case_2_Cleavage_interaction_table'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/interaction_table.png" width="950">
	<figcaption><strong>Complete abstract schema of the different particle types involved in the modeled embryo and their interactions.</strong> Red circles represent the deep cells (DC), yellow squares the yolk membrane (YM) particles, pale yellow triangles the yolk interior (YI) particles, and blue crosses the EVL cells. A: The two bigger orange squares represent YM particles that are labeled <q>margin</q> as the EVL is attached to the yolk in this position. B: Table of interactions by particles types. Each box indicates the section of the manuscript where a description of the interaction is available. Some interactions are not specified (--).</figcaption>
</figure>
<figure id='Case_2_Cleavage_simulated_embryo_stages'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/cleavages_1_16.png" width="900">
	<figcaption><strong>Division orientation in the model from cycle 1 to cycle 4.</strong> DC particles are represented by red cubes, YI in yellow, marginal YM in dark orange. Only the YM located above the margin are shown, the rest of the yolk membrane is represented by an orange surface. Top row: external view of the simulated embryo. The orthogonal axes \(A_1\) and \(A_2\) specify the axes of the division, which alternate during the first 4 cycles (red indicates the last axis). Bottom row: sagittal section of the 3D embryo rendering. A: zygote stage. B: 2-cell stage. C: 4-cell stage. D: 8-cell stage. E: 16-cell stage. A dynamical rendering is available in Movie <a name='Case_2_Cleavage_cleavages' class='fig'></a>. </figcaption>
</figure>
<figure id='Case_2_Cleavage_cleavages'>
	<video src="../../videos/MECAGEN/Case_2_Cleavage/cleavages.ogg" width="800" controls></video>
	<figcaption><strong>Division orientation in the model.</strong> For caption, see Fig. <a name='Case_2_Cleavage_simulated_embryo_stages' class='fig'></a>. <figcaption>
</figure>
<figure id='evl_separation_071222bF_071222bF_t3X_multi_cut'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/evl_separation_071222bF/071222bF_t3X_multi_cut.png" width="800">
	<figcaption><strong>Exceptional (and last) radial cell divisions in the EVL at the cycle 11 (1024 to 2048 cells).</strong> These events are not taken into account in the present model, and shown here only as complementary information. Raw image from a live specimen imaged at the oblong stage (3.65 hpf). Insets on the right focus on one particular radial cell division, which spawns a daughter cell into the DC population and keeps the other in the EVL compartment. However, the great majority of cell divisions happening at the same time in the EVL (but not visible here) are oriented tangentially to the embryo surface.</figcaption>
</figure>
<figure id='Case_2_Cleavage_evl_neighb'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/evl_neighb.png" width="700">
	<figcaption><strong>Typical EVL neighborhoods.</strong> A: Superposition of the metric neighborhood \(\mathcal{N}^{m, \mathrm{E}}_i\) (purple edges) and topological neighborhood \(\mathcal{N}^{t, \mathrm{E}}_i\) (red edges) on each EVL cell. B: The outward-pointing normal vectors \(\vec{U}_{\mathrm{E}, i}\) (blue lines) are obtained by averaging the normal vectors all the triangles composing the topological neighborhood of cell \(i\). </figcaption>
</figure>
<figure id='Case_2_Cleavage_evl_neighb_mov'>
	<video src="../../videos/MECAGEN/Case_2_Cleavage/evl_neighb.ogg" width="800" controls></video>
	<figcaption><strong>Dynamical view of the EVL neighborhoods.</strong> For caption, see Fig. <a name='Case_2_Cleavage_evl_neighb' class='fig'></a>A.<figcaption>
</figure>
<figure id='Case_2_Cleavage_evl_dc_neighb'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/evl_dc_neighb.png" width="950">
	<figcaption><strong>Spatial neighborhood between the EVL cells and the deep cells.</strong> This view is a 2D slice (not a 3D perspective) perpendicular to the EVL. It illustrates a late developmental stage to emphasize the size disparity between EVL cells and deep cells. Three EVL cells are displayed with their topological EVL-EVL neighborhood links (red edges). All the red cubes below are the deep cells. Topologically, we distinguish three types of cells: Below the dashed lines are the cells too distant from the EVL. Above the dashed line are the DC particles that belong to the metric neighborhoods of EVL cells \(\mathcal{N}^{m, \mathrm{E:D}}_j\). The cells surrounded by a green box are the ones that are eliminated by the topological rule: each one contains in its regular DC-DC metric neighborhood \(\mathcal{N}^m_i\) a neighbor \(k\) <q>higher above</q> in the normal direction. The top layer of deep cells, are the final topological neighbors of the EVL cells. The light purple lines represent the topological neighborhoods and the beige lines the metric neighborhoods.</figcaption>
</figure>
<figure id='Case_2_Cleavage_margin_attachment'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/yolk_neigh_evl_conc.png" width="750">
	<figcaption><strong>Region of attachment of the EVL to the yolk, or <q>margin</q>. </strong> A: Metric (thin purple lines) and topological (fat pink lines) neighborhoods between EVL cells and the MYM particles. B: Concentric yolk marginal circles. From the vegetal pole (bottom) to the animal pole (top) of the yolk, YM particles are classified into 25 concentric latitudes, which are displayed here with a color map from red to blue. The pink lines on top are the same as in A, but the EVL cells are hidden.</figcaption>
</figure>
<figure id='Case_2_Cleavage_yolk_normal_tangent'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/yolk_normal_tangent.png" width="700">
	<figcaption><strong>Yolk membrane particles' normal and tangential vectors.</strong> At each time step, each YM particle \(j\) updates its outward-pointing normal vector \(\vec{U}_{\mathrm{ym},j}\) (orange lines). It also updates a tangential vegetal-pole-oriented vector denoted by \(\vec{U}^{\parallel}_{\mathrm{ym},j}\) (blue lines), which will be used later on, in the fifth case study (Section 8.5).</figcaption>
</figure>
<figure id='Case_2_Cleavage_video_example_Cleavage_example'>
	<video src="../../videos/MECAGEN/Case_2_Cleavage/video_example/Cleavage_example.ogg" width="800" controls></video>
	<figcaption><strong>Superposition of nuclear centers from experimental and simulated data.</strong> The experimental data (green centers) is from the dataset ID 081014h of <a name='Olivier:2010jz' class='ref'></a>.<figcaption>
</figure>
<figure id='Case_2_Cleavage_measure2'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/measure2.png" width="900">
	<figcaption><strong>A 2D alpha shape strategy for assessing the embryo shape evolution.</strong> A: schematic view of the embryo, with the blastoderm area in blue and the yolk in beige. B: The vertical arrow indicates the animal-vegetal (AV) axis, the red line delineates the blastoderm margin, i.e. the interface between the yolk, the blastoderm and the environment. C: Each cell position is projected on a half-plane by rotation around the AV axis. D: A 2D alpha shape yields the positions of the cells at the blastoderm surface, characterizing the contour of the embryo. E: Two different 2D shapes from two embryos are superimposed. F: The discrete shapes are interpolated via analytical functions.</figcaption>
</figure>
<figure id='Case_2_Cleavage_babb_2004'>
	<img src="../../images/Cases_Studies/Case_2_Cleavage/babb_2004.png" width="700">
	<figcaption><strong>Early cleavage patterns and compaction are disrupted in injected embryos</strong>. Zebrafish embryos were injected with control morpholino oligonucleotide (MO), or cdh1 MO, and monitored by time-lapse microscopy. In control embryos (top panel), cleavage planes were regular, blastomeres were approximately equal in size, and cells were strongly adherent and formed a compact mass atop the yolk cell. In cdh1 MO-injected embryos (bottom two panels), the orientation of cleavage planes between blastomeres was irregular, cell size was unequal, adhesion between blastomeres was reduced, and the embryos did not compact normally. hpf: hours post-fertilization. Scale bar 100 \(\mu\)m. Image and caption from <a name='Babb:2004kx' class='ref'></a></figcaption>
</figure>
<figure id='growth_fink_1996'>
	<img src="../../images/Cases_Studies/Case_3_EVL/growth/fink_1996.png" width="400">
	<figcaption><strong>Regional membrane turnover within the apical membranes of <i>Fundulus</i> enveloping layer cells.</strong> Pairs of photographs showing the temporal progression of membrane turnover (seen as loss of fluorescence) of EVL cells labeled with fluorescent lectin (A-D) and lipid (E-H). Embryos stained with fluorescent lectin initially show a uniform pattern of fluorescence (A). Six hours later, apical membrane turnover has removed the fluorescent markers from regions where three or more cells come in contact. This loss of fluorescence continues, and by 18 hr (C) after staining, lectin turnover has spread from the cellular contact points along the cell margins. By 36 hr after lectin staining, fluorescence is restricted to a central domain (D). This same pattern of membrane turnover is seen when embryos are labeled with fluorescent lipid: (E) uniform staining at Time = 0 hr; (F) loss of fluorescence at cellular contact points 12 hr after staining; (G) formation of a nonfluorescent peripheral domain 29 hr after staining; and (H) lipid turnover 42 hr after staining creates a peripheral band that extends 10-12 \(\mu\)m inward from cell boundaries. Scale bar, 30 \(\mu\)m. Image and caption from Fink and Cooper, 1996 <a name='Fink:1996un' class='ref'></a></figcaption> 
</figure>
<figure id='Case_3_EVL_live_reconstruction_fusion'>
	<img src="../../images/Cases_Studies/Case_3_EVL/live_reconstruction_fusion.png" width="800">
	<figcaption><strong>Reconstruction of the EVL from 3D+time imaging.</strong> Both images belong to the same dataset 070418a and have been generated by the Mov-It software. A. Superposition of the convex hull (blue surface) calculated from the swarm of cell centers, the volume rendering of the cell membrane channel (yellow signal), the cell positions (cubes) and the cell neighborhood edges (lines connecting the cubes). The manual validation and correction of the cell positions, cell tracking and cell neighborhoods of the yellow/green population were corrected by Adeline Boyreau, the pink population corresponds to the automated reconstruction. B: Volume rendering of the nucleus channel.</figcaption>
</figure>
<figure id='Case_3_EVL_070418a_t_n10040_t10040_11931'>
	<video src="../../videos/MECAGEN/Case_3_EVL/070418a_t_n10040_t10040_11931.ogg" width="800" controls></video>
	<figcaption><strong>Dynamical evolution of the enveloping layer neighborhood from the high stage-oblong stage transition to the germ-ring stage.</strong> See Fig. <a name='Case_3_EVL_live_reconstruction_fusion' class='fig'></a> for caption.</figcaption>
</figure>
<figure id='Case_3_EVL_mitosis_superposition'>
	<img src="../../images/Cases_Studies/Case_3_EVL/mitosis_superposition.png" width="900">
	<figcaption><strong>Temporal alignment of cell divisions.</strong> The abstract examples of mitosis displayed to illustrate the strategy do not belong to the real measured population \(\mathcal{M}^l\). The colors identify individual mitoses \(m\) of \(\mathcal{M}^l\) in both panels. A: 5 mitoses occurring at various time steps. B: The same mitoses translated in the new timeline \(t'\). The measured quantities represented by the colored lines are available for each time step \(t'\). The quantities are considered if they belong to the \(\left[ -50, 50 \right]\) interval. Beyond this interval, the discarded values are covered by a white overlay.</figcaption>
</figure>
<figure id='Case_3_EVL_histogramdegree'>
	<img src="../../images/Cases_Studies/Case_3_EVL/histogram_fusion.png" width="800">
	<figcaption><strong>Histogram of the number of neighbors in the whole population \(\mathcal{E}\) and comparison with other tissues.</strong> A: In ordinate, the frequencies of the degree \(D^l_k\) observed in the corrected population \(\mathcal{E}\). Error bars indicate the standard deviation for each degree's frequency. Measures calculated and plotted by Paul Villoutreix. B: Comparison of the distributions of degrees in the <i>Drosophila</i> wing disc (blue, 2,172 cells), the <i>Hydra</i> epidermis (orange, 602 cells), the <i>Xenopus</i> tail epidermis (yellow, 1,051 cells), the chick pre-streak (green, 351 cells), the chick lateral streak (brown, 634 cells), the chick streak (pale blue, 613 cells), and the zebrafish enveloping layer during epiboly phase 1 (dark green, 5725). Data for <i>Drosophila</i>, <i>Hydra</i> and <i>Xenopus</i> are extracted from Gibson et al. <a name='Gibson:2006gia' class='ref'></a>, data for the chick epithelia are extracted from Sandersius et al. <a name='Sandersius:2011ug' class='ref'></a>.</figcaption> </figure>
<figure id='Case_3_EVL_surface_fusion'>
	<img src="../../images/Cases_Studies/Case_3_EVL/Epithelium/surface_fusion.png" width="950">
	<figcaption><strong>Evolution of the EVL cells' average surface.</strong> A: Temporal evolution of the average cell surface \(\lt\!S^l(t)\!\gt\) in the population \(\mathcal{E}\). The measure starts at time step 4 (3h52min) and ends at time step 103 (5h43min). Consecutive time steps are separated by 1min07s. An increase of 50 percent is measured during this period. B: Temporal evolution of the average cell surface \(S^l_{\mathrm{mit}}(t')\) around mitosis. All 52 mitoses are superposed as if they were occurring simultaneously at time step \(t'=0\). The cells' apical surfaces are normalized so that the surface of the cell measured immediately after mitosis is equal to 1. The average and standard deviation at each time step before and after mitosis are calculated and plotted. Both measures were calculated and plotted by Paul Villoutreix.</figcaption>
</figure>
<figure id='Case_3_EVL_degree_fusion'>
	<img src="../../images/Cases_Studies/Case_3_EVL/Epithelium/degree_fusion.png" width="950">
	<figcaption><strong>Evolution of the EVL cells' average degree.</strong> A: Temporal evolution of the averaged cell degree \(\lt\!n^l(t)\!\gt\) in the population \(\mathcal{E}\). The measure starts at the time step 4 (3h52min) and ends at the time step 103 (5h43min). Consecutive time steps are separated by 1min07s. The average degree remains roughly constant during this period. B: Temporal evolution of the average cell degree \(n^l_{\mathrm{mit}}(t')\) around mitosis. All 52 mitoses are superposed as if they were occurring simultaneously at time step \(t'=0\). The average and standard deviation at each time step before and after mitosis are calculated and plotted. Both measures were calculated and plotted by Paul Villoutreix.</figcaption>
</figure>
<figure id='Cases_Studies_Case_3_EVL_parallel_views'>
	<img src="../../images/Cases_Studies/Case_3_EVL/parallel_views.png" width="900">
	<figcaption><strong>The simulated embryo's EVL during epiboly phase 1.</strong> All deep cells are colored in blue. The color of the EVL cells indicates the degree of the cell (see legend). First row: the embryo is cut along the bilateral symmetry plane, the yolk is empty as no yolk interior particles are represented. Second row: external view of the embryo. A: onset of epiboly (approx. 3.9h simulated time). B: 30-percent epiboly stage (approx. 4.85h simulated time). C: 50-percent epiboly, the internalization is not modeled here (approx. 5.7h simulated time). The movie from which these images are extracted is Movie <a name='Case_3_EVL_parallel_views' class='fig'></a>.
	</figcaption>
</figure>
<figure id='Case_3_EVL_parallel_views'>
	<video src="../../videos/MECAGEN/Case_3_EVL/parallel_views.ogg" width="800" controls></video>
	<figcaption><strong>The simulated embryo's EVL during epiboly phase 1.</strong> see Fig. <a name='Cases_Studies_Case_3_EVL_parallel_views' class='fig'></a> for caption.</figcaption>
</figure>
<figure id='Case_3_EVL_parameter_space_table'>
	<img src="../../images/Cases_Studies/Case_3_EVL/parameter_space_table/parameter_space_table.png" width="500">
	<figcaption><strong>Range, cardinalities and units of the three parameters explored in this study.</strong></figcaption>
</figure>
<figure id='Case_3_EVL_fitnesses_surface_t'>
	<img src="../../images/Cases_Studies/Case_3_EVL/fitnesses/surface_t.png" width="700">
	<figcaption><strong>3D plot of the average surface evolution function \(F_S\).</strong> The vertical axis indicates the EVL growth ratio \(\gamma_{\mathrm{E}}\), the depth axis indicates the limit radius coefficient \(c^{\mathrm{lat,E}}_{\mathrm{lim}}\) and the horizontal axis indicates the pressure threshold \(\theta^{-}_{\mathrm{E}}\).</figcaption>
</figure>
<figure id='Case_3_EVL_fitnesses_degree_t'>
	<img src="../../images/Cases_Studies/Case_3_EVL/fitnesses/degree_t.png" width="700">
	<figcaption><strong>3D plot of the average degree evolution function \(F_n\).</strong> The red <q>potholes</q> visible on the lower limit radius coefficient \(c^{\mathrm{lat,E}}_{\mathrm{lim}}\) face of the cube are due to simulation divergence. See Fig. <a name='Case_3_EVL_fitnesses_surface_t' class='fig'></a> for details.</figcaption>
</figure>
<figure id='Case_3_EVL_fitnesses_histogram'>
	<img src="../../images/Cases_Studies/Case_3_EVL/fitnesses/histogram.png" width="700">
	<figcaption><strong>3D plot of the average degree distribution function \(F_D\).</strong> See Fig. <a name='Case_3_EVL_fitnesses_surface_t' class='fig'></a> for details.</figcaption>
</figure>
<figure id='Case_3_EVL_fitnesses_surface_mit'>
	<img src="../../images/Cases_Studies/Case_3_EVL/fitnesses/surface_mit.png" width="700">
	<figcaption><strong>3D plot of the average mitotic surface function \(F_{S,\mathrm{mit}}\).</strong> See Fig. <a name='Case_3_EVL_fitnesses_surface_t' class='fig'></a> for details.</figcaption>
</figure>
<figure id='Case_3_EVL_fitnesses_degree_mit'>
	<img src="../../images/Cases_Studies/Case_3_EVL/fitnesses/degree_mit.png" width="700">
	<figcaption><strong>3D plot of the average mitotic degree function \(F_{n,\mathrm{mit}}\).</strong> See Fig. <a name='Case_3_EVL_fitnesses_surface_t' class='fig'></a> for details.</figcaption>
</figure>
<figure id='Case_3_EVL_fitnesses_globale'>
	<img src="../../images/Cases_Studies/Case_3_EVL/fitnesses/globale.png" width="700">
	<figcaption><strong>3D plot of the global fitness function \(F\).</strong> See Fig. <a name='Case_3_EVL_fitnesses_surface_t' class='fig'></a> for details.</figcaption>
</figure>
<figure id='Case_4_intercalation_kimmel_1995'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/kimmel_1995.png" width="950">
	<figcaption><strong>Early gastrula period.</strong> Face views. A: transition between the high and oblong stages (3.5 h). B. transition between the oblong and sphere stages (3.8 h). C: Dome stage (4.3 h). D: 30-percent epiboly stage (4.7 h). E: 50-percent epiboly stage (5.25 h). Scale bar: 250 \(\mu\)m. Image and caption adapted from Kimmel et al. (1995) <a name='Kimmel:1995kn' class='ref'></a></figcaption>
</figure>
<figure id='Case_4_intercalation_071222bF_horizontal'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/071222bF_horizontal.png" width="900">
	<figcaption><strong>Wild type zebrafish embryo imaged live from the oblong stage to 30-percent epiboly. </strong> We observe here a slice passing approximately through the center and the animal pole of the embryo. A: oblong stage. B: transition between oblong and sphere stage. C: sphere stage. D: transition between sphere and dome stage. E: dome stage. F: transition from dome to 30-percent epiboly stage. Dataset ID 071222bF. The movie from which these images are extracted is available in <a name='Epiboly_071222bF_tranche_radiale_2' class='fig'></a></figcaption>
</figure>
<figure id='Epiboly_071222bF_tranche_radiale_2'>
		<video src="../../videos/Epiboly/071222bF_tranche_radiale_2.ogg" width="800" controls></video>
		<figcaption><strong>Wild type zebrafish embryo from the 32-cell stage to the 75-percent epiboly stage. </strong> We observe a slice passing approximatively through the center and the animal pole of the embryo. The period of interest is from the high stage (\(T=34\)) to the germ ring stage (\(T=110\)). We notice empty gaps that appear under the EVL during the phase 1 epibolic motion, suggesting that the deep cells would intercalate more toward the yolk than toward the EVL. Dataset ID 071222bF.</figcaption>
	</figure>
<figure id='Case_4_intercalation_margin_mechanism'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/margin_mechanism_all2.png" width="950">
	<figcaption><strong>Mechanism controlling the passive sliding of the blastoderm and EVL margin toward the vegetal pole. </strong> A: Sagittal section of the simulated embryo at the onset of epiboly. B: Sagittal section at 30 percent epiboly. C: Zoom on the marginal region of the embryo at 30 percent epiboly. When a deep cell (DC, red) \(j\) is in contact (green lines) with a margin yolk membrane (MYM, orange) particle \(i\), a <q>pushing force</q> \(F^{\mathrm{m:D}, \parallel}_{ij}\) is calculated to estimate the mechanical pressure exerted by exerted by \(j\) on the margin at the cellular level. If this force exceeds a certain <q>resistance threshold</q> \(\theta_{\mathrm{m}, \parallel}\), the MYM particle looses its marginal properties, and transmits it to one or several regular YM particle(s) at a more vegetal latitude. C: Schema describing the pushing force exerted by DC particle \(j\) over MYM cell \(i\). The pushing force is non-zero only if the distance between the positions of \(j\) and \(i\) projected on the tangential vector \(\vec{U}^{\parallel}_{\mathrm{ym},i}\) (not shown) is smaller than the equilibrium distance \(r^{\mathrm{eq}, \parallel}_{ij} = c_{\mathrm{eq}} (R_i + R_{\mathrm{ym}}) \). The orange and red circles highlight the radii of cells \(j\) and \(i\) respectively.</figcaption>
</figure>
<figure id='Case_4_intercalation_source_sink_ligand'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/source_sink_ligand/source_sink_ligand.png" width="900">
	<figcaption><strong>Ligand diffusion table \(\mathscr{L}\) for the <q>Intercalation</q> case study.</strong> Two gradient fields are established by ligand diffusion across the dome region formed by the deep cells. They contribute to the specification of the axes of the polarization: a ligand \(Q_1\) is secreted by the EVL with secretion rate \(s_1 = 1000\) and absorbed by the yolk membrane particles, more precisely the YSL. Conversely, a ligand \(Q_2\) is secreted by the YSL with rate \(s_2 = 1000\) and absorbed by the EVL.</figcaption>
</figure>
<figure id='Case_4_intercalation_params_protrusion'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/params_protrusion.png" width="700">
	<figcaption><strong>Active protrusion table \(\mathscr{A}\) for the <q>Intercalation</q> case study.</strong> A: All the cells belonging to the \(\mathcal{T}_{\mathrm{red}}\) population exert bipolar protrusions over each other along the polarization axis \(\vec{U}_1\) derived from the gradient field of ligand \(Q_1\). B: The same cells can exert another bipolar protrusion along \(\vec{U}_2\) derived from \(Q_2\). In both cases, the intensity of the protrusive force is \(f^A\). </figcaption>
</figure>
<figure id='kane_dev_fusion'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/kane_dev/fusion.png" width="800">
	<figcaption><strong>Macroscopic landmarks of the epibolic deformation from the <q>flipbook</q> specimen.</strong> Snapshots of the zebrafish development from the oblong stage to 50 percent epiboly extracted from the movie <a name='Zebra_Zebrafish_Development_kane' class='fig'></a> by Karlstrom and Kane <a name='Karlstrom:1996wo' class='ref'></a>. Colored dots have been manually added to estimate the macroscopic morphological characteristics of the embryo: red dots signal the animal pole of each embryo, green dots the vegetal pole, blue dots the animal-pole limit of the yolk, and triplets of yellow dots delineate the margin, where the left and right dots identify the external position of the margin and the central one correspond to their averaged projection on the AV axis. The time value displayed below the image id is the time in hours post fertilization given by <a name='Karlstrom:1996wo' class='ref'></a>. These timings do not scale linearly with the image ids and have been renormalized.</figcaption>
</figure>
<figure id='kane_dev_fusion_plot'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/kane_dev/fusion_plot.png" width="800">
	<figcaption><strong>Macroscopic measures of the epibolic deformation in the <q>flipbook</q> specimen.</strong> The measures defined by the macroscopic landmarks displayed in Fig. <a name='kane_dev_fusion' class='fig'></a> are shown. The red line is the plot of the distance between the animal pole (AP) of the embryo and the vegetal pole (VP) of the yolk (embryo height). The green line is the plot of the distance between the projection of the margin on the animal-vegetal (AV) axis and the VP of the yolk (margin height). The blue line is distance between the AP of the yolk and the VP of the yolk (yolk height). The yellow line is the lateral distance between the margin positions (margin diameter). The dashed lines give the absolute distance between landmarks in pixels (left ordinate axis). The continuous lines give the normalized distances (right ordinate axis). The normalization is obtained by dividing each value by the current yolk height (i.e. dashed red line). The abscissa gives the time in hour post fertilization.</figcaption>
</figure>
<figure id='Case_4_intercalation_landmarks_simulated'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/landmarks_simulated.png" width="700">
	<figcaption><strong>Macroscopic landmarks of the epibolic deformation in simulated specimens.</strong> Landmark dots have been manually added for the purpose of the illustration. They represent the macroscopic landmarks automatically calculated by the simulated measurements. The grey line is the animal-vegetal (AV) axis, <i>a priori</i> specified for the initialization. The red dot is the embryo animal pole \(\mathbf{M}^{\mathrm{E}}_{\mathrm{A}}\), the blue dot is the yolk animal pole \(\mathbf{M}^{\mathrm{Y}}_{\mathrm{A}}\), the black dot is the embryo center \(\mathbf{M}_{\mathrm{o}}\), the green dot is the embryo vegetal pole \(\mathbf{M}^{\mathrm{E}}_{\mathrm{V}}\), the orange dot is the margin <q>level</q> \(\mathbf{M}_{\mathrm{m}}\), i.e. the average of all the projections of margin yolk membrane particles (small orange circles) on the AV axis (only a few of these particles are displayed). Note that the position of the yolk animal pole, embryo animal pole and embryo vegetal pole are never exactly aligned on the AV axis as they must coincide with one particle's position. The deep cells are not displayed here, leaving a carved out domain between the yolk cell and the EVL.</figcaption>
</figure>
<figure id='Case_4_intercalation_simulated_measures_macro'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/epiboly_f3500_s0_evl60_marg7_r0_measures.png" width="900">
	<figcaption><strong>Macroscopic measures of the epibolic deformation in a simulated specimen.</strong> Six snapshots corresponding to different stages of a simulated embryo are displayed along with the temporal evolution of the corresponding macroscopic measures (blue), compared with the flipbook measures (white). Sphericity is represented by the dash-dot lines, normalized yolk height by the solid lines and normalized margin height by the dashed lines. The embryos' positions correspond approximately to the snapshot times in abscissa. The simulated embryo parameters are: ligand source \(Q_2\) on the yolk, \(\theta_{\mathrm{m}, \parallel} = 5.33\), \(\lambda_r = 0\), \(\theta^{-}_{\mathrm{E}} = 56.67\) and \(f^A = 3556\). The movie from which these snapshots are extracted can be viewed in <a name='Case_4_Intercalation_epiboly_f3500_s0_evl60_marg7_r0' class='fig'></a>.</figcaption>
</figure>
<figure id='Case_4_Intercalation_epiboly_f3500_s0_evl60_marg7_r0'>
	<video src="../../videos/MECAGEN/Case_4_Intercalation/epiboly_f3500_s0_evl60_marg7_r0.ogg" width="800" controls></video>
	<figcaption><strong>Simulated embryo during epiboly phase 1.</strong> see Fig. <a name='Case_4_intercalation_simulated_measures_macro' class='fig'></a> for caption.</figcaption>
</figure>
<figure id='Case_4_intercalation_parameter_space_table'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/parameter_space_table/parameter_space_table.png" width="500">
	<figcaption><strong>Range, cardinalities and units of the five parameters explored in this study.</strong></figcaption>
</figure>
<figure id='source_1_evlg_4_margin_5'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/plot_all_2D/source_1_evlg_4/margin_5.png" width="500">
	<figcaption><strong>Fitness landscapes as a function of the random parameter \(\lambda_r\) and the protrusive force intensity parameter \(f^A\). </strong> Top Left: global fitness function. Top Right: normalized yolk height objective function \(F_{\mathrm{Y}}\). Bottom Left: normalized margin height function  \(F_{\mathrm{m}}\). Bottom Right: sphericity objective function \(F_{\Phi}\). On each plot, the random parameter \(\lambda_r\) is shown in abscissa and the protrusive force intensity \(f^A\) in ordinate. The color maps scale between zero and the maximum value of the fitness landscape that they are associated with (to the left).</figcaption>
</figure>
<figure id='plot_all_2D_source_1_evlg_4'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/plot_all_2D/source_1_evlg_4.png" width="900">
	<figcaption><strong>Fitness landscapes as a function of the random parameter \(\lambda_r\) and the protrusive force intensity parameter \(f^A\), with the yolk as a ligand source (value 2).</strong> Each of the 9 subplots is equivalent to the landscapes presented in Fig. <a name='source_1_evlg_4_margin_5' class='fig'></a>. From top left to bottom right, the margin resistance \(\theta_{\mathrm{m}, \parallel}\) increases regularly from 1 to 35.666 (the last subplot for value 40 was removed to avoid excessive image reduction). The threshold controlling the EVL resistance \(\theta^{-}_{\mathrm{E}}\) is set to 72. The whole dataset of these fitness landscapes is <a href="http://public.iscpif.fr/~delile/files/manuscript_data/intercalation_pattern_2D.zip">available here</a>.</figcaption>
</figure>
<figure id='plot_all_2D_source_2_evlg_4'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/plot_all_2D/source_2_evlg_4.png" width="900">
	<figcaption><strong>Fitness landscapes as a function of the random parameter \(\lambda_r\) and the protrusive force intensity parameter \(f^A\), with the EVL as a ligand source (value 1).</strong> Except for the source, the parameter space mapped here is the same as in Fig. <a name='plot_all_2D_source_1_evlg_4' class='fig'></a></figcaption>
</figure>
<figure id='3D_fitnesses_macro_margin'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/3D_fitnesses_macro/margin.png" width="900">
	<figcaption><strong>3D plot of the normalized margin height objective function \(F_{\mathrm{m}}\).</strong> The isosurface represents the best fitness volume (threshold 0.38). The protrusive force intensity \(f^A\) has been set to 3555. The vertical axis indicates the random parameter \(\lambda_r\), the depth axis indicates the threshold controlling the EVL resistance \(\theta^{-}_{\mathrm{E}}\) and the horizontal axis indicates the margin resistance threshold \(\theta_{\mathrm{m}, \parallel}\). This 3D objective function landscape has been rotated, unlike the following ones, to show a better angle of view. The interactive 4D visualization tool is <a href="http://public.iscpif.fr/~delile/files/manuscript_data/8_5_matlab_fitness_data.zip">available here</a> for download (MATLAB required)</figcaption>
</figure>
<figure id='3D_fitnesses_macro_doming'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/3D_fitnesses_macro/doming.png" width="900">
	<figcaption><strong>3D plot of the normalized yolk height objective function \(F_{\mathrm{Y}}\).</strong> The threshold of the isosurface is set to 0.56. See Fig. <a name='3D_fitnesses_macro_margin' class='fig'></a> for details.</figcaption>
</figure>
<figure id='3D_fitnesses_macro_sphericity'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/3D_fitnesses_macro/sphericity.png" width="900">
	<figcaption><strong>3D plot of the sphericity objective function \(F_{\Phi}\).</strong> The threshold of the isosurface is set to 1.20. See Fig. <a name='3D_fitnesses_macro_margin' class='fig'></a> for details.</figcaption>
</figure>
<figure id='3D_fitnesses_macro_global'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/3D_fitnesses_macro/global.png" width="900">
	<figcaption><strong>3D plot of the global fitness function \(F\).</strong> The threshold of the isosurface is set to 0.69. See Fig. <a name='3D_fitnesses_macro_margin' class='fig'></a> for details.</figcaption>
</figure>
<figure id='Case_4_intercalation_micro_measure'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/micro_measures.png" width="400">
	<figcaption><strong>Normalized position of the cells along the radial axis \(\mathcal{L}^R_i\).</strong> The measure is applied to the cell \(i\) in green shape with its approximate center as a black dot, at position \(X_i\). It represents the fraction of the height of \(X_i\) on the segment going from \(X_{i, \mathrm{Y}}\) to \(X_{i, \mathrm{E}}\).</figcaption>
</figure>
<figure id='Case_4_intercalation_micro_rho'>
	<img src="../../images/Cases_Studies/Case_4_intercalation/micro_rho.png" width="800">
	<figcaption><strong>Illsutration of the microscopic measure of cell intercalation \(M^{R}_{i}\).</strong> Three fictive cell trajectories in normalized radial displacement space: Cell 0 (blue), Cell 1 (red), Cell 2 (yellow). Cell 1 performs a continously increasing radial displacement between position \(0.1\) at \(t_0\) and position \(0.55\) at \(t_9\), yielding ameasure of \(M^{R}_{1} = 0.45\). Cells 0 and 2 display random radial displacements through time, which are characterized by higher microscopic measure values \(M^{R}_{0} = 1.45\) and \(M^{R}_{2} = 1.55\).</figcaption>
</figure>
<figure id='Case_4_Intercalation_intercalation_cylinder_test'>
	<video src="../../videos/MECAGEN/Case_4_Intercalation/intercalation_cylinder_test.ogg" width="800" controls></video>
	<figcaption><strong>Tangential dispersion of the cells during intercalating behaviors. </strong> Cylinders similar to the one visible at the beginning of the video could be used at various position to characterize the tangential spreading of cell population.</figcaption>
</figure>
<figure id='Case_7_Convergence_extension_mzoep_fusion'>
	<img src="../../images/Cases_Studies/Case_7_Convergence_extension/mzoep_fusion.png" width="800">
	<figcaption><strong><i>MZoep</i> mutant.</strong> A: Live <i>MZoep</i> mutant embryo at 30 hours of development. The ventral mesoderm gave rise to a few somites in the most posterior part of the embryonic axis. The epiblast converged, shaping the antero-posterior axis. Image and caption adapted from Mathieu et al. <a name='Mathieu:2004gw' class='ref'></a> B: Dorsal markers expressed in the ectoderm (<i>otx2</i>, anteriorly, and <i>hoxb1b</i>, posteriorly), are still present in the <i>MZoep</i> mutant. Both domains are however narrower in the mutant. An interesting interpretation would be that the narrower domains are the result of a faster convergence in the mutant, due to the lack of hypoblast. Image adapted from Jia et al. <a name='Jia:2009ez' class='ref'></a>.</figcaption>
</figure>
<figure id='Case_7_Convergence_extension_carmany_rampey_2001'>
	<img src="../../images/Cases_Studies/Case_7_Convergence_extension/carmany_rampey_2001.png" width="600">
	<figcaption><strong>Dorso-ventral displacement of the cells during convergent extension.</strong> Each arc identifies individual cell positions along the dorso-ventral longitudes at a given time step. The black lines link the same individual cells at both time steps. The sum of the lengths of these links correlates with the speed of intercalation. Image and caption adapted from Carmany-Rampey and Schier <a name='CarmanyRampey:2001uy' class='ref'></a></figcaption>
</figure>
<figure id='Case_0_Yolk_1_THG_imaging_the_zebrafish_embryo_from_the_one_cell_stage'>
	<video src="../../videos/MECAGEN/Case_0_Yolk/1_THG_imaging_the_zebrafish_embryo_from_the_one_cell_stage.ogg" width="800" controls></video>
	<figcaption><i>Third harmonic generation</i> (THG) imaging of the zebrafish embryo adapted from <a name='Olivier:2010jz' class='ref'></a></figcaption>
</figure>
<figure id='Case_0_Yolk_yolk_relaxation_example'>
	<video src="../../videos/MECAGEN/Case_0_Yolk/yolk_relaxation_example.ogg" width="800" controls></video>
	<figcaption></figcaption>
</figure>
<figure id='Case_0_Yolk_yolk_external'>
	<video src="../../videos/MECAGEN/Case_0_Yolk/yolk_external.ogg" width="800" controls></video>
	<figcaption></figcaption>
</figure>
<figure id='Case_0_Yolk_yolk_relaxation_cut'>
	<video src="../../videos/MECAGEN/Case_0_Yolk/yolk_relaxation_cut.ogg" width="800" controls></video>
	<figcaption></figcaption>
</figure>
<figure id='Zebra_Zebrafish_Development_kane'>
	<video src="../../videos/in_vivo/Zebra/Zebrafish_Development_kane.ogg" width="800" controls></video>
	<figcaption><strong>Movie of Zebrafish Development, 2-cell stage to 16-somite stage (approx. 17h later).</strong> By Rolf Karlstrom and Don Kane; Development 123:461, 1996 <a name='Karlstrom:1996wo' class='ref'></a> </figcaption>
</figure>
<figure id='Case_4_Intercalation_fluid_like_no_epiboly'>
	<video src="../../videos/MECAGEN/Case_4_Intercalation/fluid_like_no_epiboly.ogg" width="800" controls></video>
	<figcaption><strong>Embryo performing fluid-like intercalation.</strong> As the active protrusion forces are exceeding the passive cohesion forces in the cell tissue, cells intercalate wildly in the cell mass, without initiating an epibolic deformation. The key parameters in this simulation are: ligand source from the YSL, \(\theta_{\mathrm{mym}, \parallel}\) = 15, \(\lambda_r\) = 0.06, \(\theta^{-}_{\mathrm{evl}}\) = 72 and \(f^A\) = 6000. See 8.5.1 for details about the simulation.</figcaption>
</figure>
<figure id='Case_4_Intercalation_yolk_perforation'>
	<video src="../../videos/MECAGEN/Case_4_Intercalation/yolk_perforation.ogg" width="800" controls></video>
	<figcaption><strong>Deep cell perforating the yolk membrane during phase 1 epiboly.</strong> This simulated embryo possesses an enveloping layer (EVL) that resists to external tension and hardly allows an increase of its surface. The consequence is dramatic when the deep cell margin reaches the yolk equatorial latitude, a second abnormal phenomenon happens as, due to a very low margin resistance, the yolk membrane starts to roll at the margin, blocking further deep cells' advance. The rolled yolk membrane gets stretched and eventually is not sufficient to restrain the deep cells progression into the yolk volume. The key parameters in this simulation are: ligand source from the YSL, \(\theta_{\mathrm{mym}, \parallel}\) = 3.6, \(\lambda_r\) = 0.0, \(\theta^{-}_{\mathrm{evl}}\) = 150 and \(f^A\) = 3500. See 8.5.1 for details about the simulation.</figcaption>
</figure>
<figure id='Case_4_Intercalation_just_margin'>
	<video src="../../videos/MECAGEN/Case_4_Intercalation/just_margin.ogg" width="800" controls></video>
	<figcaption><strong>Relaxed EVL and margin, and random polarization during phase 1 epiboly.</strong> Left: YSL as a source of the polarization field scenario. Right EVL as a source scenario. The parameters in this simulation are: \(\theta_{\mathrm{mym}, \parallel}\) = 1, \(\lambda_r\) = 0.1, \(\theta^{-}_{\mathrm{evl}}\) = 10 and \(f^A\) = 3500. Poor movie rendering have slightly elongated the shape of both embryos.</figcaption>
</figure>
<figure id='Case_4_Intercalation_constrained_deep_cells'>
	<video src="../../videos/MECAGEN/Case_4_Intercalation/constrained_deep_cells.ogg" width="800" controls></video>
	<figcaption><strong>Constraining EVL and margin, with regular polarization fields during phase 1 epiboly.</strong> Left: YSL as a source of the polarization field scenario. Right EVL as a source scenario. The parameters in this simulation are: \(\theta_{\mathrm{mym}, \parallel}\) = 35, \(\lambda_r\) = 0.005, \(\theta^{-}_{\mathrm{evl}}\) = 120 and \(f^A\) = 3500.</figcaption>
</figure>
<figure id='Case_3_EVL_video_example_zone1x3_EVLexample_1x3'>
	<video src="../../videos/MECAGEN/Case_3_EVL/video_example_zone1x3/EVLexample_1x3.ogg" width="800" controls></video>
	<figcaption>The enveloping layer compartment measures can be regionalized in spatial zone. Here, we define 3 zones along the animal-vegetal axis.</figcaption>
</figure>
